<!DOCTYPE html>
<html class="client-nojs" lang="en" dir="ltr">
<head>
<meta charset="UTF-8"/>
<title>Protein - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Protein","wgTitle":"Protein","wgCurRevisionId":859931995,"wgRevisionId":859931995,"wgArticleId":23634,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1 German-language sources (de)","Wikipedia indefinitely semi-protected pages","Articles containing potentially dated statements from 2017","All articles containing potentially dated statements","Good articles","Wikipedia articles with BNF identifiers","Wikipedia articles with GND identifiers","Wikipedia articles with LCCN identifiers","Wikipedia articles with NDL identifiers","Molecular biology","Proteins","Proteomics"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Protein","wgRelevantArticleId":23634,"wgRequestId":"W6a-4QrAEEYAACWAYp4AAAAX","wgCSPNonce":false,"wgIsProbablyEditable":false,"wgRelevantPageIsProbablyEditable":false,"wgRestrictionEdit":["autoconfirmed"],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgWikiEditorEnabledModules":[],"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsShouldSendModuleToUser":true,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFExpandAllSectionsUserOption":true,"wgMFEnableFontChanger":true,"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q8054","wgScoreNoteLanguages":{"arabic":"العربية","catalan":"català","deutsch":"Deutsch","english":"English","espanol":"español","italiano":"italiano","nederlands":"Nederlands","norsk":"norsk","portugues":"português","suomi":"suomi","svenska":"svenska","vlaams":"West-Vlams"},"wgScoreDefaultNoteLanguage":"nederlands","wgCentralAuthMobileDomain":false,"wgCodeMirrorEnabled":true,"wgVisualEditorToolbarScrollOffset":0,"wgVisualEditorUnsupportedEditParams":["undo","undoafter","veswitched"],"wgEditSubmitButtonLabelPublish":true});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@1dqfd7l",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});mw.loader.load(["ext.cite.a11y","site","mediawiki.page.startup","mediawiki.user","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging.subscriber","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.3d","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"]);});</script>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector"/>
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector"/>
<link rel="stylesheet" href="/w/load.php?debug=false&amp;lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector"/>
<meta name="generator" content="MediaWiki 1.32.0-wmf.22"/>
<meta name="referrer" content="origin"/>
<meta name="referrer" content="origin-when-crossorigin"/>
<meta name="referrer" content="origin-when-cross-origin"/>
<meta property="og:image" content="https:https://upload.wikimedia.org/wikipedia/commons/6/60/Myoglobin.png"/>
<link rel="alternate" href="android-app:https://org.wikipedia/http/en.m.wikipedia.org/wiki/Protein"/>
<link rel="apple-touch-icon" href="/static/apple-touch/wikipedia.png"/>
<link rel="shortcut icon" href="/static/favicon/wikipedia.ico"/>
<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (en)"/>
<link rel="EditURI" type="application/rsd+xml" href="https://en.wikipedia.org/w/api.php?action=rsd"/>
<link rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/"/>
<link rel="canonical" href="https:https://en.wikipedia.org/wiki/Protein"/>
<link rel="dns-prefetch" href="https://login.wikimedia.org"/>
<link rel="dns-prefetch" href="https://meta.wikimedia.org" />
<!--[if lt IE 9]><script src="/w/load.php?debug=false&amp;lang=en&amp;modules=html5shiv&amp;only=scripts&amp;skin=vector&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject page-Protein rootpage-Protein skin-vector action-view">		<div id="mw-page-base" class="noprint"></div>
		<div id="mw-head-base" class="noprint"></div>
		<div id="content" class="mw-body" role="main">
			</a>
			<div id="siteNotice" class="mw-body-content"><!-- CentralNotice --></div><div class="mw-indicators mw-body-content">
<div id="mw-indicator-good-star" class="mw-indicator"><img alt="This is a good article. Follow the link for more information." src="https://upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/19px-Symbol_support_vote.svg.png" width="19" height="20" srcset="https://upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/29px-Symbol_support_vote.svg.png 1.5x, https://upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/39px-Symbol_support_vote.svg.png 2x" data-file-width="180" data-file-height="185" /></a></div>
<div id="mw-indicator-pp-default" class="mw-indicator"><img alt="Page semi-protected" src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Padlock-silver.svg/20px-Padlock-silver.svg.png" width="20" height="20" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Padlock-silver.svg/30px-Padlock-silver.svg.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Padlock-silver.svg/40px-Padlock-silver.svg.png 2x" data-file-width="128" data-file-height="128" /></a></div>
</div>
<h1 id="firstHeading" class="firstHeading" lang="en">Protein</h1>			<div id="bodyContent" class="mw-body-content">
				<div id="siteSub" class="noprint">From Wikipedia, the free encyclopedia</div>				<div id="contentSub"></div>
				<div id="jump-to-nav"></div>				Jump to navigation</a>
				Jump to search</a>
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div role="note" class="hatnote navigation-not-searchable">This article is about a class of molecules. For protein as a nutrient, see protein (nutrient)</a>. For other uses, see protein (disambiguation)</a>.</div>
<p class="mw-empty-elt">
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/60/Myoglobin.png/220px-Myoglobin.png" width="220" height="223" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/6/60/Myoglobin.png/330px-Myoglobin.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/6/60/Myoglobin.png/440px-Myoglobin.png 2x" data-file-width="1159" data-file-height="1173" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>1</h1></center></div>A representation of the 3D structure of the protein myoglobin</a> showing turquoise α-helices</a>. This protein was the first to have its structure solved by X-ray crystallography</a>. Towards the right-center among the coils, a prosthetic group</a> called a heme group</a> (shown in gray) with a bound oxygen molecule (red).</div></div></div>
<p><b>Proteins</b> are large biomolecules</a>, or macromolecules</a>, consisting of one or more long chains of amino acid</a> residues</a>. Proteins perform a vast array of functions within organisms</a>, including catalysing metabolic reactions</a>, DNA replication</a>, responding to stimuli</a>, and transporting molecules</a> from one location to another. Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence</a> of their genes</a>, and which usually results in protein folding</a> into a specific three-dimensional structure</a> that determines its activity.
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p><p>A linear chain of amino acid residues is called a polypeptide</a>. A protein contains at least one long polypeptide. Short polypeptides, containing less than 20–30 residues, are rarely considered to be proteins and are commonly called peptides</a>, or sometimes oligopeptides</a>. The individual amino acid residues are bonded together by peptide bonds</a> and adjacent amino acid residues. The sequence</a> of amino acid residues in a protein is defined by the sequence</a> of a gene</a>, which is encoded in the genetic code</a>. In general, the genetic code specifies 20 standard amino acids; however, in certain organisms the genetic code can include selenocysteine</a> and—in certain archaea</a>—pyrrolysine</a>. Shortly after or even during synthesis, the residues in a protein are often chemically modified by post-translational modification</a>, which alters the physical and chemical properties, folding, stability, activity, and ultimately, the function of the proteins. Sometimes proteins have non-peptide groups attached, which can be called prosthetic groups</a> or cofactors</a>. Proteins can also work together to achieve a particular function, and they often associate to form stable protein complexes</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p><p>Once formed, proteins only exist for a certain period and are then degraded</a> and recycled by the cell's machinery through the process of protein turnover</a>. A protein's lifespan is measured in terms of its half-life</a> and covers a wide range. They can exist for minutes or years with an average lifespan of 1–2 days in mammalian cells. Abnormal or misfolded proteins are degraded more rapidly either due to being targeted for destruction or due to being unstable.
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p><p>Like other biological macromolecules such as polysaccharides</a> and nucleic acids</a>, proteins are essential parts of organisms and participate in virtually every process within cells</a>. Many proteins are enzymes</a> that catalyse</a> biochemical reactions and are vital to metabolism</a>. Proteins also have structural or mechanical functions, such as actin</a> and myosin</a> in muscle and the proteins in the cytoskeleton</a>, which form a system of scaffolding</a> that maintains cell shape. Other proteins are important in cell signaling</a>, immune responses</a>, cell adhesion</a>, and the cell cycle</a>. In animals, proteins are needed in the diet</a> to provide the essential amino acids</a> that cannot be synthesized</a>. Digestion</a> breaks the proteins down for use in the metabolism.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p><p>Proteins may be purified</a> from other cellular components using a variety of techniques such as ultracentrifugation</a>, precipitation</a>, electrophoresis</a>, and chromatography</a>; the advent of genetic engineering</a> has made possible a number of methods to facilitate purification. Methods commonly used to study protein structure and function include immunohistochemistry</a>, site-directed mutagenesis</a>, X-ray crystallography</a>, nuclear magnetic resonance</a> and mass spectrometry</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p>
<div id="toc" class="toc"><input type="checkbox" role="button" id="toctogglecheckbox" class="toctogglecheckbox" style="display:none" /><div class="toctitle" lang="en" dir="ltr"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">Biochemistry</span></a>
<ul>
<li class="toclevel-2 tocsection-2"><span class="tocnumber">1.1</span> <span class="toctext">Interactions</span></a></li>
<li class="toclevel-2 tocsection-3"><span class="tocnumber">1.2</span> <span class="toctext">Abundance in cells</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-4"><span class="tocnumber">2</span> <span class="toctext">Synthesis</span></a>
<ul>
<li class="toclevel-2 tocsection-5"><span class="tocnumber">2.1</span> <span class="toctext">Biosynthesis</span></a></li>
<li class="toclevel-2 tocsection-6"><span class="tocnumber">2.2</span> <span class="toctext">Chemical synthesis</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-7"><span class="tocnumber">3</span> <span class="toctext">Structure</span></a>
<ul>
<li class="toclevel-2 tocsection-8"><span class="tocnumber">3.1</span> <span class="toctext">Protein domains</span></a></li>
<li class="toclevel-2 tocsection-9"><span class="tocnumber">3.2</span> <span class="toctext">Sequence motif</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-10"><span class="tocnumber">4</span> <span class="toctext">Cellular functions</span></a>
<ul>
<li class="toclevel-2 tocsection-11"><span class="tocnumber">4.1</span> <span class="toctext">Enzymes</span></a></li>
<li class="toclevel-2 tocsection-12"><span class="tocnumber">4.2</span> <span class="toctext">Cell signaling and ligand binding</span></a></li>
<li class="toclevel-2 tocsection-13"><span class="tocnumber">4.3</span> <span class="toctext">Structural proteins</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-14"><span class="tocnumber">5</span> <span class="toctext">Methods of study</span></a>
<ul>
<li class="toclevel-2 tocsection-15"><span class="tocnumber">5.1</span> <span class="toctext">Protein purification</span></a></li>
<li class="toclevel-2 tocsection-16"><span class="tocnumber">5.2</span> <span class="toctext">Cellular localization</span></a></li>
<li class="toclevel-2 tocsection-17"><span class="tocnumber">5.3</span> <span class="toctext">Proteomics</span></a></li>
<li class="toclevel-2 tocsection-18"><span class="tocnumber">5.4</span> <span class="toctext">Bioinformatics</span></a></li>
<li class="toclevel-2 tocsection-19"><span class="tocnumber">5.5</span> <span class="toctext">Structure determination</span></a></li>
<li class="toclevel-2 tocsection-20"><span class="tocnumber">5.6</span> <span class="toctext">Structure prediction and simulation</span></a>
<ul>
<li class="toclevel-3 tocsection-21"><span class="tocnumber">5.6.1</span> <span class="toctext">Protein disorder and unstructure prediction</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-22"><span class="tocnumber">6</span> <span class="toctext">Nutrition</span></a></li>
<li class="toclevel-1 tocsection-23"><span class="tocnumber">7</span> <span class="toctext">History and etymology</span></a></li>
<li class="toclevel-1 tocsection-24"><span class="tocnumber">8</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-25"><span class="tocnumber">9</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1 tocsection-26"><span class="tocnumber">10</span> <span class="toctext">Textbooks</span></a></li>
<li class="toclevel-1 tocsection-27"><span class="tocnumber">11</span> <span class="toctext">External links</span></a>
<ul>
<li class="toclevel-2 tocsection-28"><span class="tocnumber">11.1</span> <span class="toctext">Databases and projects</span></a></li>
<li class="toclevel-2 tocsection-29"><span class="tocnumber">11.2</span> <span class="toctext">Tutorials and educational websites</span></a></li>
</ul>
</li>
</ul>
</div>

<h2><span class="mw-headline" id="Biochemistry">Biochemistry</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Peptide-Figure-Revised.png/300px-Peptide-Figure-Revised.png" width="300" height="231" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Peptide-Figure-Revised.png/450px-Peptide-Figure-Revised.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Peptide-Figure-Revised.png/600px-Peptide-Figure-Revised.png 2x" data-file-width="979" data-file-height="753" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>2</h1></center></div>Chemical structure of the peptide bond (bottom) and the three-dimensional structure of a peptide bond between an alanine</a> and an adjacent amino acid (top/inset). The bond itself is made  of the CHON</a> elements.</div></div></div>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/Peptide_group_resonance.png/300px-Peptide_group_resonance.png" width="300" height="65" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/1/17/Peptide_group_resonance.png 1.5x" data-file-width="393" data-file-height="85" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>3</h1></center></div>Resonance</a> structures of the peptide bond</a> that links individual amino acids to form a protein polymer</a></div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main articles: Biochemistry</a>, Amino acid</a>, and Peptide bond</a></div>
<p>Most proteins consist of linear polymers</a> built from series of up to 20 different <small>L</small>-α-</a>amino acids</a>. All proteinogenic amino acids</a> possess common structural features, including an α-carbon</a> to which an amino</a> group, a carboxyl</a> group, and a variable side chain</a> are bonded</a>. Only proline</a> differs from this basic structure as it contains an unusual ring to the N-end amine group, which forces the CO–NH amide moiety into a fixed conformation.<sup id="cite_ref-Nelson2005_1-0" class="reference">&#91;1&#93;</a></sup> The side chains of the standard amino acids, detailed in the list of standard amino acids</a>, have a great variety of chemical structures and properties; it is the combined effect of all of the amino acid side chains in a protein that ultimately determines its three-dimensional structure and its chemical reactivity.<sup id="cite_ref-Gutteridge2005_2-0" class="reference">&#91;2&#93;</a></sup>
The amino acids</a> in a polypeptide chain are linked by peptide bonds</a>. Once linked in the protein chain, an individual amino acid is called a <i>residue,</i> and the linked series of carbon, nitrogen, and oxygen atoms are known as the <i>main chain</i> or <i>protein backbone.</i><sup id="cite_ref-3" class="reference">&#91;3&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p><p>The peptide bond has two resonance</a> forms that contribute some double-bond</a> character and inhibit rotation around its axis, so that the alpha carbons are roughly coplanar</a>. The other two dihedral angles</a> in the peptide bond determine the local shape assumed by the protein backbone.<sup id="cite_ref-4" class="reference">&#91;4&#93;</a></sup> The end with a free amino group is known as the N-terminus</a> or amino terminus, whereas the end of the protein with a free carboxyl group is known as the C-terminus</a> or carboxy terminus (the sequence of the protein is written from N-terminus to C-terminus, from left to right).
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p><p>The words <i>protein</i>, <i>polypeptide,</i> and <i>peptide</a></i> are a little ambiguous and can overlap in meaning. <i>Protein</i> is generally used to refer to the complete biological molecule in a stable conformation</a>, whereas <i>peptide</i> is generally reserved for a short amino acid oligomers often lacking a stable three-dimensional structure. However, the boundary between the two is not well defined and usually lies near 20–30 residues.<sup id="cite_ref-Lodish2004_5-0" class="reference">&#91;5&#93;</a></sup> <i>Polypeptide</i> can refer to any single linear chain of amino acids, usually regardless of length, but often implies an absence of a defined conformation</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p>
<h3><span class="mw-headline" id="Interactions">Interactions</span></h3>
<p>Proteins</a> can interact with many types of molecules, including with other proteins</a>, with lipids</a>, with carboyhydrates</a>, and with DNA</a>.<sup id="cite_ref-6" class="reference">&#91;6&#93;</a></sup><sup id="cite_ref-7" class="reference">&#91;7&#93;</a></sup><sup id="cite_ref-8" class="reference">&#91;8&#93;</a></sup><sup id="cite_ref-9" class="reference">&#91;9&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p>
<h3><span class="mw-headline" id="Abundance_in_cells">Abundance in cells</span></h3>
<p>It has been estimated that average-sized bacteria</a> contain about 2 million proteins per cell (e.g. <i>E. coli</a></i> and <i>Staphylococcus aureus</a></i>). Smaller bacteria, such as <i>Mycoplasma</a></i> or <i>spirochetes</a></i> contain fewer molecules, on the order of 50,000 to 1 million. By contrast, eukaryotic</a> cells are larger and thus contain much more protein. For instance, yeast</a> cells have been estimated to contain about 50 million proteins and human</a> cells on the order of 1 to 3 billion.<sup id="cite_ref-10" class="reference">&#91;10&#93;</a></sup> The concentration of individual protein copies ranges from a few molecules per cell up to  20 million.<sup id="cite_ref-pmid22068332_11-0" class="reference">&#91;11&#93;</a></sup> Not all genes coding proteins are expressed in most cells and their number depends on, for example, cell type and external stimuli. For instance, of the 20,000 or so proteins encoded by the human genome, only 6,000 are detected in lymphoblastoid</a> cells.<sup id="cite_ref-12" class="reference">&#91;12&#93;</a></sup> Moreover, the number of proteins the genome</a> encodes correlates well with the organism complexity. Eukaryotes, bacteria, archaea</a> and viruses</a> have on average 15145, 3200, 2358 and 42 proteins respectively coded in their genomes.<sup id="cite_ref-Kozlowski2016_13-0" class="reference">&#91;13&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p>
<h2><span class="mw-headline" id="Synthesis">Synthesis</span></h2>
<h3><span class="mw-headline" id="Biosynthesis">Biosynthesis</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Ribosome_mRNA_translation_en.svg/220px-Ribosome_mRNA_translation_en.svg.png" width="220" height="155" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Ribosome_mRNA_translation_en.svg/330px-Ribosome_mRNA_translation_en.svg.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Ribosome_mRNA_translation_en.svg/440px-Ribosome_mRNA_translation_en.svg.png 2x" data-file-width="651" data-file-height="459" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>4</h1></center></div>A ribosome produces a protein using mRNA as template</div></div></div>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/Genetic_code.svg/220px-Genetic_code.svg.png" width="220" height="83" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/Genetic_code.svg/330px-Genetic_code.svg.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/Genetic_code.svg/440px-Genetic_code.svg.png 2x" data-file-width="580" data-file-height="220" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>5</h1></center></div>The DNA</a> sequence of a gene encodes</a> the amino acid</a> sequence of a protein</div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main article: Protein biosynthesis</a></div>
<p>Proteins are assembled from amino acids using information encoded in genes</a>. Each protein has its own unique amino acid sequence that is specified by the nucleotide</a> sequence of the gene encoding this protein. The genetic code</a> is a set of three-nucleotide sets called codons</a> and each three-nucleotide combination designates an amino acid, for example AUG (adenine</a>-uracil</a>-guanine</a>) is the code for methionine</a>. Because DNA</a> contains four nucleotides, the total number of possible codons is 64; hence, there is some redundancy in the genetic code, with some amino acids specified by more than one codon.<sup id="cite_ref-vanHolde1996_14-0" class="reference">&#91;14&#93;</a></sup> Genes encoded in DNA are first transcribed</a> into pre-messenger RNA</a> (mRNA) by proteins such as RNA polymerase</a>. Most organisms then process the pre-mRNA (also known as a <i>primary transcript</i>) using various forms of Post-transcriptional modification</a> to form the mature mRNA, which is then used as a template for protein synthesis by the ribosome</a>. In prokaryotes</a> the mRNA may either be used as soon as it is produced, or be bound by a ribosome after having moved away from the nucleoid</a>. In contrast, eukaryotes</a> make mRNA in the cell nucleus</a> and then translocate</a> it across the nuclear membrane</a> into the cytoplasm</a>, where protein synthesis</a> then takes place. The rate of protein synthesis is higher in prokaryotes than eukaryotes and can reach up to 20 amino acids per second.<sup id="cite_ref-Pain2000_15-0" class="reference">&#91;15&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p><p>The process of synthesizing a protein from an mRNA template is known as translation</a>. The mRNA is loaded onto the ribosome and is read three nucleotides at a time by matching each codon to its base pairing</a> anticodon</a> located on a transfer RNA</a> molecule, which carries the amino acid corresponding to the codon it recognizes. The enzyme aminoacyl tRNA synthetase</a> "charges" the tRNA molecules with the correct amino acids. The growing polypeptide is often termed the <i>nascent chain</i>. Proteins are always biosynthesized from N-terminus</a> to C-terminus</a>.<sup id="cite_ref-vanHolde1996_14-1" class="reference">&#91;14&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p><p>The size of a synthesized protein can be measured by the number of amino acids it contains and by its total molecular mass</a>, which is normally reported in units of <i>daltons</i> (synonymous with atomic mass units</a>), or the derivative unit kilodalton (kDa). The average size of a protein increases from Archaea to Bacteria to Eukaryote (283, 311, 438 residues and 31, 34, 49 kDa respecitvely) due to a bigger number of protein domains</a> constituting proteins in higher organisms.<sup id="cite_ref-Kozlowski2016_13-1" class="reference">&#91;13&#93;</a></sup> For instance, yeast</a> proteins are on average 466 amino acids long and 53 kDa in mass.<sup id="cite_ref-Lodish2004_5-1" class="reference">&#91;5&#93;</a></sup> The largest known proteins are the titins</a>, a component of the muscle</a> sarcomere</a>, with a molecular mass of almost 3,000 kDa and a total length of almost 27,000 amino acids.<sup id="cite_ref-Fulton1991_16-0" class="reference">&#91;16&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p>
<h3><span class="mw-headline" id="Chemical_synthesis">Chemical synthesis</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Peptide synthesis</a></div>
<p>Short proteins can also be synthesized chemically by a family of methods known as peptide synthesis</a>, which rely on organic synthesis</a> techniques such as chemical ligation</a> to produce peptides in high yield.<sup id="cite_ref-Bruckdorfer2004_17-0" class="reference">&#91;17&#93;</a></sup> Chemical synthesis allows for the introduction of non-natural amino acids into polypeptide chains, such as attachment of fluorescent</a> probes to amino acid side chains.<sup id="cite_ref-Schwarzer2005_18-0" class="reference">&#91;18&#93;</a></sup> These methods are useful in laboratory biochemistry</a> and cell biology</a>, though generally not for commercial applications. Chemical synthesis is inefficient for polypeptides longer than about 300 amino acids, and the synthesized proteins may not readily assume their native tertiary structure</a>. Most chemical synthesis methods proceed from C-terminus to N-terminus, opposite the biological reaction.<sup id="cite_ref-Kent2009_19-0" class="reference">&#91;19&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p>
<h2><span class="mw-headline" id="Structure">Structure</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Chaperonin_1AON.png/300px-Chaperonin_1AON.png" width="300" height="130" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Chaperonin_1AON.png/450px-Chaperonin_1AON.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/6/6c/Chaperonin_1AON.png/600px-Chaperonin_1AON.png 2x" data-file-width="1956" data-file-height="850" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>6</h1></center></div>The crystal structure of the chaperonin, a huge protein complex. A single protein subunit is highlighted. Chaperonins assist protein folding.</div></div></div>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Proteinviews-1tim.png/300px-Proteinviews-1tim.png" width="300" height="120" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Proteinviews-1tim.png/450px-Proteinviews-1tim.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Proteinviews-1tim.png/600px-Proteinviews-1tim.png 2x" data-file-width="750" data-file-height="300" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>7</h1></center></div>Three possible representations of the three-dimensional structure of the protein triose phosphate isomerase</a>. <b>Left</b>: All-atom representation colored by atom type. <b>Middle:</b>  Simplified representation illustrating the backbone conformation, colored by secondary structure. <b>Right</b>: Solvent-accessible surface representation colored by residue type (acidic residues red, basic residues blue, polar residues green, nonpolar residues white).</div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main article: Protein structure</a></div>
<div role="note" class="hatnote navigation-not-searchable">Further information: Protein structure prediction</a></div>
<p>Most&#160;proteins fold</a> into unique 3-dimensional structures. The shape into which a protein naturally folds is known as its native conformation</a>.<sup id="cite_ref-20" class="reference">&#91;20&#93;</a></sup> Although many proteins can fold unassisted, simply through the chemical properties of their amino acids, others require the aid of molecular chaperones</a> to fold into their native states.<sup id="cite_ref-21" class="reference">&#91;21&#93;</a></sup> Biochemists often refer to four distinct aspects of a protein's structure:<sup id="cite_ref-22" class="reference">&#91;22&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
<ul><li><i>Primary structure</a></i>: the amino acid sequence</a>. A protein is a polyamide</a>.</li>
<li><i>Secondary structure</a></i>: regularly repeating local structures stabilized by hydrogen bonds</a>. The most common examples are the α-helix</a>, β-sheet</a> and turns</a>. Because secondary structures are local, many regions of different secondary structure can be present in the same protein molecule.</li>
<li><i>Tertiary structure</a></i>: the overall shape of a single protein molecule; the spatial relationship of the secondary structures to one another. Tertiary structure is generally stabilized by nonlocal interactions, most commonly the formation of a hydrophobic core</a>, but also through salt bridges</a>, hydrogen bonds, disulfide bonds</a>, and even posttranslational modifications</a>. The term "tertiary structure" is often used as synonymous with the term <i>fold</i>. The tertiary structure is what controls the basic function of the protein.</li>
<li><i>Quaternary structure</a></i>: the structure formed by several protein molecules (polypeptide chains), usually called <i>protein subunits</a></i> in this context, which function as a single protein complex</a>.</li></ul>
<p>Proteins are not entirely rigid molecules. In addition to these levels of structure, proteins may shift between several related structures while they perform their functions. In the context of these functional rearrangements, these tertiary or quaternary structures are usually referred to as "conformations</a>", and transitions between them are called <i>conformational changes.</i> Such changes are often induced by the binding of a substrate</a> molecule to an enzyme's active site</a>, or the physical region of the protein that participates in chemical catalysis. In solution proteins also undergo variation in structure through thermal vibration and the collision with other molecules.<sup id="cite_ref-23" class="reference">&#91;23&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/Protein_composite.png/300px-Protein_composite.png" width="300" height="78" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/Protein_composite.png/450px-Protein_composite.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/Protein_composite.png/600px-Protein_composite.png 2x" data-file-width="1600" data-file-height="416" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>8</h1></center></div>Molecular surface of several proteins showing their comparative sizes. From left to right are: immunoglobulin G</a> (IgG, an antibody</a>), hemoglobin</a>, insulin</a> (a hormone), adenylate kinase</a> (an enzyme), and glutamine synthetase</a> (an enzyme).</div></div></div>
<p>Proteins can be informally divided into three main classes, which correlate with typical tertiary structures: globular proteins</a>, fibrous proteins</a>, and membrane proteins</a>. Almost all globular proteins are soluble</a> and many are enzymes. Fibrous proteins are often structural, such as collagen</a>, the major component of connective tissue, or keratin</a>, the protein component of hair and nails. Membrane proteins often serve as receptors</a> or provide channels for polar or charged molecules to pass through the cell membrane</a>.<sup id="cite_ref-24" class="reference">&#91;24&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p><p>A special case of intramolecular hydrogen bonds within proteins, poorly shielded from water attack and hence promoting their own dehydration</a>, are called dehydrons</a>.<sup id="cite_ref-Fernandez2003_25-0" class="reference">&#91;25&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
<h3><span class="mw-headline" id="Protein_domains">Protein domains</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Protein domain</a></div>
<p>Many proteins are composed of several protein domains</a>, i.e. segments of a protein that fold into distinct structural units. Domains usually also have specific functions, such as enzymatic</a> activities (e.g. kinase</a>) or they serve as binding modules (e.g. the SH3 domain</a> binds to proline-rich sequences in other proteins).
<span style="border: 3px solid blue; color:blue;font-weight: bold">20</span></p>
<h3><span class="mw-headline" id="Sequence_motif">Sequence motif</span></h3>
<p>Short amino acid sequences within proteins often act as recognition sites for other proteins.<sup id="cite_ref-26" class="reference">&#91;26&#93;</a></sup> For instance, SH3 domains</a> typically bind to short PxxP motifs (i.e. 2 prolines</a> [P], separated by 2 unspecified amino acids</a> [x], although the surrounding amino acids may determine the exact binding specificity). A large number of such motifs has been collected in the Eukaryotic Linear Motif</a> (ELM) database.
<span style="border: 3px solid blue; color:blue;font-weight: bold">21</span></p>
<h2><span class="mw-headline" id="Cellular_functions">Cellular functions</span></h2>
<p>Proteins are the chief actors within the cell, said to be carrying out the duties specified by the information encoded in genes.<sup id="cite_ref-Lodish2004_5-2" class="reference">&#91;5&#93;</a></sup> With the exception of certain types of RNA</a>, most other biological molecules are relatively inert elements upon which proteins act. Proteins make up half the dry weight of an <i>Escherichia coli</a></i> cell, whereas other macromolecules such as DNA and RNA make up only 3% and 20%, respectively.<sup id="cite_ref-Voet_27-0" class="reference">&#91;27&#93;</a></sup> The set of proteins expressed in a particular cell or cell type is known as its proteome</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">22</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png/220px-Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png" width="220" height="145" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png/330px-Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png/440px-Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png 2x" data-file-width="1760" data-file-height="1160" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>9</h1></center></div>The enzyme hexokinase</a> is shown as a conventional ball-and-stick molecular model. To scale in the top right-hand corner are two of its substrates, ATP</a> and glucose</a>.</div></div></div>
<p>The chief characteristic of proteins that also allows their diverse set of functions is their ability to bind other molecules specifically and tightly. The region of the protein responsible for binding another molecule is known as the binding site</a> and is often a depression or "pocket" on the molecular surface. This binding ability is mediated by the tertiary structure of the protein, which defines the binding site pocket, and by the chemical properties of the surrounding amino acids' side chains. Protein binding can be extraordinarily tight and specific; for example, the ribonuclease inhibitor</a> protein binds to human angiogenin</a> with a sub-femtomolar dissociation constant</a> (&lt;10<sup>−15</sup> M) but does not bind at all to its amphibian homolog onconase</a> (&gt;1 M). Extremely minor chemical changes such as the addition of a single methyl group to a binding partner can sometimes suffice to nearly eliminate binding; for example, the aminoacyl tRNA synthetase</a> specific to the amino acid valine</a> discriminates against the very similar side chain of the amino acid isoleucine</a>.<sup id="cite_ref-Sankaranarayanan2001_28-0" class="reference">&#91;28&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">23</span></p><p>Proteins can bind to other proteins as well as to small-molecule</a> substrates. When proteins bind specifically to other copies of the same molecule, they can oligomerize</a> to form fibrils; this process occurs often in structural proteins that consist of globular monomers that self-associate to form rigid fibers. Protein–protein interactions</a> also regulate enzymatic activity, control progression through the cell cycle</a>, and allow the assembly of large protein complexes</a> that carry out many closely related reactions with a common biological function. Proteins can also bind to, or even be integrated into, cell membranes. The ability of binding partners to induce conformational changes in proteins allows the construction of enormously complex signaling</a> networks.<sup id="cite_ref-29" class="reference">&#91;29&#93;</a></sup>
As interactions between proteins are reversible, and depend heavily on the availability of different groups of partner proteins to form aggregates that are capable to carry out discrete sets of function, study of the interactions between specific proteins is a key to understand important aspects of cellular function, and ultimately the properties that distinguish particular cell types.<sup id="cite_ref-Copland2009_30-0" class="reference">&#91;30&#93;</a></sup><sup id="cite_ref-Samarin2009_31-0" class="reference">&#91;31&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">24</span></p>
<h3><span class="mw-headline" id="Enzymes">Enzymes</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Enzyme</a></div>
<p>The best-known role of proteins in the cell is as enzymes</a>, which catalyse</a> chemical reactions. Enzymes are usually highly specific and accelerate only one or a few chemical reactions. Enzymes carry out most of the reactions involved in metabolism</a>, as well as manipulating DNA in processes such as DNA replication</a>, DNA repair</a>, and transcription</a>. Some enzymes act on other proteins to add or remove chemical groups in a process known as posttranslational modification. About 4,000 reactions are known to be catalysed by enzymes.<sup id="cite_ref-EXPASY2000_32-0" class="reference">&#91;32&#93;</a></sup> The rate acceleration conferred by enzymatic catalysis is often enormous—as much as 10<sup>17</sup>-fold increase in rate over the uncatalysed reaction in the case of orotate decarboxylase</a> (78 million years without the enzyme, 18 milliseconds with the enzyme).<sup id="cite_ref-Radzicka1995_33-0" class="reference">&#91;33&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">25</span></p><p>The molecules bound and acted upon by enzymes are called substrates</a>. Although enzymes can consist of hundreds of amino acids, it is usually only a small fraction of the residues that come in contact with the substrate, and an even smaller fraction—three to four residues on average—that are directly involved in catalysis.<sup id="cite_ref-urlEBI_34-0" class="reference">&#91;34&#93;</a></sup> The region of the enzyme that binds the substrate and contains the catalytic residues is known as the active site</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">26</span></p><p>Dirigent proteins</a> are members of a class of proteins that dictate the stereochemistry</a> of a compound synthesized by other enzymes.<sup id="cite_ref-Pickel_2013_35-0" class="reference">&#91;35&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">27</span></p>
<h3><span class="mw-headline" id="Cell_signaling_and_ligand_binding">Cell signaling and ligand binding</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:172px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a2/Mouse_cholera_antibody.png/170px-Mouse_cholera_antibody.png" width="170" height="227" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a2/Mouse_cholera_antibody.png/255px-Mouse_cholera_antibody.png 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/a/a2/Mouse_cholera_antibody.png/340px-Mouse_cholera_antibody.png 2x" data-file-width="900" data-file-height="1200" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>10</h1></center></div>Ribbon diagram</a> of a mouse antibody against cholera</a> that binds a carbohydrate</a> antigen</div></div></div>
<p>Many proteins are involved in the process of cell signaling</a> and signal transduction</a>. Some proteins, such as insulin</a>, are extracellular proteins that transmit a signal from the cell in which they were synthesized to other cells in distant tissues</a>. Others are membrane proteins</a> that act as receptors</a> whose main function is to bind a signaling molecule and induce a biochemical response in the cell. Many receptors have a binding site exposed on the cell surface and an effector domain within the cell, which may have enzymatic activity or may undergo a conformational change</a> detected by other proteins within the cell.<sup id="cite_ref-36" class="reference">&#91;36&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">28</span></p><p>Antibodies</a> are protein components of an adaptive immune system</a> whose main function is to bind antigens</a>, or foreign substances in the body, and target them for destruction. Antibodies can be secreted</a> into the extracellular environment or anchored in the membranes of specialized B cells</a> known as plasma cells</a>. Whereas enzymes are limited in their binding affinity for their substrates by the necessity of conducting their reaction, antibodies have no such constraints. An antibody's binding affinity to its target is extraordinarily high.<sup id="cite_ref-37" class="reference">&#91;37&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">29</span></p><p>Many ligand transport proteins bind particular small biomolecules</a> and transport them to other locations in the body of a multicellular organism. These proteins must have a high binding affinity when their ligand</a> is present in high concentrations, but must also release the ligand when it is present at low concentrations in the target tissues. The canonical example of a ligand-binding protein is haemoglobin</a>, which transports oxygen</a> from the lungs</a> to other organs and tissues in all vertebrates</a> and has close homologs</a> in every biological kingdom</a>.<sup id="cite_ref-38" class="reference">&#91;38&#93;</a></sup> Lectins</a> are sugar-binding proteins which are highly specific for their sugar moieties. Lectins</a> typically play a role in biological recognition</a> phenomena involving cells and proteins.<sup id="cite_ref-Rudiger2000_39-0" class="reference">&#91;39&#93;</a></sup> Receptors</a> and hormones</a> are highly specific binding proteins.
<span style="border: 3px solid blue; color:blue;font-weight: bold">30</span></p><p>Transmembrane proteins</a> can also serve as ligand transport proteins that alter the permeability</a> of the cell membrane to small molecules</a> and ions. The membrane alone has a hydrophobic</a> core through which polar</a> or charged molecules cannot diffuse</a>. Membrane proteins contain internal channels that allow such molecules to enter and exit the cell. Many ion channel</a> proteins are specialized to select for only a particular ion; for example, potassium</a> and sodium</a> channels often discriminate for only one of the two ions.<sup id="cite_ref-40" class="reference">&#91;40&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">31</span></p>
<h3><span class="mw-headline" id="Structural_proteins">Structural proteins</span></h3>
<p>Structural proteins confer stiffness and rigidity to otherwise-fluid biological components. Most structural proteins are fibrous proteins</a>; for example, collagen</a> and elastin</a> are critical components of connective tissue</a> such as cartilage</a>, and keratin</a> is found in hard or filamentous structures such as hair</a>, nails</a>, feathers</a>, hooves</a>, and some animal shells</a>.<sup id="cite_ref-41" class="reference">&#91;41&#93;</a></sup> Some globular proteins</a> can also play structural functions, for example, actin</a> and tubulin</a> are globular and soluble as monomers, but polymerize</a> to form long, stiff fibers that make up the cytoskeleton</a>, which allows the cell to maintain its shape and size.
<span style="border: 3px solid blue; color:blue;font-weight: bold">32</span></p><p>Other proteins that serve structural functions are motor proteins</a> such as myosin</a>, kinesin</a>, and dynein</a>, which are capable of generating mechanical forces. These proteins are crucial for cellular motility</a> of single celled organisms and the sperm</a> of many multicellular organisms which reproduce sexually</a>. They also generate the forces exerted by contracting muscles</a><sup id="cite_ref-42" class="reference">&#91;42&#93;</a></sup> and play essential roles in intracellular transport.
<span style="border: 3px solid blue; color:blue;font-weight: bold">33</span></p>
<h2><span class="mw-headline" id="Methods_of_study">Methods of study</span></h2>
<div role="note" class="hatnote navigation-not-searchable">Main article: Protein methods</a></div>
<p>The activities and structures of proteins may be examined <i>in vitro</a>,</i> <i>in vivo</a>, and in silico</a></i>. <i><b>In vitro</b></i> studies of purified proteins in controlled environments are useful for learning how a protein carries out its function: for example, enzyme kinetics</a> studies explore the chemical mechanism</a> of an enzyme's catalytic activity and its relative affinity for various possible substrate molecules. By contrast, <i><b>in vivo</b></i> experiments can provide information about the physiological role of a protein in the context of a cell</a> or even a whole organism</a>. <i><b>In silico</b></i> studies use computational methods to study proteins.
<span style="border: 3px solid blue; color:blue;font-weight: bold">34</span></p>
<h3><span class="mw-headline" id="Protein_purification">Protein purification</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Protein purification</a></div>
<p>To perform <i>in vitro</a></i> analysis, a protein must be purified away from other cellular components. This process usually begins with cell lysis</a>, in which a cell's membrane is disrupted and its internal contents released into a solution known as a crude lysate</a>. The resulting mixture can be purified using ultracentrifugation</a>, which fractionates the various cellular components into fractions containing soluble proteins; membrane lipids</a> and proteins; cellular organelles</a>, and nucleic acids</a>. Precipitation</a> by a method known as salting out</a> can concentrate the proteins from this lysate. Various types of chromatography</a> are then used to isolate the protein or proteins of interest based on properties such as molecular weight, net charge and binding affinity.<sup id="cite_ref-43" class="reference">&#91;43&#93;</a></sup> The level of purification can be monitored using various types of gel electrophoresis</a> if the desired protein's molecular weight and isoelectric point</a> are known, by spectroscopy</a> if the protein has distinguishable spectroscopic features, or by enzyme assays</a> if the protein has enzymatic activity. Additionally, proteins can be isolated according to their charge using electrofocusing</a>.<sup id="cite_ref-Hey2008_44-0" class="reference">&#91;44&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">35</span></p><p>For natural proteins, a series of purification steps may be necessary to obtain protein sufficiently pure for laboratory applications. To simplify this process, genetic engineering</a> is often used to add chemical features to proteins that make them easier to purify without affecting their structure or activity. Here, a "tag" consisting of a specific amino acid sequence, often a series of histidine</a> residues (a "His-tag</a>"), is attached to one terminus of the protein. As a result, when the lysate is passed over a chromatography column containing nickel</a>, the histidine residues ligate the nickel and attach to the column while the untagged components of the lysate pass unimpeded. A number of different tags have been developed to help researchers purify specific proteins from complex mixtures.<sup id="cite_ref-Terpe2003_45-0" class="reference">&#91;45&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">36</span></p>
<h3><span class="mw-headline" id="Cellular_localization">Cellular localization</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Localisations02eng.jpg/300px-Localisations02eng.jpg" width="300" height="395" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Localisations02eng.jpg/450px-Localisations02eng.jpg 1.5x, https://upload.wikimedia.org/wikipedia/commons/6/6e/Localisations02eng.jpg 2x" data-file-width="469" data-file-height="618" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>11</h1></center></div>Proteins in different cellular compartments</a> and structures tagged with green fluorescent protein</a> (here, white)</div></div></div>
<p>The study of proteins <i>in vivo</i> is often concerned with the synthesis and localization of the protein within the cell. Although many intracellular proteins are synthesized in the cytoplasm</a> and membrane-bound or secreted proteins in the endoplasmic reticulum</a>, the specifics of how proteins are targeted</a> to specific organelles or cellular structures is often unclear. A useful technique for assessing cellular localization uses genetic engineering to express in a cell a fusion protein</a> or chimera</a> consisting of the natural protein of interest linked to a "reporter</a>" such as green fluorescent protein</a> (GFP).<sup id="cite_ref-Stepanenko2008_46-0" class="reference">&#91;46&#93;</a></sup> The fused protein's position within the cell can be cleanly and efficiently visualized using microscopy</a>,<sup id="cite_ref-Yuste2005_47-0" class="reference">&#91;47&#93;</a></sup> as shown in the figure opposite.
<span style="border: 3px solid blue; color:blue;font-weight: bold">37</span></p><p>Other methods for elucidating the cellular location of proteins requires the use of known compartmental markers for regions such as the ER, the Golgi, lysosomes or vacuoles, mitochondria, chloroplasts, plasma membrane, etc. With the use of fluorescently tagged versions of these markers or of antibodies to known markers, it becomes much simpler to identify the localization of a protein of interest. For example, indirect immunofluorescence</a> will allow for fluorescence colocalization and demonstration of location. Fluorescent dyes are used to label cellular compartments for a similar purpose.<sup id="cite_ref-Margolin2000_48-0" class="reference">&#91;48&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">38</span></p><p>Other possibilities exist, as well. For example, immunohistochemistry</a> usually utilizes an antibody to one or more proteins of interest that are conjugated to enzymes yielding either luminescent or chromogenic signals that can be compared between samples, allowing for localization information. Another applicable technique is cofractionation in sucrose (or other material) gradients using isopycnic centrifugation</a>.<sup id="cite_ref-Walker2000_49-0" class="reference">&#91;49&#93;</a></sup> While this technique does not prove colocalization of a compartment of known density and the protein of interest, it does increase the likelihood, and is more amenable to large-scale studies.
<span style="border: 3px solid blue; color:blue;font-weight: bold">39</span></p><p>Finally, the gold-standard method of cellular localization is immunoelectron microscopy</a>. This technique also uses an antibody to the protein of interest, along with classical electron microscopy techniques. The sample is prepared for normal electron microscopic examination, and then treated with an antibody to the protein of interest that is conjugated to an extremely electro-dense material, usually gold. This allows for the localization of both ultrastructural details as well as the protein of interest.<sup id="cite_ref-Mayhew2008_50-0" class="reference">&#91;50&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">40</span></p><p>Through another genetic engineering application known as site-directed mutagenesis</a>, researchers can alter the protein sequence and hence its structure, cellular localization, and susceptibility to regulation. This technique even allows the incorporation of unnatural amino acids into proteins, using modified tRNAs,<sup id="cite_ref-Hohsaka2002_51-0" class="reference">&#91;51&#93;</a></sup> and may allow the rational design</a> of new proteins with novel properties.<sup id="cite_ref-Cedrone2000_52-0" class="reference">&#91;52&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">41</span></p>
<h3><span class="mw-headline" id="Proteomics">Proteomics</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Proteomics</a></div>
<p>The total complement of proteins present at a time in a cell or cell type is known as its proteome</a>, and the study of such large-scale data sets defines the field of proteomics</a>, named by analogy to the related field of genomics</a>. Key experimental techniques in proteomics include 2D electrophoresis</a>,<sup id="cite_ref-Gorg2008_53-0" class="reference">&#91;53&#93;</a></sup> which allows the separation of a large number of proteins, mass spectrometry</a>,<sup id="cite_ref-Conrotto2008_54-0" class="reference">&#91;54&#93;</a></sup> which allows rapid high-throughput identification of proteins and sequencing of peptides (most often after in-gel digestion</a>), protein microarrays</a>, which allow the detection of the relative levels of a large number of proteins present in a cell, and two-hybrid screening</a>, which allows the systematic exploration of protein–protein interactions</a>.<sup id="cite_ref-Koegl2007_55-0" class="reference">&#91;55&#93;</a></sup> The total complement of biologically possible such interactions is known as the interactome</a>.<sup id="cite_ref-Plewczynski2009_56-0" class="reference">&#91;56&#93;</a></sup> A systematic attempt to determine the structures of proteins representing every possible fold is known as structural genomics</a>.<sup id="cite_ref-Zhang2003_57-0" class="reference">&#91;57&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">42</span></p>
<h3><span class="mw-headline" id="Bioinformatics">Bioinformatics</span></h3>
<div role="note" class="hatnote navigation-not-searchable">Main article: Bioinformatics</a></div>
<p>A vast array of computational methods have been developed to analyze the structure, function, and evolution of proteins.
<span style="border: 3px solid blue; color:blue;font-weight: bold">43</span></p><p>The development of such tools has been driven by the large amount of genomic and proteomic data available for a variety of organisms, including the human genome</a>. It is simply impossible to study all proteins experimentally, hence only a few are subjected to laboratory experiments while computational tools are used to extrapolate to similar proteins. Such homologous</a> proteins can be efficiently identified in distantly related organisms by sequence alignment</a>. Genome and gene sequences can be searched by a variety of tools for certain properties. Sequence profiling tools</a> can find restriction enzyme</a> sites, open reading frames</a> in nucleotide</a> sequences, and predict secondary structures</a>. Phylogenetic trees</a> can be constructed and evolutionary</a> hypotheses developed using special software like ClustalW</a> regarding the ancestry of modern organisms and the genes they express. The field of bioinformatics</a> is now indispensable for the analysis of genes and proteins.
<span style="border: 3px solid blue; color:blue;font-weight: bold">44</span></p>
<h3><span class="mw-headline" id="Structure_determination">Structure determination</span></h3>
<p>Discovering the tertiary structure of a protein, or the quaternary structure of its complexes, can provide important clues about how the protein performs its function. Common experimental methods of structure determination include X-ray crystallography</a> and NMR spectroscopy</a>, both of which can produce information at atomic</a> resolution. However, NMR experiments are able to provide information from which a subset of distances between pairs of atoms can be estimated, and the final possible conformations for a protein are determined by solving a distance geometry</a> problem. Dual polarisation interferometry</a> is a quantitative analytical method for measuring the overall protein conformation</a> and conformational changes</a> due to interactions or other stimulus. Circular dichroism</a> is another laboratory technique for determining internal β-sheet / α-helical composition of proteins. Cryoelectron microscopy</a> is used to produce lower-resolution structural information about very large protein complexes, including assembled viruses</a>;<sup id="cite_ref-58" class="reference">&#91;58&#93;</a></sup> a variant known as electron crystallography</a> can also produce high-resolution information in some cases, especially for two-dimensional crystals of membrane proteins.<sup id="cite_ref-Gonen2005_59-0" class="reference">&#91;59&#93;</a></sup> Solved structures are usually deposited in the Protein Data Bank</a> (PDB), a freely available resource from which structural data about thousands of proteins can be obtained in the form of Cartesian coordinates</a> for each atom in the protein.<sup id="cite_ref-Standley2008_60-0" class="reference">&#91;60&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">45</span></p><p>Many more gene sequences are known than protein structures. Further, the set of solved structures is biased toward proteins that can be easily subjected to the conditions required in X-ray crystallography</a>, one of the major structure determination methods. In particular, globular proteins are comparatively easy to crystallize</a> in preparation for X-ray crystallography. Membrane proteins, by contrast, are difficult to crystallize and are underrepresented in the PDB.<sup id="cite_ref-Walian2004_61-0" class="reference">&#91;61&#93;</a></sup> Structural genomics</a> initiatives have attempted to remedy these deficiencies by systematically solving representative structures of major fold classes. Protein structure prediction</a> methods attempt to provide a means of generating a plausible structure for proteins whose structures have not been experimentally determined.<sup id="cite_ref-Sleator2012_62-0" class="reference">&#91;62&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">46</span></p>
<h3><span class="mw-headline" id="Structure_prediction_and_simulation">Structure prediction and simulation</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:352px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/26/225_Peptide_Bond-01.jpg/350px-225_Peptide_Bond-01.jpg" width="350" height="297" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/thumb/2/26/225_Peptide_Bond-01.jpg/525px-225_Peptide_Bond-01.jpg 1.5x, https://upload.wikimedia.org/wikipedia/commons/thumb/2/26/225_Peptide_Bond-01.jpg/700px-225_Peptide_Bond-01.jpg 2x" data-file-width="2219" data-file-height="1886" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>12</h1></center></div>Constituent amino-acids can be analyzed to predict secondary, tertiary and quaternary protein structure, in this case hemoglobin containing heme</a> units</div></div></div>
<div role="note" class="hatnote navigation-not-searchable">Main articles: Protein structure prediction</a> and List of protein structure prediction software</a></div>
<p>Complementary to the field of structural genomics, <i>protein structure prediction</i> develops efficient mathematical models</a> of proteins to computationally predict the molecular formations in theory, instead of detecting structures with laboratory observation.<sup id="cite_ref-Zhang2008_63-0" class="reference">&#91;63&#93;</a></sup> The most successful type of structure prediction, known as homology modeling</a>, relies on the existence of a "template" structure with sequence similarity to the protein being modeled; structural genomics' goal is to provide sufficient representation in solved structures to model most of those that remain.<sup id="cite_ref-Xiang2006_64-0" class="reference">&#91;64&#93;</a></sup> Although producing accurate models remains a challenge when only distantly related template structures are available, it has been suggested that sequence alignment</a> is the bottleneck in this process, as quite accurate models can be produced if a "perfect" sequence alignment is known.<sup id="cite_ref-Zhang2005_65-0" class="reference">&#91;65&#93;</a></sup> Many structure prediction methods have served to inform the emerging field of protein engineering</a>, in which novel protein folds have already been designed.<sup id="cite_ref-Kuhlman2003_66-0" class="reference">&#91;66&#93;</a></sup> A more complex computational problem is the prediction of intermolecular interactions, such as in molecular docking</a> and protein–protein interaction prediction</a>.<sup id="cite_ref-Ritchie2008_67-0" class="reference">&#91;67&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">47</span></p><p>Mathematical models to simulate dynamic processes of protein folding</a> and binding involve molecular mechanics</a>, in particular, molecular dynamics</a>.  Monte Carlo</a> techniques facilitate the computations, which exploit advances in parallel and distributed computing</a> (for example, the Folding@home</a> project<sup id="cite_ref-Scheraga2007_68-0" class="reference">&#91;68&#93;</a></sup> which performs molecular modeling</a> on GPUs</a>). <i>In silico</a></i> simulations discovered the folding of small α-helical protein domains</a> such as the villin</a> headpiece<sup id="cite_ref-Zagrovic2002_69-0" class="reference">&#91;69&#93;</a></sup> and the HIV</a> accessory protein.<sup id="cite_ref-Herges2005_70-0" class="reference">&#91;70&#93;</a></sup>  Hybrid methods combining standard molecular dynamics with quantum mechanical</a> mathematics explored the electronic states of rhodopsins</a>.<sup id="cite_ref-Hoffman2006_71-0" class="reference">&#91;71&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">48</span></p>
<h4><span class="mw-headline" id="Protein_disorder_and_unstructure_prediction">Protein disorder and unstructure prediction</span></h4>
<p>Many proteins (in Eucaryota ~33%) contain large unstructured but biologically functional segments and can be classified as intrinsically disordered proteins</a>.<sup id="cite_ref-72" class="reference">&#91;72&#93;</a></sup> Predicting and analysing protein disorder is, therefore, an important part of protein structure characterisation.<sup id="cite_ref-TompaFersht2009_73-0" class="reference">&#91;73&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">49</span></p>
<h2><span class="mw-headline" id="Nutrition">Nutrition</span></h2>
<div role="note" class="hatnote navigation-not-searchable">Further information: Protein (nutrient)</a></div>
<p>Most microorganisms</a> and plants can biosynthesize all 20 standard amino acids</a>, while animals (including humans) must obtain some of the amino acids from the diet</a>.<sup id="cite_ref-Voet_27-1" class="reference">&#91;27&#93;</a></sup> The amino acids that an organism cannot synthesize on its own are referred to as essential amino acids</a>. Key enzymes that synthesize certain amino acids are not present in animals&#160;— such as aspartokinase</a>, which catalyses the first step in the synthesis of lysine</a>, methionine</a>, and threonine</a> from aspartate</a>. If amino acids are present in the environment, microorganisms can conserve energy by taking up the amino acids from their surroundings and downregulating</a> their biosynthetic pathways.
<span style="border: 3px solid blue; color:blue;font-weight: bold">50</span></p><p>In animals, amino acids are obtained through the consumption of foods containing protein. Ingested proteins are then broken down into amino acids through digestion</a>, which typically involves denaturation</a> of the protein through exposure to acid</a> and hydrolysis</a> by enzymes called proteases</a>. Some ingested amino acids are used for protein biosynthesis, while others are converted to glucose</a> through gluconeogenesis</a>, or fed into the citric acid cycle</a>. This use of protein as a fuel is particularly important under starvation</a> conditions as it allows the body's own proteins to be used to support life, particularly those found in muscle</a>.<sup id="cite_ref-BrosnanJ_74-0" class="reference">&#91;74&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">51</span></p><p>In animals such as dogs and cats, protein maintains the health and quality of the skin by promoting hair follicle growth and keratinization, and thus reducing the likelihood of skin problems producing malodours.<sup id="cite_ref-Watson_1998_75-0" class="reference">&#91;75&#93;</a></sup> Poor-quality proteins also have a role regarding gastrointestinal health, increasing the potential for flatulence and odorous compounds in dogs because when proteins reach the colon in an undigested state, they are fermented producing hydrogen sulfide gas, indole, and skatole.<sup id="cite_ref-Case_2010_76-0" class="reference">&#91;76&#93;</a></sup> Dogs and cats digest animal proteins better than those from plants but products of low-quality animal origin are poorly digested, including skin, feathers, and connective tissue.<sup id="cite_ref-Case_2010_76-1" class="reference">&#91;76&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">52</span></p>
<h2><span class="mw-headline" id="History_and_etymology">History and etymology</span></h2>
<div role="note" class="hatnote navigation-not-searchable">Further information: History of molecular biology</a></div>
<p>Proteins were recognized as a distinct class of biological molecules in the eighteenth century by Antoine Fourcroy</a> and others, distinguished by the molecules' ability to coagulate</a> or flocculate</a> under treatments with heat or acid.<sup id="cite_ref-77" class="reference">&#91;77&#93;</a></sup> Noted examples at the time included albumin from egg whites</a>, blood serum albumin</a>, fibrin</a>, and wheat gluten</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">53</span></p><p>Proteins were first described by the Dutch</a> chemist Gerardus Johannes Mulder</a> and named by the Swedish chemist Jöns Jacob Berzelius</a> in 1838.<sup id="cite_ref-Mulder1938_78-0" class="reference">&#91;78&#93;</a></sup><sup id="cite_ref-Hartley_79-0" class="reference">&#91;79&#93;</a></sup> Mulder carried out elemental analysis</a> of common proteins and found that nearly all proteins had the same empirical formula</a>, C<sub>400</sub>H<sub>620</sub>N<sub>100</sub>O<sub>120</sub>P<sub>1</sub>S<sub>1</sub>.<sup id="cite_ref-Perrett2007_80-0" class="reference">&#91;80&#93;</a></sup> He came to the erroneous conclusion that they might be composed of a single type of (very large) molecule. The term "protein" to describe these molecules was proposed by Mulder's associate Berzelius; protein is derived from the Greek</a> word πώτειος (<i>proteios</i>), meaning "primary",<sup id="cite_ref-81" class="reference">&#91;81&#93;</a></sup> "in the lead", or "standing in front",<sup id="cite_ref-Reynolds2003_82-0" class="reference">&#91;82&#93;</a></sup> + <i>-in</a></i>. Mulder went on to identify the products of protein degradation such as the amino acid</a> leucine</a> for which he found a (nearly correct) molecular weight of 131 Da</a>.<sup id="cite_ref-Perrett2007_80-1" class="reference">&#91;80&#93;</a></sup> Prior to "protein", other names were used, like "albumins" or "albuminous materials" (<i>Eiweisskörper</i>, in German).<sup id="cite_ref-83" class="reference">&#91;83&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">54</span></p><p>Early nutritional scientists such as the German Carl von Voit</a> believed that protein was the most important nutrient for maintaining the structure of the body, because it was generally believed that "flesh makes flesh."<sup id="cite_ref-Bischoff1860_84-0" class="reference">&#91;84&#93;</a></sup> Karl Heinrich Ritthausen</a> extended known protein forms with the identification of glutamic acid</a>. At the Connecticut Agricultural Experiment Station</a> a detailed review of the vegetable proteins was compiled by Thomas Burr Osborne</a>. Working with Lafayette Mendel</a> and applying Liebig's law of the minimum</a> in feeding laboratory rats</a>, the nutritionally essential amino acids</a> were established. The work was continued and communicated by William Cumming Rose</a>. The understanding of proteins as polypeptides</a> came through the work of Franz Hofmeister</a> and Hermann Emil Fischer</a> in 1902.<sup id="cite_ref-85" class="reference">&#91;85&#93;</a></sup><sup id="cite_ref-86" class="reference">&#91;86&#93;</a></sup> The central role of proteins as enzymes</a> in living organisms was not fully appreciated until 1926, when James B. Sumner</a> showed that the enzyme urease</a> was in fact a protein.<sup id="cite_ref-Sumner1926_87-0" class="reference">&#91;87&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">55</span></p><p>The difficulty in purifying proteins in large quantities made them very difficult for early protein biochemists to study. Hence, early studies focused on proteins that could be purified in large quantities, e.g., those of blood</a>, egg white</a>, various toxins</a>, and digestive/metabolic enzymes obtained from slaughterhouses</a>. In the 1950s, the Armour Hot Dog Co.</a> purified 1&#160;kg of pure bovine pancreatic ribonuclease A</a> and made it freely available to scientists; this gesture helped ribonuclease A become a major target for biochemical study for the following decades.<sup id="cite_ref-Perrett2007_80-2" class="reference">&#91;80&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">56</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:252px;"><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ec/KendrewMyoglobin.jpg/250px-KendrewMyoglobin.jpg" width="250" height="167" class="thumbimage" srcset="https://upload.wikimedia.org/wikipedia/commons/e/ec/KendrewMyoglobin.jpg 1.5x" data-file-width="340" data-file-height="227" /></a>  <div class="thumbcaption"><div class="magnify"></a></div><div style='border: 3px solid red'><center><h1 style='color:red'>13</h1></center></div>John Kendrew</a> with model of myoglobin in progress</div></div></div>
<p>Linus Pauling</a> is credited with the successful prediction of regular protein secondary structures</a> based on hydrogen bonding</a>, an idea first put forth by William Astbury</a> in 1933.<sup id="cite_ref-Pauling1951_88-0" class="reference">&#91;88&#93;</a></sup> Later work by Walter Kauzmann</a> on denaturation</a>,<sup id="cite_ref-Kauzmann1956_89-0" class="reference">&#91;89&#93;</a></sup><sup id="cite_ref-Kauzmann1959_90-0" class="reference">&#91;90&#93;</a></sup> based partly on previous studies by Kaj Linderstrøm-Lang</a>,<sup id="cite_ref-Kalman1955_91-0" class="reference">&#91;91&#93;</a></sup> contributed an understanding of protein folding</a> and structure mediated by hydrophobic interactions</a>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">57</span></p><p>The first protein to be sequenced</a> was insulin</a>, by Frederick Sanger</a>, in 1949. Sanger correctly determined the amino acid sequence of insulin</a>, thus conclusively demonstrating that proteins consisted of linear polymers of amino acids rather than branched chains, colloids</a>, or cyclols</a>.<sup id="cite_ref-Sanger1949_92-0" class="reference">&#91;92&#93;</a></sup> He won the Nobel Prize for this achievement in 1958.<sup id="cite_ref-Lecture_1958_93-0" class="reference">&#91;93&#93;</a></sup>
<span style="border: 3px solid blue; color:blue;font-weight: bold">58</span></p><p>The first protein structures</a> to be solved were hemoglobin</a> and myoglobin</a>, by Max Perutz</a> and Sir John Cowdery Kendrew</a>, respectively, in 1958.<sup id="cite_ref-Muirhead1963_94-0" class="reference">&#91;94&#93;</a></sup><sup id="cite_ref-Kendrew1958_95-0" class="reference">&#91;95&#93;</a></sup> As of  2017<sup class="plainlinks noexcerpt noprint asof-tag update" style="display:none;">&#91;update&#93;</a></sup>, the Protein Data Bank</a> has over 126,060 atomic-resolution structures of proteins.<sup id="cite_ref-urlRCSB_Protein_Data_Bank_96-0" class="reference">&#91;96&#93;</a></sup> In more recent times, cryo-electron microscopy</a> of large macromolecular assemblies</a><sup id="cite_ref-Zhou2008_97-0" class="reference">&#91;97&#93;</a></sup> and computational protein structure prediction</a> of small protein domains</a><sup id="cite_ref-Keskin2008_98-0" class="reference">&#91;98&#93;</a></sup> are two methods approaching atomic resolution.
<span style="border: 3px solid blue; color:blue;font-weight: bold">59</span></p>
<div style="clear:both;"></div>
