<!DOCTYPE html>

<html class="client-nojs" dir="ltr" lang="en">
<head>
<meta charset="utf-8"/>
<title>P-type ATPase - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"P-type_ATPase","wgTitle":"P-type ATPase","wgCurRevisionId":890607298,"wgRevisionId":890607298,"wgArticleId":14655680,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["EC 3.6.3","Integral membrane proteins","Transport proteins","Physiology"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"P-type_ATPase","wgRelevantArticleId":14655680,"wgRequestId":"XL9GPgpAIC0AAFPUbdAAAABB","wgCSPNonce":false,"wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsReferencePreviews":false,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgWMESchemaEditAttemptStepOversample":false,"wgPoweredByHHVM":true,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q11903647","wgCentralAuthMobileDomain":false,"wgEditSubmitButtonLabelPublish":true});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});RLPAGEMODULES=["ext.cite.ux-enhancements","site","mediawiki.page.startup","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];mw.loader.load(RLPAGEMODULES);});</script>
<link href="/w/load.php?lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta content="" name="ResourceLoaderDynamicStyles"/>
<link href="/w/load.php?lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<link href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<meta content="MediaWiki 1.34.0-wmf.1" name="generator"/>
<meta content="origin" name="referrer"/>
<meta content="origin-when-crossorigin" name="referrer"/>
<meta content="origin-when-cross-origin" name="referrer"/>
<meta content="https://upload.wikimedia.org/wikipedia/commons/e/e6/1wpg_opm.png" property="og:image"/>
<link href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/P-type_ATPase" rel="alternate"/>
<link href="/w/index.php?title=P-type_ATPase&amp;action=edit" rel="alternate" title="Edit this page" type="application/x-wiki"/>
<link href="/w/index.php?title=P-type_ATPase&amp;action=edit" rel="edit" title="Edit this page"/>
<link href="/static/apple-touch/wikipedia.png" rel="apple-touch-icon"/>
<link href="/static/favicon/wikipedia.ico" rel="shortcut icon"/>
<link href="/w/opensearch_desc.php" rel="search" title="Wikipedia (en)" type="application/opensearchdescription+xml"/>
<link href="//en.wikipedia.org/w/api.php?action=rsd" rel="EditURI" type="application/rsd+xml"/>
<link href="//creativecommons.org/licenses/by-sa/3.0/" rel="license"/>
<link href="https://en.wikipedia.org/wiki/P-type_ATPase" rel="canonical"/>
<link href="//login.wikimedia.org" rel="dns-prefetch"/>
<link href="//meta.wikimedia.org" rel="dns-prefetch"/>
<!--[if lt IE 9]><script src="/w/load.php?lang=qqx&amp;modules=html5shiv&amp;only=scripts&amp;skin=fallback&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-P-type_ATPase rootpage-P-type_ATPase skin-vector action-view">
<div class="noprint" id="mw-page-base"></div>
<div class="noprint" id="mw-head-base"></div>
<div class="mw-body" id="content" role="main">

<div class="mw-body-content" id="siteNotice"><!-- CentralNotice --></div>
<div class="mw-indicators mw-body-content">
</div>
<h1 class="firstHeading" id="firstHeading" lang="en">P-type ATPase</h1>
<div class="mw-body-content" id="bodyContent">
<div class="noprint" id="siteSub">From Wikipedia, the free encyclopedia</div>
<div id="contentSub"></div>
<div id="jump-to-nav"></div>
Jump to navigation
Jump to search
<div class="mw-content-ltr" dir="ltr" id="mw-content-text" lang="en"><div class="mw-parser-output"><table class="infobox" style="width:22em"><tbody><tr><td colspan="2" style="text-align:center"><img alt="1wpg opm.png" data-file-height="742" data-file-width="549" decoding="async" height="297" src="https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https:https://upload.wikimedia.org/wikipedia/commons/thumb/e/e6/1wpg_opm.png/220px-1wpg_opm.png" srcset width="220"/><div>Calcium ATPase, E2-Pi state</div></td></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Identifiers</th></tr><tr><th scope="row" style="background-color: #e7dcc3">Symbol</th><td style="background-color: #eee">E1-E2_ATPase</td></tr><tr><th scope="row" style="background-color: #e7dcc3">Pfam</th><td class="pfam" style="background-color: #eee">PF00122</td></tr><tr><th scope="row" style="background-color: #e7dcc3">InterPro</th><td style="background-color: #eee">IPR008250</td></tr><tr><th scope="row" style="background-color: #e7dcc3">PROSITE</th><td style="background-color: #eee">PDOC00139</td></tr><tr><th scope="row" style="background-color: #e7dcc3">SCOPe</th><td style="background-color: #eee">1su4</td></tr><tr><th scope="row" style="background-color: #e7dcc3">SUPERFAMILY</th><td style="background-color: #eee">1su4</td></tr><tr><th scope="row" style="background-color: #e7dcc3">TCDB</th><td style="background-color: #eee">3.A.3</td></tr><tr><th scope="row" style="background-color: #e7dcc3">OPM superfamily</th><td style="background-color: #eee">22</td></tr><tr><th scope="row" style="background-color: #e7dcc3">OPM protein</th><td style="background-color: #eee">3b9b</td></tr><tr><th scope="row" style="background-color: #e7dcc3">Membranome</th><td style="background-color: #eee">224</td></tr><tr><td colspan="2" style="text-align:center;background-color: #eee"><table class="infobox collapsible collapsed" style="width:22em;float:none; clear:none; margin:0; border-width:0; border-collapse:collapse; text-align:left; width:100%"><tbody><tr><th colspan="2" style="text-align:center;background-color: #ddd">Available protein structures:</th></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">Pfam</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">structures</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PDB</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">RCSB PDB; PDBe; PDBj</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PDBsum</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">structure summary</td></tr></tbody></table></td></tr></tbody></table>
<p>The <b>P-type ATPases</b>, also known as <b>E<sub>1</sub>-E<sub>2</sub> ATPases</b>, are a large group of evolutionarily related ion and lipid pumps that are found in bacteria, archaea, and eukaryotes. P-type ATPases are Î±-helical bundle primary transporters named based upon their ability to catalyze auto- (or self-) phosphorylation (hence P) of a key conserved aspartate residue within the pump and their energy source, adenosine triphosphate (ATP). In addition, they all appear to interconvert between at least two different conformations, denoted by E<sub>1</sub> and E<sub>2</sub>. P-type ATPases fall under the <b>P-type ATPase (P-ATPase) Superfamily</b> (TC# 3.A.3) which, as of early 2016, includes 20 different protein families.
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p><p>Most members of this transporter superfamily catalyze cation uptake and/or efflux, however one subfamily (TC# 3.A.3.8) is involved in flipping phospholipids to maintain the asymmetric nature of the biomembrane.
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p><p>In humans, P-type ATPases serve as a basis for nerve impulses, relaxation of muscles, secretion and absorption in the kidney, absorption of nutrient in the intestine and other physiological processes. Prominent examples of P-type ATPases are the sodium-potassium pump (Na/K-ATPase), the proton-potassium pump (H/K-ATPase), the calcium pump (Ca-ATPase) and the plasma membrane proton pump (H-ATPase) of plants and fungi.
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p>
<div class="toc" id="toc"><input class="toctogglecheckbox" id="toctogglecheckbox" role="button" style="display:none" type="checkbox"/><div class="toctitle" dir="ltr" lang="en"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">General transport reaction</span></li>
<li class="toclevel-1 tocsection-2"><span class="tocnumber">2</span> <span class="toctext">Discovery</span></li>
<li class="toclevel-1 tocsection-3"><span class="tocnumber">3</span> <span class="toctext">Structure</span>
<ul>
<li class="toclevel-2 tocsection-4"><span class="tocnumber">3.1</span> <span class="toctext">Membrane section</span></li>
<li class="toclevel-2 tocsection-5"><span class="tocnumber">3.2</span> <span class="toctext">Phosphorylation (P) domain</span></li>
<li class="toclevel-2 tocsection-6"><span class="tocnumber">3.3</span> <span class="toctext">Nucleotide binding (N) domain</span></li>
<li class="toclevel-2 tocsection-7"><span class="tocnumber">3.4</span> <span class="toctext">Actuator (A) domain</span></li>
<li class="toclevel-2 tocsection-8"><span class="tocnumber">3.5</span> <span class="toctext">Regulatory (R) domain</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-9"><span class="tocnumber">4</span> <span class="toctext">Mechanism</span></li>
<li class="toclevel-1 tocsection-10"><span class="tocnumber">5</span> <span class="toctext">Classification</span></li>
<li class="toclevel-1 tocsection-11"><span class="tocnumber">6</span> <span class="toctext">P1 ATPases</span>
<ul>
<li class="toclevel-2 tocsection-12"><span class="tocnumber">6.1</span> <span class="toctext">P1A ATPases (potassium pumps)</span></li>
<li class="toclevel-2 tocsection-13"><span class="tocnumber">6.2</span> <span class="toctext">P1B ATPases (heavy metal pumps)</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-14"><span class="tocnumber">7</span> <span class="toctext">P2 ATPases</span>
<ul>
<li class="toclevel-2 tocsection-15"><span class="tocnumber">7.1</span> <span class="toctext">P2A ATPases (calcium pumps)</span></li>
<li class="toclevel-2 tocsection-16"><span class="tocnumber">7.2</span> <span class="toctext">P2B ATPases (calcium pumps)</span></li>
<li class="toclevel-2 tocsection-17"><span class="tocnumber">7.3</span> <span class="toctext">P2C ATPases (sodium/potassium and proton/potassium pumps)</span></li>
<li class="toclevel-2 tocsection-18"><span class="tocnumber">7.4</span> <span class="toctext">P2D ATPases (sodium pumps)</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-19"><span class="tocnumber">8</span> <span class="toctext">P3 ATPases</span>
<ul>
<li class="toclevel-2 tocsection-20"><span class="tocnumber">8.1</span> <span class="toctext">P3A ATPases (proton pumps)</span></li>
<li class="toclevel-2 tocsection-21"><span class="tocnumber">8.2</span> <span class="toctext">P3B ATPases (magnesium pumps)</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-22"><span class="tocnumber">9</span> <span class="toctext">P4 ATPases (phospholipid flippases)</span></li>
<li class="toclevel-1 tocsection-23"><span class="tocnumber">10</span> <span class="toctext">P5 ATPases</span>
<ul>
<li class="toclevel-2 tocsection-24"><span class="tocnumber">10.1</span> <span class="toctext">P5A ATPases</span></li>
<li class="toclevel-2 tocsection-25"><span class="tocnumber">10.2</span> <span class="toctext">P5B ATPases</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-26"><span class="tocnumber">11</span> <span class="toctext">Further phylogenetic classification</span>
<ul>
<li class="toclevel-2 tocsection-27"><span class="tocnumber">11.1</span> <span class="toctext">Horizontal Gene Transfer</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-28"><span class="tocnumber">12</span> <span class="toctext">Human genes</span></li>
<li class="toclevel-1 tocsection-29"><span class="tocnumber">13</span> <span class="toctext">See also</span></li>
<li class="toclevel-1 tocsection-30"><span class="tocnumber">14</span> <span class="toctext">References</span></li>
</ul>
</div>
<h2><span class="mw-headline" id="General_transport_reaction">General transport reaction</span></h2>
<p>The generalized reaction for P-type ATPases is:
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p><p>nLigand<sub>1</sub> (out) + mLigand<sub>2</sub> (in) + ATP â†’ nLigand<sub>1</sub> (in) + mLigand<sub>2</sub> (out) + ADP + P<sub>i</sub>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p><p>where the ligand can be either a metal ion or a phospholipid molecule.
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p>
<h2><span class="mw-headline" id="Discovery">Discovery</span></h2>
<p>The first P-type ATPase discovered was the Na/K-ATPase, which Nobel laureate Jens Christian Skou isolated in 1957. The Na/K-ATPase was only the first member of a large and still-growing protein family (see Swiss-Prot Prosite motif PS00154).
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p>
<h2><span class="mw-headline" id="Structure">Structure</span></h2>
<p>P-type ATPases have a single catalytic subunit of 70 - 140 kDa. The catalytic subunit hydrolyzes ATP, contain the aspartyl phosphorylation site and binding sites for the transported ligand(s) and catalyzes ion transport. Various subfamilies of P-type ATPases also need additional subunits for proper function. Additional subunits that lack catalytic activity are present in the ATPase complexes of P1A, P2A, P2C and P4 ATPases. E.g. the catalytic alpha subunit of Na/K-ATPase consists of two additional subunits, beta and gamma, involved in trafficking, folding, and regulation of these pumps. The first P-type ATPase to be crystallized was SERCA1a, a sarco(endo)plasmic reticulum Ca-ATPase of fast twitch muscle from adult rabbit. It is generally acknowledged that the structure of SERCA1a is representative for the superfamily of P-type ATPases.
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p><p>The catalytic subunit of P-type ATPases is composed of a cytoplasmic section and a transmembrane section with binding sites for the transported ligand(s). The cytoplasmic section consists of three cytoplasmic domains, designated the P, N, and A domains, containing over half the mass of the protein. 
<span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p>
<h3><span class="mw-headline" id="Membrane_section">Membrane section</span></h3>
<p>The transmembrane section (<u>M domain</u>) typically has ten transmembrane helices (M1-M10), with the binding sites for transported ligand(s) located near the midpoint of the bilayer. While most subfamilies have 10 transmembrane helices, there are some notable exceptions. The P1A ATPases are predicted to have 7, and the large subfamily of heavy metal pumps P1B) is predicted to have 8 transmembrane helices. P5 ATPases appear to have a total of 12 transmembrane helices.
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p><p>Common for all P-type ATPases is a core of 6 transmembrane-spanning segments (also called the 'transport (T) domain'; M1-M6 in SERCA), that harbors the binding sites for the translocated ligand(s). The ligand(s) enter through a half-channel to the binding site and leave on the other side of the membrane through another half-channel.
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p><p>Varying among P-type ATPase is the additional number of transmembrane-spanning segments (also called the 'support (S) domain', which between subfamilies ranges from 2 to 6. Extra transmembrane-segments likely provides structural support for the T domain and can also have specialized functions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p>
<h3><span id="Phosphorylation_.28P.29_domain"></span><span class="mw-headline" id="Phosphorylation_(P)_domain">Phosphorylation (P) domain</span></h3>
<p>The P domain contains the canonical aspartic acid residue phosphorylated (in a conserved DKTGT motif; the 'D' is the one letter abbreviation of the amino acid aspartate) during the reaction cycle. It is composed of two parts widely separated in sequence. These two parts assemble into a seven-strand parallel Î²-sheet with eight short associated a-helices, forming a Rossmann fold.
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p><p>The folding pattern and the locations of the critical amino acids for phosphorylation in P-type ATPases has the haloacid dehalogenase fold characteristic of the haloacid dehalogenase (HAD) superfamily, as predicted by sequence homology. The HAD superfamily functions on the common theme of an aspartate ester formation by an S<sub>N</sub>2 reaction mechanism. This S<sub>N</sub>2 reaction is clearly observed in the solved structure of SERCA with ADP plus AlF<sub>4</sub>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p>
<h3><span id="Nucleotide_binding_.28N.29_domain"></span><span class="mw-headline" id="Nucleotide_binding_(N)_domain">Nucleotide binding (N) domain</span></h3>
<p>The N domain serves as a built-in protein kinase that functions to phosphorylate the P domain. The N domain is inserted between the two segments of the P domain, and is formed of a seven-strand antiparallel Î²-sheet between two helix bundles. This domain contains the ATP-binding pocket, pointing out toward the solvent near the P-domain.
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p>
<h3><span id="Actuator_.28A.29_domain"></span><span class="mw-headline" id="Actuator_(A)_domain">Actuator (A) domain</span></h3>
<p>The A domain serves as a built-in protein phosphatase that functions to dephosphorylate the phosphorylated P domain. The A domain is the smallest of the three cytoplasmic domains. It consists of a distorted jellyroll structure and two short helices. It is the actuator domain modulating the occlusion of the transported ligand(s) in the transmembrane binding sites, and it is pivot in transposing the energy from the hydrolysis of ATP in the cytoplasmic domains to the vectorial transport of cations in the transmembrane domain. The A domain dephosphorylates the P domain as part of the reaction cycle using a highly conserved TGES motif located at one end of the jellyroll.
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
<h3><span id="Regulatory_.28R.29_domain"></span><span class="mw-headline" id="Regulatory_(R)_domain">Regulatory (R) domain</span></h3>
<p>Some members of the P-type ATPase family have additional regulatory (R) domains fused to the pump. Heavy metal P1B pumps can have several N- and C-terminal heavy metal-binding domains that have been found to be involved in regulation. The P2B Ca<sub>2+</sub> ATPases have autoinbitory domains in their amino-terminal (plants) or carboxy-terminal (animals) regions, which contain binding sites for calmodulin, which, in the presence of Ca<sub>2+</sub>, activates P2B ATPases by neutralizing the terminal constraint. The P3A plasma membrane proton pumps have a C-terminal regulatory domain, which, when unphosphorylated, inhibits pumping.
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p>
<h2><span class="mw-headline" id="Mechanism">Mechanism</span></h2>
<p>All P-type ATPases use the energy derived from ATP to drive transport. They form a high-energy aspartyl-phosphoanhydride intermediate in the reaction cycle, and they interconvert between at least two different conformations, denoted by E<sub>1</sub> and E<sub>2</sub>. The E<sub>1</sub>-E<sub>2</sub> notation stems from the initial studies on this family of enzymes made on the Na/K-ATPase, where the sodium form and the potassium form are referred to as E<sub>1</sub> and E<sub>2</sub>, respectively, in the "Post-Albers scheme". The E<sub>1</sub>-E<sub>2</sub> schema has been proven to work, but there exist more than two major conformational states. The E<sub>1</sub>-E<sub>2</sub> notation highlights the selectivity of the enzyme. In E<sub>1</sub>, the pump has high affinity for the exported substrate and low affinity for the imported substrate. In E<sub>2</sub>, it has low affinity of the exported substrate and high affinity for the imported substrate. Four major enzyme states form the cornerstones in the reaction cycle. Several additional reaction intermediates occur interposed. These are termed E<sub>1</sub>~P, E<sub>2</sub>P, E<sub>2</sub>-P*, and E<sub>1</sub>/E<sub>2</sub>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p><p>ATP hydrolysis occurs in the cytoplasmic headpiece at the interface between domain N and P. Two Mg-ion sites form part of the active site. ATP hydrolysis is tightly coupled to translocation of the transported ligand(s) through the membrane, more than 40 Ã… away, by the A domain.
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
<h2><span class="mw-headline" id="Classification">Classification</span></h2>
<p>A phylogenetic analysis of 159 sequences made in 1998 by Axelsen and Palmgren suggested that P-type ATPases can be divided into five subfamilies (types; designated as P1-P5), based strictly on a conserved sequence kernel excluding the highly variable N and C terminal regions. Chan et al. (2010) also analyzed P-type ATPases in all major prokaryotic phyla for which complete genome sequence data were available and compared the results with those for eukaryotic P-type ATPases. The phylogenetic analysis grouped the proteins independent of the organism from which they are isolated and showed that the diversification of the P-type ATPase family occurred prior to the separation of eubacteria, archaea, and eucaryota. This underlines the significance of this protein family for cell survival under stress conditions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">20</span></p>
<h2><span class="mw-headline" id="P1_ATPases">P1 ATPases</span></h2>
<p>P1 ATPases ATPases (or Type I) consists of the transition/heavy metal ATPases. Topological type I (heavy metal) P-type ATPases predominate in prokaryotes (approx. tenfold).
<span style="border: 3px solid blue; color:blue;font-weight: bold">21</span></p>
<h3><span id="P1A_ATPases_.28potassium_pumps.29"></span><span class="mw-headline" id="P1A_ATPases_(potassium_pumps)">P1A ATPases (potassium pumps)</span></h3>
<p>P1A ATPases (or Type IA) are involved in K import (TC# 3.A.3.7). They are atypical P-type ATPases because, unlike other P-type ATPases, they function as part of a heterotetrameric complex (called KdpFABC), where the actual K transport is mediated by another subcomponent of the complex.
<span style="border: 3px solid blue; color:blue;font-weight: bold">22</span></p>
<h3><span id="P1B_ATPases_.28heavy_metal_pumps.29"></span><span class="mw-headline" id="P1B_ATPases_(heavy_metal_pumps)">P1B ATPases (heavy metal pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Zn2+-exporting ATPase</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: ATP7A</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Wilson disease protein</div>
<p>P1B ATPases (or Type IB ATPases) are involved in transport of the soft Lewis acids: Cu, Ag, Cu, Zn, Cd, Pb and Co (TC#s 3.A.3.5 and 3.A.3.6). They are key elements for metal resistance and metal homeostasis in a wide range of organisms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">23</span></p><p>Metal binding to transmembrane metal-binding sites (TM-MBS) in Cu-ATPases is required for enzyme phosphorylation and subsequent transport. However, Cu does not access Cu-ATPases in a free (hydrated) form but is bound to a chaperone protein. The delivery of Cu by <i>Archaeoglobus fulgidus</i> Cu-chaperone, CopZ (see TC# 3.A.3.5.7), to the corresponding Cu-ATPase, CopA (TC# 3.A.3.5.30), has been studied. CopZ interacted with and delivered the metal to the N-terminal metal binding domain(s) of CopA (MBDs). Cu-loaded MBDs, acting as metal donors, were unable to activate CopA or a truncated CopA lacking MBDs. Conversely, Cu-loaded CopZ activated the CopA ATPase and CopA constructs in which MBDs were rendered unable to bind Cu. Furthermore, under nonturnover conditions, CopZ transferred Cu to the TM-MBS of a CopA lacking MBDs altogether. Thus, MBDs may serve a regulatory function without participating directly in metal transport, and the chaperone delivers Cu directly to transmembrane transport sites of Cu-ATPases. Wu et al. (2008) have determined structures of two constructs of the Cu (CopA) pump from <i>Archaeoglobus fulgidus</i> by cryoelectron microscopy of tubular crystals, which revealed the overall architecture and domain organization of the molecule. They localized its N-terminal MBD within the cytoplasmic domains that use ATP hydrolysis to drive the transport cycle and built a pseudoatomic model by fitting existing crystallographic structures into the cryoelectron microscopy maps for CopA. The results also similarly suggested a Cu-dependent regulatory role for the MBD.
<span style="border: 3px solid blue; color:blue;font-weight: bold">24</span></p><p>In the <i>Archaeoglobus fulgidus</i> CopA (TC# 3.A.3.5.7), invariant residues in helixes 6, 7 and 8 form two transmembrane metal binding sites (TM-MBSs). These bind Cu with high affinity in a trigonal planar geometry. The cytoplasmic Cu chaperone CopZ transfers the metal directly to the TM-MBSs; however, loading both of the TM-MBSs requires binding of nucleotides to the enzyme. In agreement with the classical transport mechanism of P-type ATPases, occupancy of both transmembrane sites by cytoplasmic Cu is a requirement for enzyme phosphorylation and subsequent transport into the periplasmic or extracellular milieu. Transport studies have shown that most Cu-ATPases drive cytoplasmic Cu efflux, albeit with quite different transport rates in tune with their various physiological roles. Archetypical Cu-efflux pumps responsible for Cu tolerance, like the <i>Escherichia coli</i> CopA, have turnover rates ten times higher than those involved in cuproprotein assembly (or alternative functions). This explains the incapability of the latter group to significantly contribute to the metal efflux required for survival in high copper environments.  Structural and mechanistic details of copper-transporting P-type ATPase functionhave been described.
<span style="border: 3px solid blue; color:blue;font-weight: bold">25</span></p>
<h2><span class="mw-headline" id="P2_ATPases">P2 ATPases</span></h2>
<p>P2 ATPases (or Type II ATPases) are split into four groups. Topological type II ATPases (specific for Na,K, H Ca, Mg and phospholipids) predominate in eukaryotes (approx. twofold).
<span style="border: 3px solid blue; color:blue;font-weight: bold">26</span></p>
<h3><span id="P2A_ATPases_.28calcium_pumps.29"></span><span class="mw-headline" id="P2A_ATPases_(calcium_pumps)">P2A ATPases (calcium pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: SERCA</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: ATP2A1</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Secretory Pathway CaÂ²âº ATPase</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Calcium pump</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Calcium ATPase</div>
<p>P2A ATPases (or Type IIA ATPases) are Ca ATPases that transport Ca. P2A ATPases are split into two groups. Members of the first group are called sarco/endoplasmatic reticulum Ca-ATPases (also referred to as SERCA). These pumps have two Ca ion binding sites and are often regulated by inhibitory accessory proteins having a single trans-membrane spanning segment (e.g.phospholamban and sarcolipin. In the cell, they are located in the sarcoplasmic  or endoplasmatic reticulum. SERCA1a is a type IIA pump. The second group of P2A ATPases is called secretory pathway Ca-ATPases (also referred to as SPCA). These pumps have a single Ca ion binding site and are located in secretory vesicles (animals) or the vacuolar membrane (fungi). (TC# 3.A.3.2)
<span style="border: 3px solid blue; color:blue;font-weight: bold">27</span></p><p>Crystal structures of Sarcoplasimc/endoplasmic reticulum ATP driven calcium pumps can be found in RCSB.
<span style="border: 3px solid blue; color:blue;font-weight: bold">28</span></p><p>SERCA1a is composed of a cytoplasmic section and a transmembrane section with two Ca-binding sites. The cytoplasmic section consists of three cytoplasmic domains, designated the P, N, and A domains, containing over half the mass of the protein. The transmembrane section has ten transmembrane helices (M1-M10), with the two Ca-binding sites located near the midpoint of the bilayer. The binding sites are formed by side-chains and backbone carbonyls from M4, M5, M6, and M8. M4 is unwound in this region due to a conserved proline (P308). This unwinding of M4 is recognised as a key structural feature of P-type ATPases.
<span style="border: 3px solid blue; color:blue;font-weight: bold">29</span></p><p>Structures are available for both the E<sub>1</sub> and E<sub>2</sub> states of the Ca ATPase showing that Ca binding induces major changes in all three cytoplasmic domains relative to each other.
<span style="border: 3px solid blue; color:blue;font-weight: bold">30</span></p><p>In the case of SERCA1a, energy from ATP is used to transport 2 Ca-ions from the cytoplasmic side to the lumen of the sarcoplasmatic reticulum, and to countertransport 1-3 protons into the cytoplasm. Starting in the E<sub>1</sub>/E<sub>2</sub> state, the reaction cycle begins as the enzyme releases 1-3 protons from the cation-ligating residues, in exchange for cytoplasmic Ca-ions. This leads to assembly of the phosphorylation site between the ATP-bound N domain and the P domain, while the A domain directs the occlusion of the bound Ca. In this occluded state, the Ca ions are buried in a proteinaceous environment with no access to either side of the membrane.The Ca<sub>2</sub>E<sub>1</sub>~P state becomes formed through a kinase reaction, where the P domain becomes phosphorylated, producing ADP. The cleavage of the Î²-phosphodiester bond releases the gamma-phosphate from ADP and unleashes the N domain from the P domain.
<span style="border: 3px solid blue; color:blue;font-weight: bold">31</span></p><p>This then allows the A domain to rotate toward the phosphorylation site, making a firm association with both the P and the N domains. This movement of the A domain exerts a downward push on M3-M4 and a drag on M1-M2, forcing the pump to open at the luminal side and forming the E<sub>2</sub>P state. During this transition, the transmembrane Ca-binding residues are forced apart, destroying the high-affinity binding site. This is in agreement with the general model form substrate translocation, showing that energy in primary transport is not used to bind the substrate but to release it again from the buried counter ions. At the same time the N domain becomes exposed to the cytosol, ready for ATP exchange at the nucleotide-binding site.
<span style="border: 3px solid blue; color:blue;font-weight: bold">32</span></p><p>As the Ca dissociate to the luminal side, the cation binding sites are neutralised by proton binding, which makes a closure of the transmembrane segments favourable. This closure is coupled to a downward rotation of the A domain and a movement of the P domain, which then leads to the E<sub>2</sub>-P* occluded state. Meanwhile, the N domain exchanges ADP for ATP.
<span style="border: 3px solid blue; color:blue;font-weight: bold">33</span></p><p>The P domain is dephosphorylated by the A domain, and the cycle completes when the phosphate is released from the enzyme, stimulated by the newly bound ATP, while a cytoplasmic pathway opens to exchange the protons for two new Ca-ions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">34</span></p><p>Xu et al. proposed how Ca binding induces conformational changes in TMS 4 and 5 in the membrane domain (M) that in turn induce rotation of the phosphorylation domain (P). The nucleotide binding (N) and Î²-sheet (Î²) domains are highly mobile, with N flexibly linked to P, and Î² flexibly linked to M. Modeling of the fungal H ATPase, based on the structures of the Ca pump, suggested a comparable 70Âº rotation of N relative to P to deliver ATP to the phosphorylation site.
<span style="border: 3px solid blue; color:blue;font-weight: bold">35</span></p><p>One report suggests that this sarcoplasmic reticulum (SR) Ca ATPase is homodimeric.
<span style="border: 3px solid blue; color:blue;font-weight: bold">36</span></p><p>Crystal structures have shown that the conserved TGES loop of the Ca-ATPase is isolated in the Ca<sub>2</sub><i>E</i><sub>1</sub> state but becomes inserted in the catalytic site in <i>E</i><sub>2</sub> states. Anthonisen et al. (2006) characterized the kinetics of the partial reaction steps of the transport cycle and the binding of the phosphoryl analogs BeF, AlF, MgF, and vanadate in mutants with alterations to conserved TGES loop residues. The data provide functional evidence supporting a role of Glu in activating the water molecule involved in the <i>E</i><sub>2</sub>P â†’ <i>E</i><sub>2</sub> dephosphorylation and suggest a direct participation of the side chains of the TGES loop in the control and facilitation of the insertion of the loop in the catalytic site. The interactions of the TGES loop furthermore seem to facilitate its disengagement from the catalytic site during the <i>E</i><sub>2</sub> â†’ Ca<sub>2</sub><i>E</i><sub>1</sub> transition.
<span style="border: 3px solid blue; color:blue;font-weight: bold">37</span></p><p>Crystal Structures of Calcium ATPase are available in RCSB and include: <span class="plainlinks">PDB: 4AQR</span>â€‹, <span class="plainlinks">2L1W</span>â€‹, <span class="plainlinks">2M7E</span>â€‹, <span class="plainlinks">2M73</span>â€‹, among others.
<span style="border: 3px solid blue; color:blue;font-weight: bold">38</span></p>
<h3><span id="P2B_ATPases_.28calcium_pumps.29"></span><span class="mw-headline" id="P2B_ATPases_(calcium_pumps)">P2B ATPases (calcium pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Plasma membrane Ca2+ ATPase</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Calcium pump</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Calcium ATPase</div>
<p>P2B (or Type IIB ATPases) are Ca ATPases that transport Ca. These pumps have a single Ca ion binding site and are regulated by binding of calmodulin to autoinhibitory built-in domains situated at either the carboxy-terminal (animals) or amino-terminal (plants) end of the pump protein. In the cell, they are situated in the plasma membrane (animals and plants) and the internal membranes (plants). Plasma membrane Ca-ATPase (also referred to as PMCA) of animals is a P2B ATPase (TC# 3.A.3.2)
<span style="border: 3px solid blue; color:blue;font-weight: bold">39</span></p>
<h3><span id="P2C_ATPases_.28sodium.2Fpotassium_and_proton.2Fpotassium_pumps.29"></span><span class="mw-headline" id="P2C_ATPases_(sodium/potassium_and_proton/potassium_pumps)">P2C ATPases (sodium/potassium and proton/potassium pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Na+/K+-ATPase</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Na /K -ATPase</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Hydrogen potassium ATPase</div>
<p>P2C ATPases (or Type IIC) include the closely related Na/K and H/K ATPases from animal cells. (TC# 3.A.3.1)
<span style="border: 3px solid blue; color:blue;font-weight: bold">40</span></p><p>The X-ray crystal structure at 3.5 Ã… resolution of the pig renal Na/K-ATPase has been determined with two rubidium ions bound in an occluded state in the transmembrane part of the Î±-subunit. Several of the residues forming the cavity for rubidium/potassium occlusion in the Na/K-ATPase are homologous to those binding calcium in the Ca-ATPase of the sarco(endo)plasmic reticulum. The carboxy terminus of the Î±-subunit is contained within a pocket between transmembrane helices and seems to be a novel regulatory element controlling sodium affinity, possibly influenced by the membrane potential.
<span style="border: 3px solid blue; color:blue;font-weight: bold">41</span></p><p>Crystal Structures are available in RCSB and include: <span class="plainlinks">PDB: 4RES</span>â€‹, <span class="plainlinks">4RET</span>â€‹, <span class="plainlinks">3WGU</span>â€‹, <span class="plainlinks">3WGV</span>â€‹, among others.
<span style="border: 3px solid blue; color:blue;font-weight: bold">42</span></p>
<h3><span id="P2D_ATPases_.28sodium_pumps.29"></span><span class="mw-headline" id="P2D_ATPases_(sodium_pumps)">P2D ATPases (sodium pumps)</span></h3>
<p>P2D ATPases (or Type IID) include a small number of Na (and K) exporting ATPases found in fungi and mosses. (Fungal K transporters; TC# 3.A.3.9)
<span style="border: 3px solid blue; color:blue;font-weight: bold">43</span></p>
<h2><span class="mw-headline" id="P3_ATPases">P3 ATPases</span></h2>
<p>P3 ATPases or (or Type III ATPases) are split into two groups.
<span style="border: 3px solid blue; color:blue;font-weight: bold">44</span></p>
<h3><span id="P3A_ATPases_.28proton_pumps.29"></span><span class="mw-headline" id="P3A_ATPases_(proton_pumps)">P3A ATPases (proton pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Plasma membrane H+-ATPase</div>
<p>P3A ATPases (or Type IIIA) contain the plasma membrane H-ATPases from prokaryotes, protists, plants and fungi.
<span style="border: 3px solid blue; color:blue;font-weight: bold">45</span></p><p>Plasma membrane H<sub>+</sub>-ATPase is best characterized in plants and yeast. It maintains the level of intracellular pH and transmembrane potential. Ten transmembrane helices and three cytoplasmic domains define the functional unit of ATP-coupled proton transport across the plasma membrane, and the structure is locked in a functional state not previously observed in P-type ATPases. The transmembrane domain reveals a large cavity, which is likely to be filled with water, located near the middle of the membrane plane where it is lined by conserved hydrophilic and charged residues. Proton transport against a high membrane potential is readily explained by this structural arrangement.
<span style="border: 3px solid blue; color:blue;font-weight: bold">46</span></p>
<h3><span id="P3B_ATPases_.28magnesium_pumps.29"></span><span class="mw-headline" id="P3B_ATPases_(magnesium_pumps)">P3B ATPases (magnesium pumps)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Magnesium-ATPase</div>
<p>P3B ATPases (or Type IIIB) are presumed Mg-ATPases found in eubacteria and plants. Fungal H transporters (TC# 3.A.3.3) and Mg (TC# 3.A.3.4)
<span style="border: 3px solid blue; color:blue;font-weight: bold">47</span></p>
<h2><span id="P4_ATPases_.28phospholipid_flippases.29"></span><span class="mw-headline" id="P4_ATPases_(phospholipid_flippases)">P4 ATPases (phospholipid flippases)</span></h2>
<div class="hatnote navigation-not-searchable" role="note">Main article: Flippase</div>
<p>P4 ATPases (or Type IV ATPases) are flippases involved in the transport of phospholipids, such as phosphatidylserine, phosphatidylcholine and phosphatidylethanolamine.
<span style="border: 3px solid blue; color:blue;font-weight: bold">48</span></p>
<h2><span class="mw-headline" id="P5_ATPases">P5 ATPases</span></h2>
<p>P5 ATPases (or Type V ATPases) have unknown specificity. This large group is found only in eukaryotes and is further divided into two groups..
<span style="border: 3px solid blue; color:blue;font-weight: bold">49</span></p>
<h3><span class="mw-headline" id="P5A_ATPases">P5A ATPases</span></h3>
<p>P5A ATPases (or Type VA) are involved in regulation of homeostasis in the endoplasmic reticulum.
<span style="border: 3px solid blue; color:blue;font-weight: bold">50</span></p>
<h3><span class="mw-headline" id="P5B_ATPases">P5B ATPases</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: ATP13A2</div>
<p>P5B ATPases (or Type VB) are found in the lysosomal membrane of animals. Mutations in these pumps are linked to a variety of neurological diseases.
<span style="border: 3px solid blue; color:blue;font-weight: bold">51</span></p>
<h2><span class="mw-headline" id="Further_phylogenetic_classification">Further phylogenetic classification</span></h2>
<p>In addition to the subfamilies of P-type ATPases listed above, several prokaryotic families of unknown function have been identified. The Transporter Classification Database provides a representative list of members of the P-ATPase superfamily, which as of early 2016 consisting of 20 families. Members of the P-ATPase superfamily are found in bacteria, archaea and eukaryotes. Clustering on the phylogenetic tree is usually in accordance with specificity for the transported ion(s).
<span style="border: 3px solid blue; color:blue;font-weight: bold">52</span></p><p>In eukaryotes, they are present in the plasma membranes or endoplasmic reticular membranes. In prokaryotes, they are localized to the cytoplasmic membranes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">53</span></p><p>P-type ATPases from 26 eukaryotic species were analyzed later.
<span style="border: 3px solid blue; color:blue;font-weight: bold">54</span></p><p>Chan et al., (2010) conducted an equivalent but more extensive analysis of the P-type ATPase Superfamily in Prokaryotes and compared them with those from Eukaryotes. While some families are represented in both types of organisms, others are found only in one of the other type. The primary functions of prokaryotic P-type ATPases appear to be protection from environmental stress conditions. Only about half of the P-type ATPase families are functionally characterized.
<span style="border: 3px solid blue; color:blue;font-weight: bold">55</span></p>
<h3><span class="mw-headline" id="Horizontal_Gene_Transfer">Horizontal Gene Transfer</span></h3>
<p>Many P-type ATPase families are found exclusively in prokaryotes (e.g. Kdp-type K uptake ATPases (type III) and all prokaryotic functionally uncharacterized P-type ATPase (FUPA) families), while others are restricted to eukaryotes (e.g. phospholipid flippases and all 13 eukaryotic FUPA families). Horizontal gene transfer has occurred frequently among bacteria and archaea, which have similar distributions of these enzymes, but rarely between most eukaryotic kingdoms, and even more rarely between eukaryotes and prokaryotes. In some bacterial phyla (e.g. Bacteroidetes, Flavobacteria and Fusobacteria), ATPase gene gain and loss as well as horizontal transfer occurred seldom in contrast to most other bacterial phyla. Some families (i.e., Kdp-type ATPases) underwent far less horizontal gene transfer than other prokaryotic families, possibly due to their multisubunit characteristics. Functional motifs are better conserved across family lines than across organismal lines, and these motifs can be family specific, facilitating functional predictions. In some cases, gene fusion events created P-type ATPases covalently linked to regulatory catalytic enzymes. In one family (FUPA Family 24), a type I ATPase gene (N-terminal) is fused to a type II ATPase gene (C-terminal) with retention of function only for the latter. Genome minimalization led to preferential loss of P-type ATPase genes. Chan et al. (2010) suggested that in prokaryotes and some unicellular eukaryotes, the primary function of P-type ATPases is protection from extreme environmental stress conditions. The classification of P-type ATPases of unknown function into phylogenetic families provides guides for future molecular biological studies.
<span style="border: 3px solid blue; color:blue;font-weight: bold">56</span></p>
<h2><span class="mw-headline" id="Human_genes">Human genes</span></h2>
<p>Human genes encoding P-type ATPases or P-type ATPase-like proteins include:
<span style="border: 3px solid blue; color:blue;font-weight: bold">57</span></p>
<ul><li>P1B: Cu ATPase: ATP7A, ATP7B</li>
<li>P2A: SERCA Ca ATPase: ATP2A1, ATP2A2, ATP2A3</li>
<li>P2A: secretory pathway Ca-ATPase: ATP2C2, ATP2C2</li>
<li>P2B: Ca ATPase: ATP2B1, ATP2B2, ATP2B3, ATP2B4, ATP2C1</li>
<li>P2C: Na/K ATPase: ATP1A1, ATP1A2, ATP1A3, ATP1A4, ATP1B1, ATP1B2, ATP1B3, ATP1B4</li>
<li>P2C: H/K ATPase, gastric: ATP4A;</li>
<li>P2C: H/K ATPase, nongastric: ATP12A</li>
<li>P4: Flippase: ATP8A1, ATP8B1, ATP8B2, ATP8B3, ATP8B4, ATP9A, ATP9B, ATP10A, ATP10B, ATP10D, ATP11A, ATP11B, ATP11C</li>
<li>P5: ATP13A1, ATP13A2, ATP13A3, ATP13A4, ATP13A5</li></ul>
<script>function offsetAnchor() {if(location.hash.length !== 0) {window.scrollTo(window.scrollX, window.scrollY - 250);}}window.addEventListener("hashchange", offsetAnchor);window.setTimeout(offsetAnchor, 1);</script>