<!DOCTYPE html>

<html class="client-nojs" dir="ltr" lang="en">
<head>
<meta charset="utf-8"/>
<title>Gene Disease Database - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Gene_Disease_Database","wgTitle":"Gene Disease Database","wgCurRevisionId":893553973,"wgRevisionId":893553973,"wgArticleId":44248347,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["Use dmy dates from November 2014","Biological databases"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Gene_Disease_Database","wgRelevantArticleId":44248347,"wgRequestId":"XMZDkQpAICoAADIA@sYAAADR","wgCSPNonce":false,"wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsReferencePreviews":false,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgWMESchemaEditAttemptStepOversample":false,"wgPoweredByHHVM":true,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q18392279","wgCentralAuthMobileDomain":false,"wgEditSubmitButtonLabelPublish":true});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});RLPAGEMODULES=["ext.cite.ux-enhancements","site","mediawiki.page.startup","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];mw.loader.load(RLPAGEMODULES);});</script>
<link href="/w/load.php?lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta content="" name="ResourceLoaderDynamicStyles"/>
<link href="/w/load.php?lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<link href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<meta content="MediaWiki 1.34.0-wmf.1" name="generator"/>
<meta content="origin" name="referrer"/>
<meta content="origin-when-crossorigin" name="referrer"/>
<meta content="origin-when-cross-origin" name="referrer"/>
<link href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Gene_Disease_Database" rel="alternate"/>
<link href="/w/index.php?title=Gene_Disease_Database&amp;action=edit" rel="alternate" title="Edit this page" type="application/x-wiki"/>
<link href="/w/index.php?title=Gene_Disease_Database&amp;action=edit" rel="edit" title="Edit this page"/>
<link href="/static/apple-touch/wikipedia.png" rel="apple-touch-icon"/>
<link href="/static/favicon/wikipedia.ico" rel="shortcut icon"/>
<link href="/w/opensearch_desc.php" rel="search" title="Wikipedia (en)" type="application/opensearchdescription+xml"/>
<link href="//en.wikipedia.org/w/api.php?action=rsd" rel="EditURI" type="application/rsd+xml"/>
<link href="//creativecommons.org/licenses/by-sa/3.0/" rel="license"/>
<link href="//en.wikipedia.org/wiki/Gene_Disease_Database" rel="canonical"/>
<link href="//login.wikimedia.org" rel="dns-prefetch"/>
<link href="//meta.wikimedia.org" rel="dns-prefetch"/>
<!--[if lt IE 9]><script src="/w/load.php?lang=qqx&amp;modules=html5shiv&amp;only=scripts&amp;skin=fallback&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-Gene_Disease_Database rootpage-Gene_Disease_Database skin-vector action-view">
<div class="noprint" id="mw-page-base"></div>
<div class="noprint" id="mw-head-base"></div>
<div class="mw-body" id="content" role="main">

<div class="mw-body-content" id="siteNotice"><!-- CentralNotice --></div>
<div class="mw-indicators mw-body-content">
</div>
<h1 class="firstHeading" id="firstHeading" lang="en">Gene Disease Database</h1>
<div class="mw-body-content" id="bodyContent">
<div class="noprint" id="siteSub">From Wikipedia, the free encyclopedia</div>
<div id="contentSub"></div>
<div id="jump-to-nav"></div>
Jump to navigation
Jump to search
<div class="mw-content-ltr" dir="ltr" id="mw-content-text" lang="en"><div class="mw-parser-output"><div class="hatnote navigation-not-searchable" role="note">For Bioinformatics Databases, see List of biological databases.</div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Biological database</div>
<table class="infobox" style="width:22em"><caption>Gene Disease Database</caption><tbody><tr><th scope="row">Classification</th><td>Bioinformatics</td></tr><tr><th scope="row">Subclassification</th><td>Databases</td></tr><tr><th scope="row">Type of Databases</th><td>Biological</td></tr><tr><th scope="row">Subtype of Databases</th><td>Gene-Disease</td></tr></tbody></table>
<p class="mw-empty-elt">
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p><p>In Bioinformatics, a <b>Gene Disease Database</b> is a systematized collection of data, typically structured to model aspects of reality, in a way to comprehend the underlying mechanisms of complex diseases, by understanding multiple composite interactions between phenotype-genotype relationships and gene-disease mechanisms. Gene Disease Databases integrate human gene-disease associations from various expert curated databases and text-mining derived associations including Mendelian, complex and environmental diseases.
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p>
<div class="toc" id="toc"><input class="toctogglecheckbox" id="toctogglecheckbox" role="button" style="display:none" type="checkbox"/><div class="toctitle" dir="ltr" lang="en"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">Introduction</span>
<ul>
<li class="toclevel-2 tocsection-2"><span class="tocnumber">1.1</span> <span class="toctext">Challenges creating Gene Disease Databases</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-3"><span class="tocnumber">2</span> <span class="toctext">Types of Databases</span>
<ul>
<li class="toclevel-2 tocsection-4"><span class="tocnumber">2.1</span> <span class="toctext">Curated databases</span>
<ul>
<li class="toclevel-3 tocsection-5"><span class="tocnumber">2.1.1</span> <span class="toctext">The Comparative Toxicogenomics Database (CTD)</span></li>
<li class="toclevel-3 tocsection-6"><span class="tocnumber">2.1.2</span> <span class="toctext">The Universal Protein Resource (UNIPROT)</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-7"><span class="tocnumber">2.2</span> <span class="toctext">Predictive databases</span>
<ul>
<li class="toclevel-3 tocsection-8"><span class="tocnumber">2.2.1</span> <span class="toctext">The Mouse genome Database (MGD)</span></li>
<li class="toclevel-3 tocsection-9"><span class="tocnumber">2.2.2</span> <span class="toctext">The Rat Genome Database (RGD)</span></li>
<li class="toclevel-3 tocsection-10"><span class="tocnumber">2.2.3</span> <span class="toctext">The Online Mendelian Inheritance in Man (OMIM)</span></li>
<li class="toclevel-3 tocsection-11"><span class="tocnumber">2.2.4</span> <span class="toctext">Ensembl SIFT tool</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-12"><span class="tocnumber">2.3</span> <span class="toctext">Literature databases</span>
<ul>
<li class="toclevel-3 tocsection-13"><span class="tocnumber">2.3.1</span> <span class="toctext">Genetic Association Database (GAD)</span></li>
<li class="toclevel-3 tocsection-14"><span class="tocnumber">2.3.2</span> <span class="toctext">The literature-derived human gene-disease network (LHGDN)</span></li>
<li class="toclevel-3 tocsection-15"><span class="tocnumber">2.3.3</span> <span class="toctext">BeFree Data</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-16"><span class="tocnumber">2.4</span> <span class="toctext">Integrative databases</span>
<ul>
<li class="toclevel-3 tocsection-17"><span class="tocnumber">2.4.1</span> <span class="toctext">The Gene Disease Associations Database DisGeNET</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-18"><span class="tocnumber">3</span> <span class="toctext">Some use cases of Gene-Disease Databases</span></li>
<li class="toclevel-1 tocsection-19"><span class="tocnumber">4</span> <span class="toctext">Remarks about the future in Gene Disease Databases</span></li>
<li class="toclevel-1 tocsection-20"><span class="tocnumber">5</span> <span class="toctext">See also</span></li>
<li class="toclevel-1 tocsection-21"><span class="tocnumber">6</span> <span class="toctext">References</span></li>
</ul>
</div>
<h2><span class="mw-headline" id="Introduction">Introduction</span></h2>
<p>Experts in different areas of biology and bioinformatics have been trying to comprehend the molecular mechanisms of diseases to design preventive and therapeutic strategies for a long time. For some illnesses, it has become apparent that it is the right amount of animosity is made for  not enough to obtain an index of the disease-related genes but to uncover how disruptions of molecular grids in the cell give rise to disease phenotypes. Moreover, even with the unprecedented wealth of information available, obtaining such catalogues is extremely difficult.
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p><p>Genetic Broadly speaking, genetic diseases are caused by aberrations in genes or chromosomes. Many genetic diseases are developed from before birth. Genetic disorders account for a significant number of the health care problems in our society. Advances in the understanding of this diseases have increased both the life span and quality of life for many of those affected by genetic disorders. Recent developments in bioinformatics and laboratory genetics have made possible the better delineation of certain malformation and mental retardation syndromes, so that their mode of inheritance can be understood. This information enables the genetic counselor to predict the risk for occurrence of a large number of genetic disorders. Most genetic counseling is done, however, only after the birth of at least one affected individual has alerted the family to their predilection for having children with a genetic disorder. The association of a single gene to a disease is rare and a genetic disease may or may not be a transmissible disorder. Some genetic diseases are inherited from the parentâ€™s genes, but others are caused by new mutations or changes to the DNA. In other occurrences, the same disease, for instance, some forms of carcinoma or melanoma, may stem from an inbred condition in some people, from new changes in other people, and from non-genetic causes in still other individuals.
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p><p>There are more than six thousand known single-gene disorders (monogenic), which occur in about 1 out of every 200 births. As their term suggests, these diseases are caused by a mutation in one gene. By contrast, polygenic disorders are caused by several genes, regularly in combination with environmental factors. Examples of genetic phenotypes include Alzheimer's disease, breast cancer, leukemia, Down syndrome, heart defects, and deafness; therefore, cataloguing to sort out all the diseases related to genes is needed.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p>
<h3><span class="mw-headline" id="Challenges_creating_Gene_Disease_Databases">Challenges creating Gene Disease Databases</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:502px;"><img alt="A Gene prioritization" class="thumbimage" data-file-height="444" data-file-width="650" decoding="async" height="342" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e0/Geneprioritizationworkflow.jpg/500px-Geneprioritizationworkflow.jpg" srcset width="500"/> <a name="image1"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>1</h1></center></a></div><b>Gene prioritization workflow of human diseases</b>: Typical lists come from linkage regions, chromosomal aberrations, association study loci, deferentially expressed gene lists or genes identified by sequencing variants. Alternatively, the complete genome can be prioritized, but substantially more false positives would then be expected.</div></div></div>
<p>At different stages of any gene disease project, molecular biologists need to choose, even after careful statistical data analysis, which genes or proteins to investigate further experimentally and which to leave out because of limited resources. Computational methods that integrate complex, heterogeneous data sets, such as expression data, sequence information, functional annotation and the biomedical literature, allow prioritizing genes for future study in a more informed way. Such methods can substantially increase the yield of downstream studies and are becoming invaluable to researchers. So one of the main concerns in biological and biomedical research is to recognise the underlying mechanisms behind this intricate genetic phenotypes. Great effort has been spent on finding the genes related to diseases
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p><p>However, increasingly evidences point out that most human diseases cannot be attributed to a single gene but arise due to complex interactions among multiple genetic variants and environmental risk factors. Several databases have been developed storing associations between genes and diseases such as the Comparative Toxicogenomics Database (CTD), Online Mendelian Inheritance in Man (OMIM), the genetic Association Database (GAD) or the Disease genetic Association Database (DisGeNET). Each of these databases focuses on different aspects of the phenotype-genotype relationship, and due to the nature of the database curation process, they are not complete, but in a way they are fully complementary between each other.
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p>
<h2><span class="mw-headline" id="Types_of_Databases">Types of Databases</span></h2>
<p>Essentially, there are four types of databases: curated databases, predictive databases, literature databases and integrative databases
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p>
<h3><span class="mw-headline" id="Curated_databases">Curated databases</span></h3>
<p>The term curated data refers to information, that may comprise the most sophisticated computational formats for structured data, scientific updates, and curated knowledge, that has been composed and prepared under the regulation of one or more experts considered to be qualified to engage in such an activity The implication is that the resulting database is of high quality. The contrast is with data which may have been gathered through some automated process or using particularly low or inexpert unsupported data quality and possibly untrustworthy. Some of the most common examples include: CTD and UNIPROT.
<span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p>
<h4><span id="The_Comparative_Toxicogenomics_Database_.28CTD.29"></span><span class="mw-headline" id="The_Comparative_Toxicogenomics_Database_(CTD)">The Comparative Toxicogenomics Database (CTD)</span></h4>
<p>The Comparative Toxicogenomics Database, helps to understand about the effects of environmental compounds on human health by integrating data from curated scientific literature to describe biochemical interactions with genes and proteins, and links between diseases and chemicals, and diseases and genes or proteins. CTD contains curated data defining cross-species chemicalâ€“gene/protein interactions and chemicalâ€“ and geneâ€“disease associations to illuminate molecular mechanisms underlying variable susceptibility and environmentally influenced diseases. These data deliver insights into complex chemicalâ€“gene and protein interaction networks. One of the main sources in this Database is curated information from OMIM.
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p><p>CTD is a unique resource where bioinformatics specialists read the scientific literature and manually curate four types of core data:
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p>
<ul><li>Chemical-gene interactions</li>
<li>Chemical-disease associations</li>
<li>Gene-disease associations</li>
<li>Chemical-phenotype associations</li></ul>
<h4><span id="The_Universal_Protein_Resource_.28UNIPROT.29"></span><span class="mw-headline" id="The_Universal_Protein_Resource_(UNIPROT)">The Universal Protein Resource (UNIPROT)</span></h4>
<p>The Universal Protein Resource (UniProt) is an inclusive resource for protein sequence and annotation data. It is a comprehensive, first-class and freely accessible database of protein sequence and functional information, that has many entries being derived from genome sequencing projects. It contains a large amount of information about the biological function of proteins derived from the study literature, which can hint to a direct connection between gene-protein-disease.
<span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p>
<table class="infobox vevent" style="width:22em;width:"><caption class="summary">UniProt</caption><tbody><tr><th colspan="2" style="text-align:center;background-color: lavender">Content</th></tr><tr><th scope="row" style="white-space: nowrap">Description</th><td><b>UniProt</b> is the <i>universal protein</i> resource, a central repository of protein data created by combining the Swiss-Prot, TrEMBL and PIR-PSD databases.</td></tr><tr><th scope="row" style="white-space: nowrap">Data types<br/>captured</th><td>Protein annotation</td></tr><tr><th scope="row" style="white-space: nowrap">Organisms</th><td>All</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Contact</th></tr><tr><th scope="row" style="white-space: nowrap">Research center</th><td>EMBL-EBI, UK; SIB, Switzerland; PIR, US.</td></tr><tr><th scope="row" style="white-space: nowrap">Primary citation</th><td>Ongoing and future developments at the Universal Protein Resource</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Access</th></tr><tr><th scope="row" style="white-space: nowrap">Data format</th><td>Custom flat file, FASTA, GFF, RDF, XML.</td></tr><tr><th scope="row" style="white-space: nowrap">Website</th><td><span class="url">www<wbr/>.uniprot<wbr/>.org</span><br/><span class="url">www<wbr/>.uniprot<wbr/>.org<wbr/>/news<wbr/>/</span></td></tr><tr><th scope="row" style="white-space: nowrap">Download URL</th><td><span class="url">www<wbr/>.uniprot<wbr/>.org<wbr/>/downloads</span> &amp; for downloading complete data sets <span class="url">ftp<wbr/>.uniprot<wbr/>.org</span></td></tr><tr><th scope="row" style="white-space: nowrap">Web service URL</th><td>Yes â€“ JAVA API see info <span class="url">here</span> &amp; REST see info <span class="url">here</span></td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Tools</th></tr><tr><th scope="row" style="white-space: nowrap">Web</th><td>Advanced search, BLAST, ClustalO, bulk retrieval/download, ID mapping</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Miscellaneous</th></tr><tr><th scope="row" style="white-space: nowrap">License</th><td>Creative Commons Attribution-NoDerivs</td></tr><tr><th scope="row" style="white-space: nowrap">Versioning</th><td>Yes</td></tr><tr><th scope="row" style="white-space: nowrap">Data release<br/>frequency</th><td>4 weeks</td></tr><tr><th scope="row" style="white-space: nowrap">Curation policy</th><td>Yes â€“ manual and automatic. Rules for automatic annotation generated by database curators and computational algorithms.</td></tr><tr><th scope="row" style="white-space: nowrap">Bookmarkable<br/>entities</th><td>Yes â€“ both individual protein entries and searches</td></tr></tbody></table>
<div class="center"><div class="thumb tnone"><div class="thumbinner" style="width:602px;"><img alt="" class="thumbimage" data-file-height="527" data-file-width="899" decoding="async" height="352" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/19/Databasecompilation1.png/600px-Databasecompilation1.png" srcset width="600"/> <a name="image2"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>2</h1></center></a></div><b>The process of database compilation and curation</b><br/> The curated data may comprise a process from practical experience and literature review to web publication of the database</div></div></div></div>
<h3><span class="mw-headline" id="Predictive_databases">Predictive databases</span></h3>
<p>A predictive database is one based on statistical inference. One particular approach to such inference is known as predictive inference, but the prediction can be undertaken within any of the several approaches to statistical inference. Indeed, one description of biostatistics is that it provides a means of transferring knowledge about a sample of a genetic population to the whole population (genomics), and to other related genes or genomes, which the same as prediction over time is not necessarily. When information is transferred across time, often to specific points in time, the process is known as forecasting. Three of the main examples of databases that can be considered in this category include: The Mouse genome Database (MGD), The Rat genome Database (RGD), OMIM and the SIFT Tool from Ensembl.
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p>
<h4><span id="The_Mouse_genome_Database_.28MGD.29"></span><span class="mw-headline" id="The_Mouse_genome_Database_(MGD)">The Mouse genome Database (MGD)</span></h4>
<p>The Mouse genome Database (MGD) is the international community resource for integrated genetic, genomic and biological data about the laboratory mouse. MGD provides full annotation of phenotypes and human disease associations for mouse models (genotypes) using terms from the Mammalian Phenotype Ontology and disease names from OMIM.
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p>
<h4><span id="The_Rat_Genome_Database_.28RGD.29"></span><span class="mw-headline" id="The_Rat_Genome_Database_(RGD)">The Rat Genome Database (RGD)</span></h4>
<table class="infobox vevent" style="width:22em;width:"><caption class="summary">RGD</caption><tbody><tr><th colspan="2" style="text-align:center;background-color: lavender">Content</th></tr><tr><th scope="row" style="white-space: nowrap">Description</th><td>The <b>Rat Genome Database</b></td></tr><tr><th scope="row" style="white-space: nowrap">Organisms</th><td><i>Rattus norvegicus</i> (rat)</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Contact</th></tr><tr><th scope="row" style="white-space: nowrap">Research center</th><td>Medical College of Wisconsin</td></tr><tr><th scope="row" style="white-space: nowrap">Laboratory</th><td>Human Molecular and Genetics Center</td></tr><tr><th scope="row" style="white-space: nowrap">Authors</th><td>Mary E. Shimoyama, PhD; Howard J. Jacob, PhD</td></tr><tr><th scope="row" style="white-space: nowrap">Primary citation</th><td><style data-mw-deduplicate="TemplateStyles:r886058088">.mw-parser-output cite.citation{font-style:inherit}.mw-parser-output .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .citation .cs1-lock-free a{background:url("https://upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background:url("https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .citation .cs1-lock-subscription a{background:url("https://upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-parser-output .cs1-registration span{border-bottom:1px dotted;cursor:help}.mw-parser-output .cs1-ws-icon a{background:url("https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png")no-repeat;background-position:right .1em center}.mw-parser-output code.cs1-code{color:inherit;background:inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error{font-size:100%}.mw-parser-output .cs1-maint{display:none;color:#33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-wl-right{padding-right:0.2em}</style>PMIDÂ 25355511</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Access</th></tr><tr><th scope="row" style="white-space: nowrap">Website</th><td><span class="url">rgd<wbr/>.mcw<wbr/>.edu</span></td></tr><tr><th scope="row" style="white-space: nowrap">Download URL</th><td>RGD Data Release</td></tr></tbody></table>
<p>The Rat Genome Database (RGD) began as a collaborative effort between leading research institutions involved in rat genetic and genomic research. The rat continues to be extensively used by researchers as a model organism for investigating the biology and pathophysiology of disease. In the past several years, there has been a rapid increase in rat genetic and genomic data. This explosion of information highlighted the need for a centralized database to efficiently and effectively collect, manage, and distribute a rat-centric view of this data to researchers around the world. The Rat Genome Database was created to serve as a repository of rat genetic and genomic data, as well as mapping, strain, and physiological information. It also facilitates investigators research efforts by providing tools to search, mine, and predict this data.
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p><p>Data at RGD that is useful for researchers investigating disease genes include disease annotations for rat, mouse and human genes. Annotations are manually curated from the literature, or downloaded via automated pipelines from other disease-related databases. Downloaded annotations are mapped to the same disease vocabulary used for manual annotations to provide consistency across the dataset. RGD also maintains disease-related quantitative phenotype data for the rat (PhenoMiner).
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
<h4><span id="The_Online_Mendelian_Inheritance_in_Man_.28OMIM.29"></span><span class="mw-headline" id="The_Online_Mendelian_Inheritance_in_Man_(OMIM)">The Online Mendelian Inheritance in Man (OMIM)</span></h4>
<table class="infobox vevent" style="width:22em;width:"><caption class="summary">The Online Mendelian Inheritance in Man</caption><tbody><tr><th colspan="2" style="text-align:center;background-color: lavender">Content</th></tr><tr><th scope="row" style="white-space: nowrap">Description</th><td>OMIM is a compendium of human genes and genetic phenotypes.</td></tr><tr><th scope="row" style="white-space: nowrap">Organisms</th><td>Human (<i>H. Sapiens</i>)</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Contact</th></tr><tr><th scope="row" style="white-space: nowrap">Research center</th><td>NCBI</td></tr><tr><th scope="row" style="white-space: nowrap">Primary citation</th><td><link href="mw-data:TemplateStyles:r886058088" rel="mw-deduplicated-inline-style"/>PMIDÂ 25398906</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Access</th></tr><tr><th scope="row" style="white-space: nowrap">Website</th><td><span class="url">www<wbr/>.ncbi<wbr/>.nlm<wbr/>.nih<wbr/>.gov<wbr/>/omim</span></td></tr></tbody></table>
<p>Supported by the NCBI, The Online Mendelian Inheritance in Man (OMIM) is a database that catalogues all the known diseases with a genetic component, and predicts their relationship to relevant genes in the human genome and provides references for further research and tools for genomic analysis of a catalogued gene. OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes that is freely available and updated daily. The database has been used as a resource for predicting relevant information to inherited conditions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p>
<div class="thumb tleft"><div class="thumbinner" style="width:222px;"><img alt="A Pathway Hogeneity" class="thumbimage" data-file-height="975" data-file-width="1345" decoding="async" height="159" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/13/Pathwayhomogeneity1.png/220px-Pathwayhomogeneity1.png" srcset width="220"/> <a name="image3"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>3</h1></center></a></div><b>Pathway Hogeneity vs Associated Genes</b> Showing the concept that diseases have large association with a variety of genes, a mean pathway homogeneity values of single diseases and random controls are plotted for four networks binned by the number of associated gene products per disease. <i>This graph shows how difficult is to correlate a bigger number of diseases vs concordance in 4 different databases, hence Gene Disease Databases test these relationships</i></div></div></div>
<h4><span class="mw-headline" id="Ensembl_SIFT_tool">Ensembl SIFT tool</span></h4>
<table class="infobox vevent" style="width:22em;width:"><caption class="summary">The Ensembl genome database project.</caption><tbody><tr><th colspan="2" style="text-align:center;font-size:125%;font-weight:bold"><img alt="Ensembl release58 sgcb screenshot.png" data-file-height="439" data-file-width="844" decoding="async" height="156" src="https://upload.wikimedia.org/wikipedia/en/thumb/e/ef/Ensembl_release58_sgcb_screenshot.png/300px-Ensembl_release58_sgcb_screenshot.png" srcset width="300"/></th></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Content</th></tr><tr><th scope="row" style="white-space: nowrap">Description</th><td><b>Ensembl</b></td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Contact</th></tr><tr><th scope="row" style="white-space: nowrap">Research center</th><td><div class="plainlist"><ul><li>Wellcome Trust Sanger Institute</li><li>European Bioinformatics Institute</li></ul></div></td></tr><tr><th scope="row" style="white-space: nowrap">Primary citation</th><td>Hubbard, <i>et al.</i> (2002)</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Access</th></tr><tr><th scope="row" style="white-space: nowrap">Website</th><td><span class="url">www<wbr/>.ensembl<wbr/>.org</span></td></tr></tbody></table>
<p>This one of the largest resources available for all genomic and genetic studies, it provides a centralized resource for geneticists, molecular biologists and other researchers studying the genomes of our own species and other vertebrates and model disease organisms. Ensembl is one of several well-known genome browsers for the retrieval of genomic-disease information. Ensembl imports variation data from a variety of different sources, Ensembl predicts the effects of variants. For each variation that is mapped to the reference genome, each Ensembl transcript is identified that overlap the variation. Then it uses a rule-based approach to predict the effects that each allele of the variation may have on the transcript. The set of consequence terms, defined by the Sequence Ontology (SO) can be currently assigned to each combination of an allele and a transcript. Each allele of each variation may have a different effect in different transcripts. A variety of different tools are used to predict human mutations in the Ensembl database, one of the most widely used is SIFT, that predicts whether an amino acid substitution is likely to affect protein function based on sequence homology and the physic-chemical similarity between the alternate amino acids. The data provided for each amino acid substitution is a score and a qualitative prediction (either 'tolerated' or 'deleterious'). The score is the normalized probability that the amino acid change is tolerated so scores near 0 are more likely to be deleterious. The qualitative prediction is derived from this score such that substitutions with a score &lt; 0.05 are called 'deleterious' and all others are called 'tolerated'.SIFT can be applied to naturally occurring nonsynonymous polymorphisms and laboratory-induced missense mutations, that will lead to build relationships in phenotype characteristics, proteomics and genomics
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p>
<h3><span class="mw-headline" id="Literature_databases">Literature databases</span></h3>
<p>This sort of databases summarize books, articles, book reviews, dissertations, and annotations about gene-disease databases. Some of the following are examples of this type: GAD, LGHDN and BeFree Data.
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
<h4><span id="Genetic_Association_Database_.28GAD.29"></span><span class="mw-headline" id="Genetic_Association_Database_(GAD)">Genetic Association Database (GAD)</span></h4>
<p>The Genetic Association Database is an archive of human genetic association studies of complex diseases. GAD is primarily focused on archiving information on common complex human disease rather than rare Mendelian disorders as found in the OMIM. It includes curated summary data extracted from published papers in peer reviewed journals on candidate gene and genome Wide Association Studies (GWAS). The GAD was frozen as of 09/01/2014 but is still available for download.
<span style="border: 3px solid blue; color:blue;font-weight: bold">20</span></p>
<h4><span id="The_literature-derived_human_gene-disease_network_.28LHGDN.29"></span><span class="mw-headline" id="The_literature-derived_human_gene-disease_network_(LHGDN)">The literature-derived human gene-disease network (LHGDN)</span></h4>
<p>The literature-derived human gene-disease network (LHGDN) is a text mining derived database with focus on extracting and classifying gene-disease associations with respect to several biomolecular conditions. It uses a machine learning based algorithm to extract semantic gene-disease relations from a textual source of interest. It is part of the Linked Life Data, of the LMU in Munchen, Germany.
<span style="border: 3px solid blue; color:blue;font-weight: bold">21</span></p>
<h4><span class="mw-headline" id="BeFree_Data">BeFree Data</span></h4>
<p>Extracts gene-disease associations from MEDLINE abstract using the BeFree system. BeFree is composed of a biomedical Named Entity Recognition (BioNER) module to detect diseases and genes and a relation extraction module based on morphosyntactic information.
<span style="border: 3px solid blue; color:blue;font-weight: bold">22</span></p>
<h3><span class="mw-headline" id="Integrative_databases">Integrative databases</span></h3>
<p>This sort of databases include Mendelian, compound and environmental diseases in an integrated gene-disease association archive and show that the concept of modularity applies for all of them They provide a functional analysis of diseases in case of important new biological insights, which might not be discovered when considering each of the gene-disease associations independently. Hence, they present a suitable framework for the study of how genetic and environmental factors, such as drugs, contribute to diseases. The best example for this sort of database is DisGeNET.
<span style="border: 3px solid blue; color:blue;font-weight: bold">23</span></p>
<h4><span class="mw-headline" id="The_Gene_Disease_Associations_Database_DisGeNET">The Gene Disease Associations Database DisGeNET</span></h4>
<table class="infobox vevent" style="width:22em;width:"><caption class="summary">DisGeNET</caption><tbody><tr><th colspan="2" style="text-align:center;background-color: lavender">Content</th></tr><tr><th scope="row" style="white-space: nowrap">Description</th><td>Integrates human gene-disease associations</td></tr><tr><th scope="row" style="white-space: nowrap">Data types<br/>captured</th><td>Associations Database</td></tr><tr><th scope="row" style="white-space: nowrap">Organisms</th><td>Human (<i>H. Sapiens</i>)</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Contact</th></tr><tr><th scope="row" style="white-space: nowrap">Research center</th><td>Research Programme on Biomedical Informatics (GRIB) IMIM-UPF</td></tr><tr><th scope="row" style="white-space: nowrap">Laboratory</th><td>Integrative Biomedical Informatics Group</td></tr><tr><th scope="row" style="white-space: nowrap">Authors</th><td>Ferran Sanz and Laura I. Furlong (Pinero et al, 2015)</td></tr><tr><th scope="row" style="white-space: nowrap">Primary citation</th><td><link href="mw-data:TemplateStyles:r886058088" rel="mw-deduplicated-inline-style"/>PMIDÂ 25877637</td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Access</th></tr><tr><th scope="row" style="white-space: nowrap">Website</th><td><span class="url">www<wbr/>.disgenet<wbr/>.org</span></td></tr><tr><th colspan="2" style="text-align:center;background-color: lavender">Miscellaneous</th></tr><tr><th scope="row" style="white-space: nowrap">Data release<br/>frequency</th><td>annual</td></tr><tr><th scope="row" style="white-space: nowrap">Version</th><td>3</td></tr></tbody></table>
<p>DisGeNET is a comprehensive gene-disease association database that integrates associations from several sources that covers different biomedical aspects of diseases. In particular, it is focused on the current knowledge of human genetic diseases including Mendelian, complex and environmental diseases. To assess the concept of modularity of human diseases, this database performs a systematic study of the emergent properties of human gene-disease networks by means of network topology and functional annotation analysis. The results indicate a highly shared genetic origin of human diseases and show that for most diseases, including Mendelian, complex and environmental diseases, functional modules exist. Moreover, a core set of biological pathways is found to be associated with most human diseases. Obtaining similar results when studying clusters of diseases, the findings in this database suggest that related diseases might arise due to dysfunction of common biological processes in the cell. The network analysis of this integrated database points out that data integration is needed to obtain a comprehensive view of the genetic landscape of human diseases and that the genetic origin of complex diseases is much more common than expected.
<span style="border: 3px solid blue; color:blue;font-weight: bold">24</span></p>
<div class="center"><div class="thumb tnone"><div class="thumbinner" style="width:502px;"><img alt="" class="thumbimage" data-file-height="889" data-file-width="1352" decoding="async" height="329" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Disgenet.png/500px-Disgenet.png" srcset width="500"/> <a name="image4"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>4</h1></center></a></div><b>DisGeNET gene-disease association ontology </b><br/> The description of each association type in this ontology is: #Therapeutic Association: The gene/protein has a therapeutic role in the amelioration of the disease. #Biomarker Association: The gene/protein either plays a role in the etiology of the disease (e.g. participates in the molecular mechanism that leads to disease) or is a biomarker for a disease. #Genetic Variation Association: Used when a sequence variation (a mutation, a SNP) is associated to the disease phenotype, but there is still no evidence to say that the variation causes the disease. In some cases the presence of the variants increase the susceptibility to the disease. In general, the NCBI SNP identifiers are provided. #Altered Expression Association: Alterations in the function of the protein by means of altered expression of the gene are associated with the disease phenotype. #Post-translational Modification Association: Alterations in the function of the protein by means of post-translational modifications (methylation or phosphorylation of the protein) are associated with the disease phenotype. </div></div></div></div>
<h2><span class="mw-headline" id="Some_use_cases_of_Gene-Disease_Databases">Some use cases of Gene-Disease Databases</span></h2>
<p>Some of the most interesting cases using Gene-Disease Databases can be found in the following papers:
<span style="border: 3px solid blue; color:blue;font-weight: bold">25</span></p>
<ul><li>A network approach to clinical intervention in neurodegenerative diseases. Santiago, JA, Potashkin JA. Trends in Molecular Medicine (2014) doi:10.1016/j.molmed.2014.10.002</li>
<li>Control of VEGF-A transcriptional programs by pausing and genomic compartmentalization. Kaikkonen MU, Niskanen H, Romanoski CE, Kansanen E, KivelÃ¤ AM, Laitalainen J, Heinz S, Benner C, Glass CK, YlÃ¤-Herttuala S. Nucleic Acids Res. (2014) doi:10.1093/nar/gku1036</li>
<li>Network medicine analysis of COPD multimorbidities. Grosdidier S, Ferrer A, Faner R, PiÃ±ero J, Roca J, CosÃ­o B, AgustÃ­ A, Gea J, Sanz F, Furlong LI. Respir Res. (2014) doi: 10.1186/s12931-014-0111-4</li>
<li>An R-based tool for miRNA data analysis and correlation with clinical ontologies. Cristiano F, Veltri P. Proceedings of the 5th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (2014) doi:10.1145/2649387.2660847</li>
<li>Using 2-node hypergraph clustering coefficients to analyze disease-gene networks. Renick Gallagher S, Dombrower M, Goldberg DS. Proceedings of the 5th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (2014) doi:10.1145/2649387.2660817</li>
<li>Organ system heterogeneity DB: a database for the visualization of phenotypes at the organ system level. Mannil D, Vogt I, Prinz J, Campillos M. Nucleic Acids Res. (2014) doi:10.1093/nar/gku948</li>
<li>Molecularly and clinically related drugs and diseases are enriched in phenotypically similar drug-disease pairs. Vogt I, Prinz J, Campillos M. Genome Med. (2014) doi: 10.1186/s13073-014-0052-z</li>
<li>System-based approaches to decode the molecular links in Parkinson's disease and diabetes. Santiago, JA, Potashkin JA. Neurobiol Dis. (2014) doi:10.1016/j.nbd.2014.03.019</li>
<li>Prioritizing Diseaseâ€Linked Variants, Genes, and Pathways with an Interactive whole Genome Analysis Pipeline. Lee IH, Lee K, Hsing M, Choe Y, Park JH, Kim SH, Bohn JM, Neu MB, Hwang KB, Green RC, Kohane IS, Kong SW. Hum Mutat. (2014) doi:10.1002/humu.22520</li>
<li>A Computational Framework to Infer Human Disease-Associated Long Noncoding RNAs. Liu MX, Chen X, Chen G, Cui QH, Yan GY. PLOS ONE 9.1 (2014) doi:10.1371/journal.pone.0084408</li>
<li>Choline protects against cardiac hypertrophy induced by increased after-load. Zhao Y, Wang C, Wu J, Wang Y, Zhu W, Zhang Y, Du Z. Int J Biol Sci. (2013) doi:10.7150/ijbs.5976</li>
<li>Detection of differentially methylated gene promoters in failing and nonfailing human left ventricle myocardium using computation analysis. Koczor CA, Lee EK, Torres RA, Boyd A, Vega JD, Uppal K, Yuan F, Fields EJ, Samarel AM, Lewis W. Physiol Genomics. (2013) doi:10.1152/physiolgenomics.00013.2013</li>
<li>Global DNA methylation and transcriptional analyses of human ESC-derived cardiomyocytes. Gu Y, Liu GH, Plongthongkum N, Benner C, Yi F, Qu J, Suzuki K, Yang J, Zhang W, Li M, Montserrat N, Crespo I, Del Sol A, Esteban CR, Zhang K, Belmonte JC. Protein Cell. (2013) doi:10.1007/s13238-013-0016-x</li>
<li>Integrated analysis of transcript-level regulation of metabolism reveals disease-relevant nodes of the human metabolic network. Galhardo M1, Sinkkonen L, Berninger P, Lin J, Sauter T, HeinÃ¤niemi M. Nucleic Acids Res. (2013) doi:10.1093/nar/gkt989</li>
<li>Charting the NF-ÎºB Pathway Interactome Map. Tieri P1, Termanini A, Bellavista E, Salvioli S, Capri M, Franceschi C. PLOS ONE (2012) doi:10.1371/journal.pone.0032678</li></ul>
<h2><span class="mw-headline" id="Remarks_about_the_future_in_Gene_Disease_Databases">Remarks about the future in Gene Disease Databases</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" class="thumbimage" data-file-height="775" data-file-width="800" decoding="async" height="213" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/10/Relationships_in_Gene_Diseases.jpg/220px-Relationships_in_Gene_Diseases.jpg" srcset width="220"/> <a name="image5"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>5</h1></center></a></div><b>Relationships in Gene Diseases</b></div></div></div>
<p>The completion of the human genome has changed the way the search for disease genes is performed. In the past, the approach was to focus on one or a few genes at a time. Now, projects like the DisGeNET exemplify the efforts to systematically analyze all the gene alterations involved in a single or multiple diseases. The next step is to produce a complete picture of the mechanistic aspects of the diseases and the design of drugs against them. For that, a combination of two approaches will be needed: a systematic search and in-depth study of each gene.
The future of the field will be defined by new techniques to integrate large bodies of data from different sources and to incorporate functional information into the analysis of large-scale data generated by bioinformatics studies.
<span style="border: 3px solid blue; color:blue;font-weight: bold">26</span></p><p>Bioinformatics is both a term for the body of biological gene disease studies that use computer programming as part of their methodology, as well as a reference to specific analysis pipelines that are repeatedly used, particularly in the fields of genetics and genomics. Common uses of bioinformatics include the identification of candidate genes and nucleotides, SNPs. Often, such identification is made with the aim of better understanding the genetic basis of disease, unique adaptations, desirable properties, or differences between populations. In a less formal way, bioinformatics also tries to understand the organisational principles within nucleic acid and protein sequences.
<span style="border: 3px solid blue; color:blue;font-weight: bold">27</span></p><p>The response of bioinformatics to new experimental techniques brings a new perspective into the analysis of the experimental data, as demonstrated by the advances in the analysis of information from gene disease databases and other technologies. It is expected that this trend will continue with novel approaches to respond to new techniques, such as next-generation sequencing technologies. For instance, the availability of large numbers of individual human genomes will promote the development of computational analyses of rare variants, including the statistical mining of their relations to lifestyles, drug interactions and other factors.
Biomedical research will also be driven by our ability to efficiently mine the large body of existing and continuously generated biomedical data. Text-mining techniques, in particular, when combined with other molecular data, can provide information about gene mutations and interactions and will become crucial to stay ahead of the exponential growth of data generated in biomedical research. Another field that is benefiting from the advances in mining and integration of molecular, clinical and drug analysis is pharmacogenomics. <i>In silico</i> studies of the relationships between human variations and their effect on diseases will be key to the development of personalized medicine.
In summary, Gene Disease Databases have already transformed the search for disease genes and has the potential to become a crucial component of other areas of medical research.
<span style="border: 3px solid blue; color:blue;font-weight: bold">28</span></p>
<script>function offsetAnchor() {if(location.hash.length !== 0) {window.scrollTo(window.scrollX, window.scrollY - 250);}}window.addEventListener("hashchange", offsetAnchor);window.setTimeout(offsetAnchor, 1);</script>