<!DOCTYPE html>

<html class="client-nojs" dir="ltr" lang="en">
<head>
<meta charset="utf-8"/>
<title>Glucose-6-phosphate dehydrogenase - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Glucose-6-phosphate_dehydrogenase","wgTitle":"Glucose-6-phosphate dehydrogenase","wgCurRevisionId":852384155,"wgRevisionId":852384155,"wgArticleId":3064920,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1: long volume value","SWL/mutation results in","SWL","CS1 French-language sources (fr)","EC 1.1.1","NADPH-dependent enzymes","Enzymes of known structure"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Glucose-6-phosphate_dehydrogenase","wgRelevantArticleId":3064920,"wgRequestId":"XL8uSwpAADkAAAkG1L4AAAAL","wgCSPNonce":false,"wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsReferencePreviews":false,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgWMESchemaEditAttemptStepOversample":false,"wgPoweredByHHVM":true,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q14877634","wgCentralAuthMobileDomain":false,"wgEditSubmitButtonLabelPublish":true});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.page.gallery.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});RLPAGEMODULES=["ext.cite.ux-enhancements","site","mediawiki.page.startup","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];mw.loader.load(RLPAGEMODULES);});</script>
<link href="/w/load.php?lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.page.gallery.styles%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<script async="" src="/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta content="" name="ResourceLoaderDynamicStyles"/>
<link href="/w/load.php?lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<link href="/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<meta content="MediaWiki 1.34.0-wmf.1" name="generator"/>
<meta content="origin" name="referrer"/>
<meta content="origin-when-crossorigin" name="referrer"/>
<meta content="origin-when-cross-origin" name="referrer"/>
<meta content="https://upload.wikimedia.org/wikipedia/commons/thumb/5/50/G6PD_sites_labeled.png/1200px-G6PD_sites_labeled.png" property="og:image"/>
<link href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Glucose-6-phosphate_dehydrogenase" rel="alternate"/>
<link href="/w/index.php?title=Glucose-6-phosphate_dehydrogenase&amp;action=edit" rel="alternate" title="Edit this page" type="application/x-wiki"/>
<link href="/w/index.php?title=Glucose-6-phosphate_dehydrogenase&amp;action=edit" rel="edit" title="Edit this page"/>
<link href="/static/apple-touch/wikipedia.png" rel="apple-touch-icon"/>
<link href="/static/favicon/wikipedia.ico" rel="shortcut icon"/>
<link href="/w/opensearch_desc.php" rel="search" title="Wikipedia (en)" type="application/opensearchdescription+xml"/>
<link href="//en.wikipedia.org/w/api.php?action=rsd" rel="EditURI" type="application/rsd+xml"/>
<link href="//creativecommons.org/licenses/by-sa/3.0/" rel="license"/>
<link href="//en.wikipedia.org/wiki/Glucose-6-phosphate_dehydrogenase" rel="canonical"/>
<link href="//login.wikimedia.org" rel="dns-prefetch"/>
<link href="//meta.wikimedia.org" rel="dns-prefetch"/>
<!--[if lt IE 9]><script src="/w/load.php?lang=qqx&amp;modules=html5shiv&amp;only=scripts&amp;skin=fallback&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-Glucose-6-phosphate_dehydrogenase rootpage-Glucose-6-phosphate_dehydrogenase skin-vector action-view">
<div class="noprint" id="mw-page-base"></div>
<div class="noprint" id="mw-head-base"></div>
<div class="mw-body" id="content" role="main">

<div class="mw-body-content" id="siteNotice"><!-- CentralNotice --></div>
<div class="mw-indicators mw-body-content">
</div>
<h1 class="firstHeading" id="firstHeading" lang="en">Glucose-6-phosphate dehydrogenase</h1>
<div class="mw-body-content" id="bodyContent">
<div class="noprint" id="siteSub">From Wikipedia, the free encyclopedia</div>
<div id="contentSub"></div>
<div id="jump-to-nav"></div>
Jump to navigation
Jump to search
<div class="mw-content-ltr" dir="ltr" id="mw-content-text" lang="en"><div class="mw-parser-output"><div class="hatnote navigation-not-searchable" role="note">Not to be confused with Glucose 6-phosphatase.</div>
<table class="infobox" style="width:22em"><tbody><tr><th colspan="2" style="text-align:center;font-size:125%;font-weight:bold">Glucose-6-phosphate dehydrogenase, NAD binding domain</th></tr><tr><td colspan="2" style="text-align:center"><img alt="G6PD sites labeled.png" data-file-height="1693" data-file-width="2891" decoding="async" height="129" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/50/G6PD_sites_labeled.png/220px-G6PD_sites_labeled.png" srcset width="220"/><div>glucose 6-phosphate dehydrogenase from leuconostoc mesenteroides</div></td></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Identifiers</th></tr><tr><th scope="row" style="background-color: #e7dcc3">Symbol</th><td style="background-color: #eee">G6PD_N</td></tr><tr><th scope="row" style="background-color: #e7dcc3">Pfam</th><td class="pfam" style="background-color: #eee">PF00479</td></tr><tr><th scope="row" style="background-color: #e7dcc3">Pfam clan</th><td style="background-color: #eee">CL0063</td></tr><tr><th scope="row" style="background-color: #e7dcc3">InterPro</th><td style="background-color: #eee">IPR022674</td></tr><tr><th scope="row" style="background-color: #e7dcc3">PROSITE</th><td style="background-color: #eee">PDOC00067</td></tr><tr><th scope="row" style="background-color: #e7dcc3">SCOPe</th><td style="background-color: #eee">1dpg</td></tr><tr><th scope="row" style="background-color: #e7dcc3">SUPERFAMILY</th><td style="background-color: #eee">1dpg</td></tr><tr><td colspan="2" style="text-align:center;background-color: #eee"><table class="infobox collapsible collapsed" style="width:22em;float:none; clear:none; margin:0; border-width:0; border-collapse:collapse; text-align:left; width:100%"><tbody><tr><th colspan="2" style="text-align:center;background-color: #ddd">Available protein structures:</th></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">Pfam</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">structures</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PDB</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">RCSB PDB; PDBe; PDBj</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PDBsum</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">structure summary</td></tr></tbody></table></td></tr></tbody></table><table class="infobox" style="width:22em"><tbody><tr><th colspan="2" style="text-align:center;font-size:125%;font-weight:bold">Glucose-6-phosphate dehydrogenase</th></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Identifiers</th></tr><tr><th scope="row" style="background-color: #e7dcc3">EC number</th><td style="background-color: #eee">1.1.1.49</td></tr><tr><th scope="row" style="background-color: #e7dcc3">CAS number</th><td style="background-color: #eee">9001-40-5 </td></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Databases</th></tr><tr><th scope="row" style="background-color: #e7dcc3">IntEnz</th><td style="background-color: #eee">IntEnz view</td></tr><tr><th scope="row" style="background-color: #e7dcc3">BRENDA</th><td style="background-color: #eee">BRENDA entry</td></tr><tr><th scope="row" style="background-color: #e7dcc3">ExPASy</th><td style="background-color: #eee">NiceZyme view</td></tr><tr><th scope="row" style="background-color: #e7dcc3">KEGG</th><td style="background-color: #eee">KEGG entry</td></tr><tr><th scope="row" style="background-color: #e7dcc3">MetaCyc</th><td style="background-color: #eee">metabolic pathway</td></tr><tr><th scope="row" style="background-color: #e7dcc3">PRIAM</th><td style="background-color: #eee">profile</td></tr><tr><th scope="row" style="background-color: #e7dcc3">PDB structures</th><td style="background-color: #eee"><span style="font-size:85%;">RCSB PDB PDBe PDBsum</span></td></tr><tr><th scope="row" style="background-color: #e7dcc3">Gene Ontology</th><td style="background-color: #eee"><span style="font-size:85%;">AmiGO  / QuickGO</span></td></tr><tr><td colspan="2" style="text-align:center;background-color: #eee"><table class="infobox collapsible collapsed" style="width:22em;float:none; clear:none; margin:0; border-width:0; border-collapse:collapse; text-align:left; width:100%"><tbody><tr><th colspan="2" style="text-align:center;background-color: #ddd">Search</th></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PMC</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">articles</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PubMed</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">articles</td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">NCBI</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;">proteins</td></tr></tbody></table></td></tr></tbody></table><p><b>Glucose-6-phosphate dehydrogenase</b> (<b>G6PD</b> or <b>G6PDH</b>) (EC 1.1.1.49) is a cytosolic enzyme that catalyzes the chemical reaction
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p><dl><dd>D-glucose 6-phosphate + NADP  ⇌ 6-phospho-D-glucono-1,5-lactone + NADPH + H</dd></dl>
<p>This enzyme participates in the pentose phosphate pathway (see image), a metabolic pathway that supplies reducing energy to cells (such as erythrocytes) by maintaining the level of the co-enzyme nicotinamide adenine dinucleotide phosphate (NADPH). The NADPH in turn maintains the level of glutathione in these cells that helps protect the red blood cells against oxidative damage from compounds like hydrogen peroxide. Of greater quantitative importance is the production of NADPH for tissues involved in biosynthesis of fatty acids or isoprenoids, such as the liver, mammary glands, adipose tissue, and the adrenal glands. G6PD reduces NADP to NADPH while oxidizing glucose-6-phosphate.
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p><p>Clinically, an X-linked genetic deficiency of G6PD predisposes a person to non-immune <span class="swl"> <span class="mutation"> <span style="border-bottom:1px dotted orange" title="Glucose-6-phosphate dehydrogenase mutation results in hemolytic anemia">hemolytic anemia</span></span></span>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p>
<div class="toc" id="toc"><input class="toctogglecheckbox" id="toctogglecheckbox" role="button" style="display:none" type="checkbox"/><div class="toctitle" dir="ltr" lang="en"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">Species distribution</span></li>
<li class="toclevel-1 tocsection-2"><span class="tocnumber">2</span> <span class="toctext">Enzyme structure</span></li>
<li class="toclevel-1 tocsection-3"><span class="tocnumber">3</span> <span class="toctext">NADP structural site</span></li>
<li class="toclevel-1 tocsection-4"><span class="tocnumber">4</span> <span class="toctext">Regulation</span></li>
<li class="toclevel-1 tocsection-5"><span class="tocnumber">5</span> <span class="toctext">Clinical significance</span></li>
<li class="toclevel-1 tocsection-6"><span class="tocnumber">6</span> <span class="toctext">See also</span></li>
<li class="toclevel-1 tocsection-7"><span class="tocnumber">7</span> <span class="toctext">References</span></li>
<li class="toclevel-1 tocsection-8"><span class="tocnumber">8</span> <span class="toctext">Further reading</span></li>
<li class="toclevel-1 tocsection-9"><span class="tocnumber">9</span> <span class="toctext">External links</span></li>
</ul>
</div>
<h2><span class="mw-headline" id="Species_distribution">Species distribution</span></h2>
<p>G6PD is widely distributed in many species from bacteria to humans. Multiple sequence alignment of over 100 known G6PDs from different organisms reveal sequence identity ranging from 30% to 94%. Human G6PD has over 30% identity in amino acid sequence to G6PD sequences from other species. Humans also have two isoforms of a single gene coding for G6PD. Moreover, 150 different human G6PD mutants have been documented. These mutations are mainly missense mutations that result in amino acid substitutions, and while some of them result in G6PD deficiency, others do not seem to result in any noticeable functional differences. Some scientists have proposed that some of the genetic variation in human G6PD resulted from generations of adaptation to malarial infection.
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p><p>Other species experience a variation in G6PD as well. In higher plants, several isoforms of G6PDH have been reported, which are localized in the cytosol, the plastidic stroma, and peroxisomes. A modified F<sub>420</sub>-dependent (as opposed to NADP-dependent) G6PD is found in <i>Mycobacterium tuberculosis</i>, and is of interest for treating tuberculosis. The bacterial G6PD found in <i>Leuconostoc mesenteroides</i> was shown to be reactive toward 4-Hydroxynonenal, in addition to G6P.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p>
<h2><span class="mw-headline" id="Enzyme_structure">Enzyme structure</span></h2>
<div class="thumb tleft"><div class="thumbinner" style="width:302px;"><img alt="" class="thumbimage" data-file-height="1438" data-file-width="1437" decoding="async" height="300" src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2d/G6PD_active_site_labeled_2.png/300px-G6PD_active_site_labeled_2.png" srcset width="300"/> <a name="image1"><div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>1</h1></center></a></div>Substrate binding site of G6PD bound to G6P (shown in cream), from 2BHL. Phosphorus is shown in orange. Oxygen atoms of crystallographic waters are shown as red spheres. The conserved 9-peptide sequence of G6PD, and the partially conserved 5-residue sequence of G6PD are shown in cyan and magenta respectively. All other amino acids from G6PD are shown in black. Hydrogen bonding and electrostatic interactions are shown by green dashed lines. All green dashes represent distances of less than 3.7 Å.</div></div></div>
<p>G6PD is generally found as a dimer of two identical monomers (see main thumbnail). Depending on conditions, such as pH, these dimers can themselves dimerize to form tetramers. Each monomer in the complex has a substrate binding site that binds to G6P, and a catalytic coenzyme binding site that binds to NADP/NADPH using the Rossman fold. For some higher organisms, such as humans, G6PD contains an additional NADP binding site, called the NADP structural site, that does not seem to participate directly in the reaction catalyzed by G6PD. The evolutionary purpose of the NADP structural site is unknown. As for size, each monomer is approximately 500 amino acids long (514 amino acids for humans).
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p><p>Functional and structural conservation between human G6PD and <i>Leuconostoc mesenteroides</i> G6PD points to 3 widely conserved regions on the enzyme: a 9 residue peptide in the substrate binding site, RIDHYLGKE (residues 198-206 on human G6PD), a nucleotide-binding fingerprint, GxxGDLA (residues 38-44 on human G6PD), and a partially conserved sequence EKPxG near the substrate binding site (residues 170-174 on human G6PD), where we have use "x" to denote a variable amino acid. The crystal structure of G6PD reveals an extensive network of electrostatic interactions and hydrogen bonding involving G6P, 3 water molecules, 3 lysines, 1 arginine, 2 histidines, 2 glutamic acids, and other polar amino acids.
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p><p>The proline at position 172 is thought to play a crucial role in positioning Lys171 correctly with respect to the substrate, G6P. In the two crystal structures of normal human G6P, Pro172 is seen exclusively in the cis confirmation, while in the crystal structure of one disease causing mutant (variant Canton R459L), Pro172 is seen almost exclusively in the trans confirmation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p><p>
With access to crystal structures, some scientists have tried to model the structures of other mutants. For example, in German ancestry, where enzymopathy due to G6PD deficiency is rare, mutation sites on G6PD have been shown to lie near the NADP binding site, the G6P binding site, and near the interface between the two monomers. Thus, mutations in these critical areas are possible without completely disrupting the function of G6PD. In fact, it has been shown that most disease causing mutations of G6PD occur near the NADP structural site. <span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p><div style="clear:both;"></div>
<h2><span id="NADP.2B_structural_site"></span><span class="mw-headline" id="NADP+_structural_site">NADP structural site</span></h2>
<p>The NADP structural site is located greater than 20Å away from the substrate binding site and the catalytic coenzyme NADP binding site. Its purpose in the enzyme catalyzed reaction has been unclear for many years. For some time, it was thought that NADP binding to the structural site was necessary for dimerization of the enzyme monomers. However, this was shown to be incorrect. On the other hand, it was shown that the presence of NADP at the structural site promotes the dimerization of dimers to form enzyme tetramers. It was also thought that the tetramer state was necessary for catalytic activity; however, this too was shown to be false. The NADP structural site is quite different from the NADP catalytic coenzyme binding site, and contains the nucleotide-binding fingerprint.
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p><p>The structural site bound to NADP possesses favorable interactions that keep it tightly bound. In particular, there is a strong network of hydrogen bonding with electrostatic charges being diffused across multiple atoms through hydrogen bonding with 4 water molecules (see figure). Moreover, there is an extremely strong set of hydrophobic stacking interactions that result in overlapping π systems.   
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p>
<style data-mw-deduplicate="TemplateStyles:r889293853/mw-parser-output/.tmulti">.mw-parser-output .tmulti .thumbinner{display:flex;flex-direction:column}.mw-parser-output .tmulti .trow{display:flex;flex-direction:row;clear:left;flex-wrap:wrap;width:100%;box-sizing:border-box}.mw-parser-output .tmulti .tsingle{margin:1px;float:left}.mw-parser-output .tmulti .theader{clear:both;font-weight:bold;text-align:center;align-self:center;background-color:transparent;width:100%}.mw-parser-output .tmulti .thumbcaption{text-align:left;background-color:transparent}.mw-parser-output .tmulti .text-align-left{text-align:left}.mw-parser-output .tmulti .text-align-right{text-align:right}.mw-parser-output .tmulti .text-align-center{text-align:center}@media all and (max-width:720px){.mw-parser-output .tmulti .thumbinner{width:100%!important;box-sizing:border-box;max-width:none!important;align-items:center}.mw-parser-output .tmulti .trow{justify-content:center}.mw-parser-output .tmulti .tsingle{float:none!important;max-width:100%!important;box-sizing:border-box;text-align:center}.mw-parser-output .tmulti .thumbcaption{text-align:center}}</style><div class="thumb tmulti tnone center"><div class="thumbinner" style="width:708px;max-width:708px"><div class="trow"><div class="tsingle" style="width:352px;max-width:352px"><div class="thumbimage"><img alt="Hydrogen bonding and electrostatic interaction network. All green dashes represent distances less than 3.8 Å" data-file-height="1598" data-file-width="1597" decoding="async" height="350" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/G6PD_structural_site_hydrogen.png/350px-G6PD_structural_site_hydrogen.png" srcset width="350"/></div><div class="thumbcaption text-align-center">Hydrogen bonding and electrostatic interaction network (green). All green dashes represent distances less than 3.8 Å</div></div><div class="tsingle" style="width:352px;max-width:352px"><div class="thumbimage"><img alt="Hydrophobic stacking interactions (green). All green dashes represent distances less than 4.4 Å." data-file-height="1598" data-file-width="1597" decoding="async" height="350" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/15/G6PD_Structural_site_hydrophobic.png/350px-G6PD_Structural_site_hydrophobic.png" srcset width="350"/></div><div class="thumbcaption text-align-center">Hydrophobic stacking interactions (green). All green dashes represent distances less than 4.4 Å. Slightly different view than the first panel.</div></div></div><div class="trow"><div class="thumbcaption" style="text-align:left;background-color:transparent">NADP structural site of G6PD. NADP is shown in cream. Phosphorus is shown in orange. The oxygen atoms of crystallographic water molecules are shown as red spheres. The conserved 9-peptide sequence of G6PD is show in cyan.</div></div></div></div><p>The structural site has been shown to be important for maintaining the long term stability of the enzyme. More than 40 severe class I mutations involve mutations near the structural site, thus affecting the long term stability of these enzymes in the body, ultimately resulting in G6PD deficiency. For example, two severe class I mutations, G488S and G488V, drastically increase the dissociation constant between NADP and the structural site by a factor of 7 to 13. With the proximity of residue 488 to Arg487, it is thought that a mutation at position 488 could affect the positioning of Arg487 relative to NADP, and thus disrupt binding. <span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p><div style="clear:both;"></div>
<h2><span class="mw-headline" id="Regulation">Regulation</span></h2>
<p>G6PD converts G6P into 6-phosphoglucono-δ-lactone and is the rate-limiting enzyme of the <i>pentose phosphate pathway</i>. Thus, regulation of G6PD has downstream consequences for the activity of the rest of the <i>pentose phosphate pathway</i>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p><p>Glucose-6-phosphate dehydrogenase is stimulated by its substrate G6P. The usual ratio of NADPH/NADP in the cytosol of tissues engaged in biosyntheses is about 100/1. Increased utilization of NADPH for fatty acid biosynthesis will dramatically increase the level of NADP, thus stimulating G6PD to produce more NADPH. Yeast G6PD is inhibited by long chain fatty acids according to two older publications and might be product inhibition in fatty acid synthesis which requires NADPH.
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p><p>G6PD is negatively regulated by acetylation on lysine 403 (Lys403), an evolutionarily conserved residue. The K403 acetylated G6PD is incapable of forming active dimers and displays a complete loss of activity. Mechanistically, acetylating Lys304 sterically hinders the NADP from entering the NADP structural site, which reduces the stability of the enzyme. Cells sense extracellular oxidative stimuli to decrease G6PD acetylation in a SIRT2-dependent manner. The SIRT2-mediated deacetylation and activation of G6PD stimulates pentose phosphate pathway to supply cytosolic NADPH to counteract oxidative damage and protect mouse erythrocytes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p><p>Regulation can also occur through genetic pathways. The isoform, G6PDH, is regulated by transcription and posttranscription factors. Moreover, G6PD is one of a number of glycolytic enzymes activated by the transcription factor hypoxia-inducible factor 1 (HIF1).
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
<h2><span class="mw-headline" id="Clinical_significance">Clinical significance</span></h2>
<p>G6PD is remarkable for its genetic diversity. Many variants of G6PD, mostly produced from missense mutations, have been described with wide-ranging levels of enzyme activity and associated clinical symptoms. Two transcript variants encoding different isoforms have been found for this gene.
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p><p>Glucose-6-phosphate dehydrogenase deficiency is very common worldwide, and causes acute hemolytic anemia in the presence of simple infection, ingestion of fava beans, or reaction with certain medicines, antibiotics, antipyretics, and antimalarials.
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p>
<blockquote>
<div class="center"><div class="floatnone"><img alt="Pathology of G6PD deficiency.png" data-file-height="6967" data-file-width="4430" decoding="async" height="629" src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Pathology_of_G6PD_deficiency.png/400px-Pathology_of_G6PD_deficiency.png" srcset width="400"/></div></div>
</blockquote>
<p>Cell growth and proliferation are affected by G6PD. G6PD inhibitors are under investigation to treat cancers and other conditions. <i>In vitro</i> cell proliferation assay indicates that G6PD inhibitors, DHEA (dehydroepiandrosterone) and ANAD (6-aminonicotinamide), effectively decrease the growth of AML cell lines. G6PD is hypomethylated at K403 in acute myeloid leukemia, SIRT2 activates G6PD to enhance NADPH production and promote leukemia cell proliferation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
<script>function offsetAnchor() {if(location.hash.length !== 0) {window.scrollTo(window.scrollX, window.scrollY - 250);}}window.addEventListener("hashchange", offsetAnchor);window.setTimeout(offsetAnchor, 1);</script>