<!DOCTYPE html>

<html class="client-nojs" dir="ltr" lang="en">
<head>
<meta charset="utf-8"/>
<title>Mitochondrion - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Mitochondrion","wgTitle":"Mitochondrion","wgCurRevisionId":880859104,"wgRevisionId":880859104,"wgArticleId":19588,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["Wikipedia indefinitely move-protected pages","Articles containing Greek-language text","All articles with specifically marked weasel-worded phrases","Articles with specifically marked weasel-worded phrases from January 2016","Wikipedia articles incorporating text from the United States National Library of Medicine","Good articles","Wikipedia articles with BNE identifiers","Wikipedia articles with BNF identifiers","Wikipedia articles with GND identifiers","Wikipedia articles with LCCN identifiers","Wikipedia articles with NDL identifiers","Mitochondria","Cellular respiration","Endosymbiotic events"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Mitochondrion","wgRelevantArticleId":19588,"wgRequestId":"XFITOgpAICEAAHDhUDsAAABK","wgCSPNonce":false,"wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":["sysop"],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgWikiEditorEnabledModules":[],"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsShouldSendModuleToUser":true,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFExpandAllSectionsUserOption":true,"wgMFEnableFontChanger":true,"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgWMESchemaEditAttemptStepOversample":false,"wgPoweredByHHVM":true,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q39572","wgScoreNoteLanguages":{"arabic":"Ø§Ù„Ø¹Ø±Ø¨ÙŠØ©","catalan":"catalÃ ","deutsch":"Deutsch","english":"English","espanol":"espaÃ±ol","italiano":"italiano","nederlands":"Nederlands","norsk":"norsk","portugues":"portuguÃªs","suomi":"suomi","svenska":"svenska","vlaams":"West-Vlams"},"wgScoreDefaultNoteLanguage":"nederlands","wgCentralAuthMobileDomain":false,"wgCodeMirrorEnabled":true,"wgVisualEditorToolbarScrollOffset":0,"wgVisualEditorUnsupportedEditParams":["undo","undoafter","veswitched"],"wgEditSubmitButtonLabelPublish":true,"oresWikiId":"enwiki","oresBaseUrl":"http://ores.discovery.wmnet:8081/","oresApiVersion":3});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});RLPAGEMODULES=["ext.cite.ux-enhancements","site","mediawiki.page.startup","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.quicksurveys.init","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];mw.loader.load(RLPAGEMODULES);});</script>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta content="" name="ResourceLoaderDynamicStyles"/>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<meta content="MediaWiki 1.33.0-wmf.14" name="generator"/>
<meta content="origin" name="referrer"/>
<meta content="origin-when-crossorigin" name="referrer"/>
<meta content="origin-when-cross-origin" name="referrer"/>
<meta content="https://upload.wikimedia.org/wikipedia/commons/0/0c/Mitochondria%2C_mammalian_lung_-_TEM.jpg" property="og:image"/>
<link href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Mitochondrion" rel="alternate"/>
<link href="/w/index.php?title=Mitochondrion&amp;action=edit" rel="alternate" title="Edit this page" type="application/x-wiki"/>
<link href="/w/index.php?title=Mitochondrion&amp;action=edit" rel="edit" title="Edit this page"/>
<link href="/static/apple-touch/wikipedia.png" rel="apple-touch-icon"/>
<link href="/static/favicon/wikipedia.ico" rel="shortcut icon"/>
<link href="/w/opensearch_desc.php" rel="search" title="Wikipedia (en)" type="application/opensearchdescription+xml"/>
<link href="//en.wikipedia.org/w/api.php?action=rsd" rel="EditURI" type="application/rsd+xml"/>
<link href="//creativecommons.org/licenses/by-sa/3.0/" rel="license"/>
<link href="https://en.wikipedia.org/wiki/Mitochondrion" rel="canonical"/>
<link href="//login.wikimedia.org" rel="dns-prefetch"/>
<link href="//meta.wikimedia.org" rel="dns-prefetch"/>
<!--[if lt IE 9]><script src="/w/load.php?debug=false&amp;lang=en&amp;modules=html5shiv&amp;only=scripts&amp;skin=vector&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-Mitochondrion rootpage-Mitochondrion skin-vector action-view"> <div class="noprint" id="mw-page-base"></div>
<div class="noprint" id="mw-head-base"></div>
<div class="mw-body" id="content" role="main">

<div class="mw-body-content" id="siteNotice"><!-- CentralNotice --></div><div class="mw-indicators mw-body-content">
<div class="mw-indicator" id="mw-indicator-good-star"><img alt="This is a good article. Follow the link for more information." data-file-height="185" data-file-width="180" decoding="async" height="20" src="https://upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/19px-Symbol_support_vote.svg.png" srcset width="19"/></div>
</div>
<h1 class="firstHeading" id="firstHeading" lang="en">Mitochondrion</h1> <div class="mw-body-content" id="bodyContent">
<div class="noprint" id="siteSub">From Wikipedia, the free encyclopedia</div> <div id="contentSub"></div>
<div id="jump-to-nav"></div> Jump to navigation
Jump to search
<div class="mw-content-ltr" dir="ltr" id="mw-content-text" lang="en"><div class="mw-parser-output"><div class="thumb tright"><div class="thumbinner" style="width:297px;"><img alt="" class="thumbimage" data-file-height="480" data-file-width="640" decoding="async" height="221" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/0c/Mitochondria%2C_mammalian_lung_-_TEM.jpg/295px-Mitochondria%2C_mammalian_lung_-_TEM.jpg" srcset width="295"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>1</h1></center></div>Two mitochondria from mammalian lung tissue displaying their matrix and membranes as shown by electron microscopy</div></div></div>
<table class="infobox" style="width:22em;width: 300px; padding: 0; border-collapse:collapse; background:transparent;"><tbody><tr><th colspan="2" style="text-align:center;font-size:125%;font-weight:bold;background: rgba(0,0,0,0.1); padding: 5px; width: 290px;"><span style="color:#000;">Cell biology</span></th></tr><tr><td colspan="2" style="text-align:center;padding: 0;"><img alt="Animal Cell.svg" data-file-height="811" data-file-width="1405" decoding="async" height="173" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/11/Animal_Cell.svg/300px-Animal_Cell.svg.png" srcset width="300"/><div style="text-align:left; padding: 10px; background: rgba(0,0,0,0.02);">Components of a typical animal cell:
<ol><li>Nucleolus</li>
<li>Nucleus</li>
<li>Ribosome (little dots)</li>
<li>Vesicle</li>
<li>Rough endoplasmic reticulum</li>
<li>Golgi apparatus (or "Golgi body")</li>
<li>Cytoskeleton</li>
<li>Smooth endoplasmic reticulum</li>
<li>Mitochondrion</li>
<li>Vacuole</li>
<li>Cytosol (fluid that contains organelles, comprising the cytoplasm)</li>
<li>Lysosome</li>
<li>Centrosome</li>
<li>Cell membrane</li></ol></div>
</td></tr><tr><td colspan="2" style="text-align:center;padding: 0;"><img alt="Mitochondrion mini.svg" data-file-height="490" data-file-width="600" decoding="async" height="245" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/Mitochondrion_mini.svg/300px-Mitochondrion_mini.svg.png" srcset width="300"/><div style="text-align:left; padding: 10px; background: rgba(0,0,0,0.02);"><b>Components of a typical mitochondrion</b>
<p><b>1</b> Outer membrane<br/>
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p>
<dl><dd><b>1.1</b> Porin<br/></dd></dl>
<p><b>2</b> Intermembrane space<br/>
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p>
<dl><dd><b>2.1</b> Intracristal space<br/></dd>
<dd><b>2.2</b> Peripheral space<br/></dd></dl>
<p><b>3</b> Lamella<br/>
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p>
<dl><dd><b>3.1</b> Inner membrane<br/>
<dl><dd><b>3.11</b> Inner boundary membrane<br/></dd>
<dd><b>3.12</b> Cristal membrane<br/></dd></dl></dd>
<dd><b>3.2</b> Matrix<br/></dd>
<dd><b>3.3</b> CristÃ¦ <br/></dd></dl>
<p><b>4</b> Mitochondrial DNA<br/>
<b>5</b> Matrix granule<br/>
<b>6</b> Ribosome<br/>
<b>7</b> ATP synthase
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p>
<br/></div></td></tr></tbody></table>
<p>The <b>mitochondrion</b> (plural <b>mitochondria</b>) is a double-membrane-bound organelle found in most eukaryotic organisms. Some cells in some multicellular organisms may, however, lack them (for example, mature mammalian red blood cells). A number of unicellular organisms, such as microsporidia, parabasalids, and diplomonads, have also reduced or transformed their mitochondria into other structures. To date, only one eukaryote, <i>Monocercomonoides</i>, is known to have completely lost its mitochondria. The word mitochondrion comes from the Greek <span lang="el" title="Greek language text">Î¼Î¯Ï„Î¿Ï‚</span>, <i lang="el-Latn" title="Greek-language transliteration">mitos</i>, "thread", and <span lang="el" title="Greek language text">Ï‡Î¿Î½Î´ÏÎ¯Î¿Î½</span>, <i lang="el-Latn" title="Greek-language transliteration">chondrion</i>, "granule" or "grain-like". Mitochondria generate most of the cell's supply of adenosine triphosphate (ATP), used as a source of chemical energy.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p><p>Mitochondria are commonly between 0.75 and 3Â Î¼m in diameter but vary considerably in size and structure. Unless specifically stained, they are not visible.  In addition to supplying cellular energy, mitochondria are involved in other tasks, such as signaling, cellular differentiation, and cell death, as well as maintaining control of the cell cycle and cell growth. Mitochondrial biogenesis is in turn temporally coordinated with these cellular processes. Mitochondria have been implicated in several human diseases, including mitochondrial disorders, cardiac dysfunction, heart failure and autism.
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p><p>The number of mitochondria in a cell can vary widely by organism, tissue, and cell type. For instance, red blood cells have no mitochondria, whereas liver cells can have more than 2000.  The organelle is composed of compartments that carry out specialized functions. These compartments or regions include the outer membrane, the intermembrane space, the inner membrane, and the cristae and matrix.
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p><p>Although most of a cell's DNA is contained in the cell nucleus, the mitochondrion has its own independent genome that shows substantial similarity to bacterial genomes.  Mitochondrial proteins (proteins transcribed from mitochondrial DNA) vary depending on the tissue and the species. In humans, 615 distinct types of protein have been identified from cardiac mitochondria, whereas in rats, 940 proteins have been reported. The mitochondrial proteome is thought to be dynamically regulated.
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p>
<div class="toc" id="toc"><input class="toctogglecheckbox" id="toctogglecheckbox" role="button" style="display:none" type="checkbox"/><div class="toctitle" dir="ltr" lang="en"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">History</span></li>
<li class="toclevel-1 tocsection-2"><span class="tocnumber">2</span> <span class="toctext">Origin and evolution</span></li>
<li class="toclevel-1 tocsection-3"><span class="tocnumber">3</span> <span class="toctext">Structure</span>
<ul>
<li class="toclevel-2 tocsection-4"><span class="tocnumber">3.1</span> <span class="toctext">Outer membrane</span></li>
<li class="toclevel-2 tocsection-5"><span class="tocnumber">3.2</span> <span class="toctext">Intermembrane space</span></li>
<li class="toclevel-2 tocsection-6"><span class="tocnumber">3.3</span> <span class="toctext">Inner membrane</span>
<ul>
<li class="toclevel-3 tocsection-7"><span class="tocnumber">3.3.1</span> <span class="toctext">Cristae</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-8"><span class="tocnumber">3.4</span> <span class="toctext">Matrix</span></li>
<li class="toclevel-2 tocsection-9"><span class="tocnumber">3.5</span> <span class="toctext">Mitochondria-associated ER membrane (MAM)</span>
<ul>
<li class="toclevel-3 tocsection-10"><span class="tocnumber">3.5.1</span> <span class="toctext">Phospholipid transfer</span></li>
<li class="toclevel-3 tocsection-11"><span class="tocnumber">3.5.2</span> <span class="toctext">Calcium signaling</span></li>
<li class="toclevel-3 tocsection-12"><span class="tocnumber">3.5.3</span> <span class="toctext">Molecular basis for tethering</span></li>
<li class="toclevel-3 tocsection-13"><span class="tocnumber">3.5.4</span> <span class="toctext">Perspective</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-14"><span class="tocnumber">4</span> <span class="toctext">Organization and distribution</span></li>
<li class="toclevel-1 tocsection-15"><span class="tocnumber">5</span> <span class="toctext">Function</span>
<ul>
<li class="toclevel-2 tocsection-16"><span class="tocnumber">5.1</span> <span class="toctext">Energy conversion</span>
<ul>
<li class="toclevel-3 tocsection-17"><span class="tocnumber">5.1.1</span> <span class="toctext">Pyruvate and the citric acid cycle</span></li>
<li class="toclevel-3 tocsection-18"><span class="tocnumber">5.1.2</span> <span class="toctext">NADH and FADH<sub>2</sub>: the electron transport chain</span></li>
<li class="toclevel-3 tocsection-19"><span class="tocnumber">5.1.3</span> <span class="toctext">Heat production</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-20"><span class="tocnumber">5.2</span> <span class="toctext">Storage of calcium ions</span></li>
<li class="toclevel-2 tocsection-21"><span class="tocnumber">5.3</span> <span class="toctext">Additional functions</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-22"><span class="tocnumber">6</span> <span class="toctext">Cellular proliferation regulation</span></li>
<li class="toclevel-1 tocsection-23"><span class="tocnumber">7</span> <span class="toctext">Genome</span>
<ul>
<li class="toclevel-2 tocsection-24"><span class="tocnumber">7.1</span> <span class="toctext">Alternative genetic code</span></li>
<li class="toclevel-2 tocsection-25"><span class="tocnumber">7.2</span> <span class="toctext">Evolution and diversity</span></li>
<li class="toclevel-2 tocsection-26"><span class="tocnumber">7.3</span> <span class="toctext">Replication and inheritance</span></li>
<li class="toclevel-2 tocsection-27"><span class="tocnumber">7.4</span> <span class="toctext">DNA repair</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-28"><span class="tocnumber">8</span> <span class="toctext">Population genetic studies</span></li>
<li class="toclevel-1 tocsection-29"><span class="tocnumber">9</span> <span class="toctext">Dysfunction and disease</span>
<ul>
<li class="toclevel-2 tocsection-30"><span class="tocnumber">9.1</span> <span class="toctext">Mitochondrial diseases</span></li>
<li class="toclevel-2 tocsection-31"><span class="tocnumber">9.2</span> <span class="toctext">Possible relationships to aging</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-32"><span class="tocnumber">10</span> <span class="toctext">In popular culture</span></li>
<li class="toclevel-1 tocsection-33"><span class="tocnumber">11</span> <span class="toctext">See also</span></li>
<li class="toclevel-1 tocsection-34"><span class="tocnumber">12</span> <span class="toctext">References</span></li>
<li class="toclevel-1 tocsection-35"><span class="tocnumber">13</span> <span class="toctext">External links</span></li>
</ul>
</div>
<h2><span class="mw-headline" id="History">History</span></h2>
<p>The first observations of intracellular structures that probably represented mitochondria were published in the 1840s. Richard Altmann, in 1890, established them as cell organelles and called them "bioblasts". The term "mitochondria" was coined by Carl Benda in 1898. Leonor Michaelis discovered that Janus green can be used as a supravital stain for mitochondria in 1900. In 1904, Friedrich Meves, made the first recorded observation of mitochondria in plants in cells of the white waterlily, <i>Nymphaea alba</i> and in 1908, along with Claudius Regaud, suggested that they contain proteins and lipids. Benjamin F. Kingsbury, in 1912, first related them with cell respiration, but almost exclusively based on morphological observations. In 1913, particles from extracts of guinea-pig liver were linked to respiration by Otto Heinrich Warburg, which he called "grana". Warburg and Heinrich Otto Wieland, who had also postulated a similar particle mechanism, disagreed on the chemical nature of the respiration. It was not until 1925, when David Keilin discovered cytochromes, that the respiratory chain was described.
<span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p><p>In 1939, experiments using minced muscle cells demonstrated that cellular respiration using one oxygen atom can form two adenosine triphosphate (ATP) molecules, and, in 1941, the concept of the phosphate bonds of ATP being a form of energy in cellular metabolism was developed by Fritz Albert Lipmann. In the following years, the mechanism behind cellular respiration was further elaborated, although its link to the mitochondria was not known. The introduction of tissue fractionation by Albert Claude allowed mitochondria to be isolated from other cell fractions and biochemical analysis to be conducted on them alone. In 1946, he concluded that cytochrome oxidase and other enzymes responsible for the respiratory chain were isolated to the mitochondria. Eugene Kennedy and Albert Lehninger discovered in 1948 that mitochondria are the site of oxidative phosphorylation in eukaryotes. Over time, the fractionation method was further developed, improving the quality of the mitochondria isolated, and other elements of cell respiration were determined to occur in the mitochondria.
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p><p>The first high-resolution electron micrographs appeared in 1952, replacing the Janus Green stains as the preferred way of visualising the mitochondria. This led to a more detailed analysis of the structure of the mitochondria, including confirmation that they were surrounded by a membrane. It also showed a second membrane inside the mitochondria that folded up in ridges dividing up the inner chamber and that the size and shape of the mitochondria varied from cell to cell.
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p><p>The popular term "powerhouse of the cell" was coined by Philip Siekevitz in 1957.
<span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p><p>In 1967, it was discovered that mitochondria contained ribosomes. In 1968, methods were developed for mapping the mitochondrial genes, with the genetic and physical map of yeast mitochondrial DNA being completed in 1976.
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p>
<h2><span class="mw-headline" id="Origin_and_evolution">Origin and evolution</span></h2>
<div class="hatnote navigation-not-searchable" role="note">Main article: Endosymbiotic theory</div>
<p>There are two hypotheses about the origin of mitochondria: endosymbiotic and autogenous. The endosymbiotic hypothesis suggests that mitochondria were originally prokaryotic cells, capable of implementing oxidative mechanisms that were not possible for eukaryotic cells; they became endosymbionts living inside the eukaryote. In the autogenous hypothesis, mitochondria were born by splitting off a portion of DNA from the nucleus of the eukaryotic cell at the time of divergence with the prokaryotes; this DNA portion would have been enclosed by membranes, which could not be crossed by proteins. Since mitochondria have many features in common with bacteria, the endosymbiotic hypothesis is more widely accepted.
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p><p>A mitochondrion contains DNA, which is organized as several copies of a single, usually circular, chromosome. This mitochondrial chromosome contains genes for redox proteins, such as those of the respiratory chain. The CoRR hypothesis proposes that this co-location is required for redox regulation. The mitochondrial genome codes for some RNAs of ribosomes, and the 22 tRNAs necessary for the translation of mRNAs into protein. The circular structure is also found in prokaryotes. The proto-mitochondrion was probably closely related to the <i>Rickettsia</i>. However, the exact relationship of the ancestor of mitochondria to the alphaproteobacteria and whether the mitochondrion was formed at the same time or after the nucleus, remains controversial. For example, it has been suggested that the SAR11 clade of bacteria shares a relatively recent common ancestor with the mitochondria, while phylogenomic analyses indicate that mitochondria evolved from a proteobacteria lineage that branched off before the divergence of all sampled alphaproteobacteria.
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p>
<table style="border: 1px solid #ccc; background-color: white; vertical-align: middle; width: 248px; margin:0.2em auto;">
<tbody><tr>
<td style="text-align: center; font-weight: bold; border: 0; padding: 0;">Schematic ribosomal RNA phylogeny of Alphaproteobacteria
</td></tr>
<tr>
<td style="border: 1px solid #dedede; padding: 3px">
<style data-mw-deduplicate="TemplateStyles:r853264556">.mw-parser-output table.clade{border-spacing:0;margin:0;font-size:100%;line-height:100%;border-collapse:separate;width:auto}.mw-parser-output table.clade table.clade{width:100%}.mw-parser-output table.clade td{border:0;padding:0;vertical-align:middle;text-align:center}.mw-parser-output table.clade td.clade-label{width:0.8em;border:0;padding:0 0.2em;vertical-align:bottom;text-align:center}.mw-parser-output table.clade td.clade-slabel{border:0;padding:0 0.2em;vertical-align:top;text-align:center}.mw-parser-output table.clade td.clade-bar{vertical-align:middle;text-align:left;padding:0 0.5em}.mw-parser-output table.clade td.clade-leaf{border:0;padding:0;text-align:left;vertical-align:middle}.mw-parser-output table.clade td.clade-leafR{border:0;padding:0;text-align:right}</style>
<table class="clade" style="font-size:100%;line-height:125%;width:770px;">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â MagnetococcidaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Magnetococcus marinus</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â CaulobacteridaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  Rhodospirillales, Sphingomonadales, <br/> Â  Rhodobacteraceae, Rhizobiales, <i>etc</i>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  Holosporales
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">Â Â RickettsidaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â PelagibacteralesÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â PelagibacteraceaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Pelagibacter</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  Subgroups Ib, II, IIIa, IIIb, IV and V
<span style="border: 3px solid blue; color:blue;font-weight: bold">20</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  Proto-mitochondria
<span style="border: 3px solid blue; color:blue;font-weight: bold">21</span></p>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â AnaplasmataceaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Ehrlichia</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">22</span></p>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Anaplasma</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">23</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Wolbachia</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">24</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Neorickettsia</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">25</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">Â Â MidichloriaceaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Midichloria</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">26</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">Â Â RickettsiaceaeÂ Â 
</td>
<td class="clade-leaf" rowspan="2">
<link href="mw-data:TemplateStyles:r853264556" rel="mw-deduplicated-inline-style"/>
<table class="clade">
<tbody><tr>
<td class="clade-label" style="border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Rickettsia</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">27</span></p>
</td></tr>
<tr>
<td class="clade-slabel" style="border-left:1px solid;">â€¯
</td></tr>
<tr>
<td class="clade-label" style="border-left:1px solid;border-bottom:1px solid;">â€¯
</td>
<td class="clade-leaf" rowspan="2">
<p>Â  <i>Orientia</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">28</span></p>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td class="clade-slabel">â€¯
</td></tr></tbody></table>
</td></tr>
<tr>
<td style="text-align: left; font-size: 88%; border: 0; padding: 0;">The cladogram of Rickettsidae has been inferred by Ferla <i>et al.</i>  from the comparison of 16S + 23S ribosomal RNA sequences.
</td></tr></tbody></table>
<p>The ribosomes coded for by the mitochondrial DNA are similar to those from bacteria in size and structure. They closely resemble the bacterial 70S ribosome and not the 80S cytoplasmic ribosomes, which are coded for by nuclear DNA.
<span style="border: 3px solid blue; color:blue;font-weight: bold">29</span></p><p>The endosymbiotic relationship of mitochondria with their host cells was popularized by Lynn Margulis. The endosymbiotic hypothesis suggests that mitochondria descended from bacteria that somehow survived endocytosis by another cell, and became incorporated into the cytoplasm. The ability of these bacteria to conduct respiration in host cells that had relied on glycolysis and fermentation would have provided a considerable evolutionary advantage. This symbiotic relationship probably developed 1.7 to 2 billion years ago.
A few groups of unicellular eukaryotes have only vestigial mitochondria or derived structures: the microsporidians, metamonads, and archamoebae. These groups appear as the most primitive eukaryotes on phylogenetic trees constructed using rRNA information, which once suggested that they appeared before the origin of mitochondria. However, this is now known to be an artifact of long-branch attractionâ€”they are derived groups and retain genes or organelles derived from mitochondria (e.g., mitosomes and hydrogenosomes).
<span style="border: 3px solid blue; color:blue;font-weight: bold">30</span></p><p>Monocercomonoides appear to have lost their mitochondria completely and at least some of the mitochondrial functions seem to be carried out by cytoplasmic proteins now<i>.</i>
<span style="border: 3px solid blue; color:blue;font-weight: bold">31</span></p>
<h2><span class="mw-headline" id="Structure">Structure</span></h2>
<div class="floatright">
<div class="thumbinner" style="background:none;border:none;max-width:500px;">
<div style="position: relative; left: 2px;"><div style="position:relative; float: right; width: 500px; height: 350px;">
<div style="position:relative; font-size: 11px; color: rgba(6, 69, 173, 1);"><img alt="Mitochondrion structure labels.svg" data-file-height="700" data-file-width="1000" decoding="async" height="350" src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Mitochondrion_structure_labels.svg/500px-Mitochondrion_structure_labels.svg.png" srcset width="500"/>
<div style="position:absolute; background: white; background: rgba(255, 255, 255, 0); top: 0; left: 0; width: 500px; height: 350px;"></div>
<div style="position:absolute; font-size: 11px; color: rgba(6, 69, 173, 0); top: 0; left: 0;"><img alt="Mitochondrion structure" data-file-height="700" data-file-width="1000" decoding="async" height="350" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Mitochondrion_structure_drawing.svg/500px-Mitochondrion_structure_drawing.svg.png" srcset width="500"/>
<div style="position:absolute; left: 30px; bottom: 64px; padding-left: 5px; padding-right: 5px;"><b>1 <span style="color: rgba(6, 69, 173, 0);">Outer membrane</span></b></div>
<div style="position:absolute; left: 76px; bottom: 54px; padding-left: 5px; padding-right: 5px;"><b>1.1</b> <span style="color: rgba(6, 69, 173, 0);">Porins</span></div>
<div style="font-size: 12px; position:absolute; right: 60px; bottom: 133px; padding-left: 5px; padding-right: 5px;"><b>2 <span style="color: rgba(6, 69, 173, 0);">Intermembrane space</span></b></div>
<div style="font-size: 12px; position:absolute; right: 76px; bottom: 123px; padding-left: 5px; padding-right: 5px;"><b>2.1</b> <span style="color: rgba(6, 69, 173, 0);">Intracristal space</span></div>
<div style="font-size: 12px; position:absolute; right: 76px; bottom: 113px; padding-left: 5px; padding-right: 5px;"><b>2.2</b> <span style="color: rgba(6, 69, 173, 0);">Peripheral space</span></div>
<div style="font-size: 12px; position:absolute; left: 30px; bottom: 314px; padding-left: 5px; padding-right: 5px;"><b>3 <span style="color: rgba(6, 69, 173, 0);">LamellÃ¦</span></b></div>
<div style="font-size: 12px; position:absolute; left: 30px; bottom: 304px; padding-left: 5px; padding-right: 5px;"><b>3.1</b> <span style="color: rgba(6, 69, 173, 0);">Inner membrane</span></div>
<div style="font-size: 12px; position:absolute; left: 32px; bottom: 294px; padding-left: 5px; padding-right: 5px;"><b>3.11</b> <span style="color: rgba(6, 69, 173, 0);">Inner boundary membrane</span></div>
<div style="font-size: 12px; position:absolute; left: 32px; bottom: 284px; padding-left: 5px; padding-right: 5px;"><b>3.12</b> <span style="color: rgba(6, 69, 173, 0);">Cristal membrane</span></div>
<div style="font-size: 12px; position:absolute; left: 30px; bottom: 274px; padding-left: 5px; padding-right: 5px;"><b>3.2</b> <span style="color: rgba(6, 69, 173, 0);">Matrix</span></div>
<div style="font-size: 12px; position:absolute; left: 30px; bottom: 264px; padding-left: 5px; padding-right: 5px;"><b>3.3</b> <span style="color: rgba(6, 69, 173, 0);">CristÃ¦</span></div>
<div style="font-size: 12px; position:absolute; right: 157px; bottom: 324px; padding-left: 5px; padding-right: 5px;"><b>4 <span style="color: rgba(6, 69, 173, 0);">Mitochondrial DNA</span></b></div>
<div style="font-size: 12px; position:absolute; right: 73px; bottom: 314px; padding-left: 5px; padding-right: 5px;"><b>5 <span style="color: rgba(6, 69, 173, 0);">Matix granule</span></b></div>
<div style="font-size: 12px; position:absolute; right: 28px; bottom: 294px; padding-left: 5px; padding-right: 5px;"><b>6 <span style="color: rgba(6, 69, 173, 0);">Ribosome</span></b></div>
<div style="font-size: 12px; position:absolute; right: 3px; bottom: 153px; padding-left: 5px; padding-right: 5px;"><b>7 <span style="color: rgba(6, 69, 173, 0);">ATP synthase</span></b></div>
<div style="position:absolute; right: 20px; bottom: 10px; text-align: left; font-size: 11.5px; line-height: 110%; color: black;"><b>Edit Â· Source image</b></div>
</div>
</div>
</div>
</div><div class="thumbcaption"><div style="float: left; margin-top: 0.25em; width: 11px; height: 9.5px;"><div style="position: relative; width: 0px; height: 0px; border-style: solid inset; border-width: 0 5.5px 9.5px 5.5px; border-color: transparent transparent #bd0926 transparent;"></div></div><div style="padding-left:24px;"><b>Mitochondrion ultrastructure</b> <i>(interactive diagram)</i> A mitochondrion has a double membrane; the inner one contains its chemiosmotic apparatus and has deep grooves which increase its surface area. While commonly depicted as an "orange sausage with a blob inside of it" (like it is here), mitochondria can take many shapes and their intermembrane space is quite thin.</div> </div>
</div>
</div>
<p>A mitochondrion contains outer and inner membranes composed of phospholipid bilayers and proteins. The two membranes have different properties. Because of this double-membraned organization, there are five distinct parts to a mitochondrion. They are:
<span style="border: 3px solid blue; color:blue;font-weight: bold">32</span></p>
<ol><li>the outer mitochondrial membrane,</li>
<li>the intermembrane space (the space between the outer and inner membranes),</li>
<li>the inner mitochondrial membrane,</li>
<li>the cristae space (formed by infoldings of the inner membrane), and</li>
<li>the matrix (space within the inner membrane).</li></ol>
<p>Mitochondria stripped of their outer membrane are called mitoplasts.
<span style="border: 3px solid blue; color:blue;font-weight: bold">33</span></p>
<h3><span class="mw-headline" id="Outer_membrane">Outer membrane</span></h3>
<p>The <b>outer mitochondrial membrane</b>, which encloses the entire organelle, is 60 to 75 angstroms (Ã…) thick. It has a protein-to-phospholipid ratio similar to that of the eukaryotic plasma membrane (about 1:1 by weight). It contains large numbers of integral membrane proteins called porins. These porins form channels that allow molecules of 5000Â daltons or less in molecular weight to freely diffuse from one side of the membrane to the other. Larger proteins can enter the mitochondrion if a signaling sequence at their N-terminus binds to a large multisubunit protein called translocase in the outer membrane, which then actively moves them across the membrane. Mitochondrial pro-proteins are imported through specialised translocation complexes. The outer membrane also contains enzymes involved in such diverse activities as the elongation of fatty acids, oxidation of epinephrine, and the degradation of tryptophan. These enzymes include monoamine oxidase, rotenone-insensitive NADH-cytochrome c-reductase, kynurenine hydroxylase and fatty acid Co-A ligase. Disruption of the outer membrane permits proteins in the intermembrane space to leak into the cytosol, leading to certain cell death. The mitochondrial outer membrane can associate with the endoplasmic reticulum (ER) membrane, in a structure called MAM (mitochondria-associated ER-membrane). This is important in the ER-mitochondria calcium signaling and is involved in the transfer of lipids between the ER and mitochondria.
 Outside the outer membrane there are small (diameter: 60Ã…) particles named sub-units of Parson.
<span style="border: 3px solid blue; color:blue;font-weight: bold">34</span></p>
<h3><span class="mw-headline" id="Intermembrane_space">Intermembrane space</span></h3>
<p>The intermembrane space is the space between the outer membrane and the inner membrane. It is also known as perimitochondrial space. Because the outer membrane is freely permeable to small molecules, the concentrations of small molecules, such as ions and sugars, in the intermembrane space is the same as in the cytosol. However, large proteins must have a specific signaling sequence to be transported across the outer membrane, so the protein composition of this space is different from the protein composition of the cytosol. One protein that is localized to the intermembrane space in this way is cytochrome c.
<span style="border: 3px solid blue; color:blue;font-weight: bold">35</span></p>
<h3><span class="mw-headline" id="Inner_membrane">Inner membrane</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Inner mitochondrial membrane</div>
<p>The inner mitochondrial membrane contains proteins with five types of functions:
<span style="border: 3px solid blue; color:blue;font-weight: bold">36</span></p>
<ol><li>Those that perform the redox reactions of oxidative phosphorylation</li>
<li>ATP synthase, which generates ATP in the matrix</li>
<li>Specific transport proteins that regulate metabolite passage into and out of the matrix</li>
<li>Protein import machinery</li>
<li>Mitochondrial fusion and fission protein</li></ol>
<p>It contains more than 151 different polypeptides, and has a very high protein-to-phospholipid ratio (more than 3:1 by weight, which is about 1Â protein for 15Â phospholipids). The inner membrane is home to around 1/5 of the total protein in a mitochondrion. In addition, the inner membrane is rich in an unusual phospholipid, cardiolipin. This phospholipid was originally discovered in cow hearts in 1942, and is usually characteristic of mitochondrial and bacterial plasma membranes. Cardiolipin contains four fatty acids rather than two, and may help to make the inner membrane impermeable. Unlike the outer membrane, the inner membrane doesn't contain porins, and is highly impermeable to all molecules. Almost all ions and molecules require special membrane transporters to enter or exit the matrix. Proteins are ferried into the matrix via the translocase of the inner membrane (TIM) complex or via Oxa1. In addition, there is a membrane potential across the inner membrane, formed by the action of the enzymes of the electron transport chain.
<span style="border: 3px solid blue; color:blue;font-weight: bold">37</span></p>
<h4><span class="mw-headline" id="Cristae">Cristae</span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:252px;"><img alt="" class="thumbimage" data-file-height="409" data-file-width="618" decoding="async" height="165" src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d8/MitochondrionCAM.jpg/250px-MitochondrionCAM.jpg" srcset width="250"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>2</h1></center></div>Cross-sectional image of cristae in rat liver mitochondrion to demonstrate the likely 3D structure and relationship to the inner membrane</div></div></div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Cristae</div>
<p>The inner mitochondrial membrane is compartmentalized into numerous cristae, which expand the surface area of the inner mitochondrial membrane, enhancing its ability to produce ATP. For typical liver mitochondria, the area of the inner membrane is about five times as large as the outer membrane. This ratio is variable and mitochondria from cells that have a greater demand for ATP, such as muscle cells, contain even more cristae. These folds are studded with small round bodies known as F<sub>1</sub> particles or oxysomes. These are not simple random folds but rather invaginations of the inner membrane, which can affect overall chemiosmotic function.
<span style="border: 3px solid blue; color:blue;font-weight: bold">38</span></p><p>One recent mathematical modeling study has suggested that the optical properties of the cristae in filamentous mitochondria may affect the generation and propagation of light within the tissue.
<span style="border: 3px solid blue; color:blue;font-weight: bold">39</span></p>
<h3><span class="mw-headline" id="Matrix">Matrix</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Mitochondrial matrix</div>
<p>The matrix is the space enclosed by the inner membrane. It contains about 2/3 of the total protein in a mitochondrion. The matrix is important in the production of ATP with the aid of the ATP synthase contained in the inner membrane. The matrix contains a highly concentrated mixture of hundreds of enzymes, special mitochondrial ribosomes, tRNA, and several copies of the mitochondrial DNA genome. Of the enzymes, the major functions include oxidation of pyruvate and fatty acids, and the citric acid cycle.
<span style="border: 3px solid blue; color:blue;font-weight: bold">40</span></p><p>Mitochondria have their own genetic material, and the machinery to manufacture their own RNAs and proteins (<i>see: protein biosynthesis</i>). A published human mitochondrial DNA sequence revealed 16,569Â base pairs encoding 37 genes: 22Â tRNA, 2 rRNA, and 13Â peptide genes. The 13 mitochondrial peptides in humans are integrated into the inner mitochondrial membrane, along with proteins encoded by genes that reside in the host cell's nucleus.
<span style="border: 3px solid blue; color:blue;font-weight: bold">41</span></p>
<h3><span id="Mitochondria-associated_ER_membrane_.28MAM.29"></span><span class="mw-headline" id="Mitochondria-associated_ER_membrane_(MAM)">Mitochondria-associated ER membrane (MAM)</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Mitochondria associated membranes (MAM)</div>
<p>The mitochondria-associated ER membrane (MAM) is another structural element that is increasingly recognized for its critical role in cellular physiology and homeostasis. Once considered a technical snag in cell fractionation techniques, the alleged ER vesicle contaminants that invariably appeared in the mitochondrial fraction have been re-identified as membranous structures derived from the MAMâ€”the interface between mitochondria and the ER. Physical coupling between these two organelles had previously been observed in electron micrographs and has more recently been probed with fluorescence microscopy. Such studies estimate that at the MAM, which may comprise up to 20% of the mitochondrial outer membrane, the ER and mitochondria are separated by a mere 10â€“25Â nm and held together by protein tethering complexes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">42</span></p><p>Purified MAM from subcellular fractionation has been shown to be enriched in enzymes involved in phospholipid exchange, in addition to channels associated with Ca signaling. These hints of a prominent role for the MAM in the regulation of cellular lipid stores and signal transduction have been borne out, with significant implications for mitochondrial-associated cellular phenomena, as discussed below. Not only has the MAM provided insight into the mechanistic basis underlying such physiological processes as intrinsic apoptosis and the propagation of calcium signaling, but it also favors a more refined view of the mitochondria. Though often seen as static, isolated 'powerhouses' hijacked for cellular metabolism through an ancient endosymbiotic event, the evolution of the MAM underscores the extent to which mitochondria have been integrated into overall cellular physiology, with intimate physical and functional coupling to the endomembrane system.
<span style="border: 3px solid blue; color:blue;font-weight: bold">43</span></p>
<h4><span class="mw-headline" id="Phospholipid_transfer">Phospholipid transfer</span></h4>
<p>The MAM is enriched in enzymes involved in lipid biosynthesis, such as phosphatidylserine synthase on the ER face and phosphatidylserine decarboxylase on the mitochondrial face. Because mitochondria are dynamic organelles constantly undergoing fission and fusion events, they require a constant and well-regulated supply of phospholipids for membrane integrity. But mitochondria are not only a destination for the phospholipids they finish synthesis of; rather, this organelle also plays a role in inter-organelle trafficking of the intermediates and products of phospholipid biosynthetic pathways, ceramide and cholesterol metabolism, and glycosphingolipid anabolism.
<span style="border: 3px solid blue; color:blue;font-weight: bold">44</span></p><p>Such trafficking capacity depends on the MAM, which has been shown to facilitate transfer of lipid intermediates between organelles. In contrast to the standard vesicular mechanism of lipid transfer, evidence indicates that the physical proximity of the ER and mitochondrial membranes at the MAM allows for lipid flipping between opposed bilayers. Despite this unusual and seemingly energetically unfavorable mechanism, such transport does not require ATP. Instead, in yeast, it has been shown to be dependent on a multiprotein tethering structure termed the ER-mitochondria encounter structure, or ERMES, although it remains unclear whether this structure directly mediates lipid transfer or is required to keep the membranes in sufficiently close proximity to lower the energy barrier for lipid flipping.
<span style="border: 3px solid blue; color:blue;font-weight: bold">45</span></p><p>The MAM may also be part of the secretory pathway, in addition to its role in intracellular lipid trafficking. In particular, the MAM appears to be an intermediate destination between the rough ER and the Golgi in the pathway that leads to very-low-density lipoprotein, or VLDL, assembly and secretion. The MAM thus serves as a critical metabolic and trafficking hub in lipid metabolism.
<span style="border: 3px solid blue; color:blue;font-weight: bold">46</span></p>
<h4><span class="mw-headline" id="Calcium_signaling">Calcium signaling</span></h4>
<p>A critical role for the ER in calcium signaling was acknowledged before such a role for the mitochondria was widely accepted, in part because the low affinity of Ca channels localized to the outer mitochondrial membrane seemed to contradict this organelle's purported responsiveness to changes in intracellular Ca flux. But the presence of the MAM resolves this apparent contradiction: the close physical association between the two organelles results in Ca microdomains at contact points that facilitate efficient Ca transmission from the ER to the mitochondria. Transmission occurs in response to so-called "Ca puffs" generated by spontaneous clustering and activation of IP3R, a canonical ER membrane Ca channel.
<span style="border: 3px solid blue; color:blue;font-weight: bold">47</span></p><p>The fate of these puffsâ€”in particular, whether they remain restricted to isolated locales or integrated into Ca waves for propagation throughout the cellâ€”is determined in large part by MAM dynamics. Although reuptake of Ca by the ER (concomitant with its release) modulates the intensity of the puffs, thus insulating mitochondria to a certain degree from high Ca exposure, the MAM often serves as a firewall that essentially buffers Ca puffs by acting as a sink into which free ions released into the cytosol can be funneled. This Ca tunneling occurs through the low-affinity Ca receptor VDAC1, which recently has been shown to be physically tethered to the IP3R clusters on the ER membrane and enriched at the MAM. The ability of mitochondria to serve as a Ca sink is a result of the electrochemical gradient generated during oxidative phosphorylation, which makes tunneling of the cation an exergonic process. Normal, mild calcium influx from cytosol into the mitochondrial matrix causes transient depolarization that is corrected by pumping out protons.
<span style="border: 3px solid blue; color:blue;font-weight: bold">48</span></p><p>But transmission of Ca is not unidirectional; rather, it is a two-way street. The properties of the Ca pump SERCA and the channel IP3R present on the ER membrane facilitate feedback regulation coordinated by MAM function. In particular, the clearance of Ca by the MAM allows for spatio-temporal patterning of Ca signaling because Ca alters IP3R activity in a biphasic manner. SERCA is likewise affected by mitochondrial feedback: uptake of Ca by the MAM stimulates ATP production, thus providing energy that enables SERCA to reload the ER with Ca for continued Ca efflux at the MAM. Thus, the MAM is not a passive buffer for Ca puffs; rather it helps modulate further Ca signaling through feedback loops that affect ER dynamics.
<span style="border: 3px solid blue; color:blue;font-weight: bold">49</span></p><p>Regulating ER release of Ca at the MAM is especially critical because only a certain window of Ca uptake sustains the mitochondria, and consequently the cell, at homeostasis. Sufficient intraorganelle Ca signaling is required to stimulate metabolism by activating dehydrogenase enzymes critical to flux through the citric acid cycle. However, once Ca signaling in the mitochondria passes a certain threshold, it stimulates the intrinsic pathway of apoptosis in part by collapsing the mitochondrial membrane potential required for metabolism. Studies examining the role of pro- and anti-apoptotic factors support this model; for example, the anti-apoptotic factor Bcl-2 has been shown to interact with IP3Rs to reduce Ca filling of the ER, leading to reduced efflux at the MAM and preventing collapse of the mitochondrial membrane potential post-apoptotic stimuli. Given the need for such fine regulation of Ca signaling, it is perhaps unsurprising that dysregulated mitochondrial Ca has been implicated in several neurodegenerative diseases, while the catalogue of tumor suppressors includes a few that are enriched at the MAM.
<span style="border: 3px solid blue; color:blue;font-weight: bold">50</span></p>
<h4><span class="mw-headline" id="Molecular_basis_for_tethering">Molecular basis for tethering</span></h4>
<p>Recent advances in the identification of the tethers between the mitochondrial and ER membranes suggest that the scaffolding function of the molecular elements involved is secondary to other, non-structural functions. In yeast, ERMES, a multiprotein complex of interacting ER- and mitochondrial-resident membrane proteins, is required for lipid transfer at the MAM and exemplifies this principle. One of its components, for example, is also a constituent of the protein complex required for insertion of transmembrane beta-barrel proteins into the lipid bilayer. However, a homologue of the ERMES complex has not yet been identified in mammalian cells. Other proteins implicated in scaffolding likewise have functions independent of structural tethering at the MAM; for example, ER-resident and mitochondrial-resident mitofusins form heterocomplexes that regulate the number of inter-organelle contact sites, although mitofusins were first identified for their role in fission and fusion events between individual mitochondria. Glucose-related protein 75 (grp75) is another dual-function protein. In addition to the matrix pool of grp75, a portion serves as a chaperone that physically links the mitochondrial and ER Ca channels VDAC and IP3R for efficient Ca transmission at the MAM. Another potential tether is Sigma-1R, a non-opioid receptor whose stabilization of ER-resident IP3R may preserve communication at the MAM during the metabolic stress response.
<span style="border: 3px solid blue; color:blue;font-weight: bold">51</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="ERMES tethering complex." class="thumbimage" data-file-height="236" data-file-width="230" decoding="async" height="226" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/ERMES.png/220px-ERMES.png" srcset width="220"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>3</h1></center></div>Model of the yeast multimeric tethering complex, ERMES</div></div></div>
<h4><span class="mw-headline" id="Perspective">Perspective</span></h4>
<p>The MAM is a critical signaling, metabolic, and trafficking hub in the cell that allows for the integration of ER and mitochondrial physiology. Coupling between these organelles is not simply structural but functional as well and critical for overall cellular physiology and homeostasis. The MAM thus offers a perspective on mitochondria that diverges from the traditional view of this organelle as a static, isolated unit appropriated for its metabolic capacity by the cell. Instead, this mitochondrial-ER interface emphasizes the integration of the mitochondria, the product of an endosymbiotic event, into diverse cellular processes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">52</span></p>
<h2><span class="mw-headline" id="Organization_and_distribution">Organization and distribution</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" class="thumbimage" data-file-height="1024" data-file-width="1024" decoding="async" height="220" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/90/HeLa_mtGFP.tif/lossy-page1-220px-HeLa_mtGFP.tif.jpg" srcset width="220"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>4</h1></center></div>Typical mitochondrial network (green) in two human cells (HeLa cells)</div></div></div>
<p>Mitochondria (and related structures) are found in all eukaryotes (except oneâ€”the Oxymonad <i>Monocercomonoides</i> sp.). Although commonly depicted as bean-like structures they form a highly dynamic network in the majority of cells where they constantly undergo fission and fusion. The population of all the mitochondria of a given cell constitutes the chondriome. Mitochondria vary in number and location according to cell type. A single mitochondrion is often found in unicellular organisms. Conversely, the chondriome size of human liver cells is large, with about 1000â€“2000 mitochondria per cell, making up 1/5 of the cell volume. The mitochondrial content of otherwise similar cells can vary substantially in size and membrane potential, with differences arising from sources including uneven partitioning at cell divisions, leading to extrinsic differences in ATP levels and downstream cellular processes. The mitochondria can be found nestled between myofibrils of muscle or wrapped around the sperm flagellum. Often, they form a complex 3D branching network inside the cell with the cytoskeleton. The association with the cytoskeleton determines mitochondrial shape, which can affect the function as well: different structures of the mitochondrial network may afford the population a variety of physical, chemical, and signalling advantages or disadvantages. Mitochondria in cells are always distributed along microtubules and the distribution of these organelles is also correlated with the endoplasmic reticulum. Recent evidence suggests that vimentin, one of the components of the cytoskeleton, is also critical to the association with the cytoskeleton.
<span style="border: 3px solid blue; color:blue;font-weight: bold">53</span></p>
<h2><span class="mw-headline" id="Function">Function</span></h2>
<p>The most prominent roles of mitochondria are to produce the energy currency of the cell, ATP (i.e., phosphorylation of ADP), through respiration, and to regulate cellular metabolism. The central set of reactions involved in ATP production are collectively known as the citric acid cycle, or the Krebs cycle. However, the mitochondrion has many other functions in addition to the production of ATP.
<span style="border: 3px solid blue; color:blue;font-weight: bold">54</span></p>
<h3><span class="mw-headline" id="Energy_conversion">Energy conversion</span></h3>
<p>A dominant role for the mitochondria is the production of ATP, as reflected by the large number of proteins in the inner membrane for this task. This is done by oxidizing the major products of glucose: pyruvate, and NADH, which are produced in the cytosol. This type of cellular respiration known as aerobic respiration, is dependent on the presence of oxygen. When oxygen is limited, the glycolytic products will be metabolized by anaerobic fermentation, a process that is independent of the mitochondria. The production of ATP from glucose has an approximately 13-times higher yield during aerobic respiration compared to fermentation. Plant mitochondria can also produce a limited amount of ATP without oxygen by using the alternate substrate nitrite. ATP crosses out through the inner membrane with the help of a specific protein, and across the outer membrane via porins. ADP returns via the same route.
<span style="border: 3px solid blue; color:blue;font-weight: bold">55</span></p>
<h4><span class="mw-headline" id="Pyruvate_and_the_citric_acid_cycle">Pyruvate and the citric acid cycle</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main articles: Pyruvate dehydrogenase, Pyruvate carboxylase, and Citric acid cycle</div>
<p>Pyruvate molecules produced by glycolysis are actively transported across the inner mitochondrial membrane, and into the matrix where they can either be oxidized and combined with coenzyme A to form CO<sub>2</sub>, acetyl-CoA, and NADH, or they can be carboxylated (by pyruvate carboxylase) to form oxaloacetate. This latter reaction â€fills upâ€ the amount of oxaloacetate in the citric acid cycle, and is therefore an anaplerotic reaction, increasing the cycleâ€™s capacity to metabolize acetyl-CoA when the tissue's energy needs (e.g. in muscle) are suddenly increased by activity.
<span style="border: 3px solid blue; color:blue;font-weight: bold">56</span></p><p>In the citric acid cycle, all the intermediates (e.g. citrate, iso-citrate, alpha-ketoglutarate, succinate, fumarate, malate and oxaloacetate) are regenerated during each turn of the cycle. Adding more of any of these intermediates to the mitochondrion therefore means that the additional amount is retained within the cycle, increasing all the other intermediates as one is converted into the other. Hence, the addition of any one of them to the cycle has an anaplerotic effect, and its removal has a cataplerotic effect. These anaplerotic and cataplerotic reactions will, during the course of the cycle, increase or decrease the amount of oxaloacetate available to combine with acetyl-CoA to form citric acid. This in turn increases or decreases the rate of ATP production by the mitochondrion, and thus the availability of ATP to the cell.
<span style="border: 3px solid blue; color:blue;font-weight: bold">57</span></p><p>Acetyl-CoA, on the other hand, derived from pyruvate oxidation, or from the beta-oxidation of fatty acids, is the only fuel to enter the citric acid cycle. With each turn of the cycle one molecule of acetyl-CoA is consumed for every molecule of oxaloacetate present in the mitochondrial matrix, and is never regenerated. It is the oxidation of the acetate portion of acetyl-CoA that produces CO<sub>2</sub> and water, with the energy thus released captured in the form of ATP.
<span style="border: 3px solid blue; color:blue;font-weight: bold">58</span></p><p>In the liver, the carboxylation of cytosolic pyruvate into intra-mitochondrial oxaloacetate is an early step in the gluconeogenic pathway, which converts lactate and de-aminated alanine into glucose, under the influence of high levels of glucagon and/or epinephrine in the blood. Here, the addition of oxaloacetate to the mitochondrion does not have a net anaplerotic effect, as another citric acid cycle intermediate (malate) is immediately removed from the mitochondrion to be converted into cytosolic oxaloacetate, which is ultimately converted into glucose, in a process that is almost the reverse of glycolysis.
<span style="border: 3px solid blue; color:blue;font-weight: bold">59</span></p><p>The enzymes of the citric acid cycle are located in the mitochondrial matrix, with the exception of succinate dehydrogenase, which is bound to the inner mitochondrial membrane as part of Complex II. The citric acid cycle oxidizes the acetyl-CoA to carbon dioxide, and, in the process, produces reduced cofactors (three molecules of NADH and one molecule of FADH<sub>2</sub>) that are a source of electrons for the <i>electron transport chain</i>, and a molecule of GTP (that is readily converted to an ATP).
<span style="border: 3px solid blue; color:blue;font-weight: bold">60</span></p>
<h4><span class="mw-headline" id="NADH_and_FADH2:_the_electron_transport_chain">NADH and FADH<sub>2</sub>: the electron transport chain</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main articles: Electron transport chain and Oxidative phosphorylation</div>
<div class="thumb tright"><div class="thumbinner" style="width:402px;"><img alt="" class="thumbimage" data-file-height="435" data-file-width="822" decoding="async" height="212" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8b/Electron_transport_chain.svg/400px-Electron_transport_chain.svg.png" srcset width="400"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>5</h1></center></div>Electron transport chain in the mitochondrial intermembrane space</div></div></div>
<p>The redox energy from NADH and FADH<sub>2</sub> is transferred to oxygen (O<sub>2</sub>) in several steps via the electron transport chain. These energy-rich molecules are produced within the matrix via the citric acid cycle but are also produced in the cytoplasm by glycolysis. Reducing equivalents from the cytoplasm can be imported via the malate-aspartate shuttle system of antiporter proteins or feed into the electron transport chain using a glycerol phosphate shuttle. Protein complexes in the inner membrane (NADH dehydrogenase (ubiquinone), cytochrome c reductase, and cytochrome c oxidase) perform the transfer and the incremental release of energy is used to pump protons (H) into the intermembrane space. This process is efficient, but a small percentage of electrons may prematurely reduce oxygen, forming reactive oxygen species such as superoxide. This can cause oxidative stress in the mitochondria and may contribute to the decline in mitochondrial function associated with the aging process.
<span style="border: 3px solid blue; color:blue;font-weight: bold">61</span></p><p>As the proton concentration increases in the intermembrane space, a strong electrochemical gradient is established across the inner membrane. The protons can return to the matrix through the ATP synthase complex, and their potential energy is used to synthesize ATP from ADP and inorganic phosphate (P<sub>i</sub>). This process is called chemiosmosis, and was first described by Peter Mitchell who was awarded the 1978 Nobel Prize in Chemistry for his work. Later, part of the 1997 Nobel Prize in Chemistry was awarded to Paul D. Boyer and John E. Walker for their clarification of the working mechanism of ATP synthase.
<span style="border: 3px solid blue; color:blue;font-weight: bold">62</span></p>
<h4><span class="mw-headline" id="Heat_production">Heat production</span></h4>
<p>Under certain conditions, protons can re-enter the mitochondrial matrix without contributing to ATP synthesis. This process is known as <i>proton leak</i> or <i>mitochondrial uncoupling</i> and is due to the facilitated diffusion of protons into the matrix. The process results in the unharnessed potential energy of the proton electrochemical gradient being released as heat. The process is mediated by a proton channel called thermogenin, or UCP1. Thermogenin is a 33 kDa protein first discovered in 1973. Thermogenin is primarily found in brown adipose tissue, or brown fat, and is responsible for non-shivering thermogenesis. Brown adipose tissue is found in mammals, and is at its highest levels in early life and in hibernating animals. In humans, brown adipose tissue is present at birth and decreases with age.
<span style="border: 3px solid blue; color:blue;font-weight: bold">63</span></p>
<h3><span class="mw-headline" id="Storage_of_calcium_ions">Storage of calcium ions</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:402px;"><img alt="" class="thumbimage" data-file-height="331" data-file-width="648" decoding="async" height="204" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/83/Chondrocyte-_calcium_stain.jpg/400px-Chondrocyte-_calcium_stain.jpg" srcset width="400"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>6</h1></center></div>Transmission electron micrograph of a chondrocyte, stained for calcium, showing its nucleus (N) and mitochondria (M).</div></div></div>
<p>The concentrations of free calcium in the cell can regulate an array of reactions and is important for signal transduction in the cell. Mitochondria can transiently store calcium, a contributing process for the cell's homeostasis of calcium.
 In fact, their ability to rapidly take in calcium for later release makes them very good "cytosolic buffers" for calcium. The endoplasmic reticulum (ER) is the most significant storage site of calcium, and there is a significant interplay between the mitochondrion and ER with regard to calcium. The calcium is taken up into the matrix by the mitochondrial calcium uniporter on the inner mitochondrial membrane. It is primarily driven by the mitochondrial membrane potential. Release of this calcium back into the cell's interior can occur via a sodium-calcium exchange protein or via "calcium-induced-calcium-release" pathways. This can initiate calcium spikes or calcium waves with large changes in the membrane potential. These can activate a series of second messenger system proteins that can coordinate processes such as neurotransmitter release in nerve cells and release of hormones in endocrine cells.
<span style="border: 3px solid blue; color:blue;font-weight: bold">64</span></p><p>Ca influx to the mitochondrial matrix has recently been implicated as a mechanism to regulate respiratory bioenergetics by allowing the electrochemical potential across the membrane to transiently "pulse" from Î”Î¨-dominated to pH-dominated, facilitating a reduction of oxidative stress. In neurons, concomitant increases in cytosolic and mitochondrial calcium act to synchronize neuronal activity with mitochondrial energy metabolism. Mitochondrial matrix calcium levels can reach the tens of micromolar levels, which is necessary for the activation of isocitrate dehydrogenase, one of the key regulatory enzymes of the Krebs cycle.
<span style="border: 3px solid blue; color:blue;font-weight: bold">65</span></p>
<h3><span class="mw-headline" id="Additional_functions">Additional functions</span></h3>
<p>Mitochondria play a central role in many other metabolic tasks, such as:
<span style="border: 3px solid blue; color:blue;font-weight: bold">66</span></p>
<ul><li>Signaling through mitochondrial reactive oxygen species</li>
<li>Regulation of the membrane potential</li>
<li>Apoptosis-programmed cell death</li>
<li>Calcium signaling (including calcium-evoked apoptosis)</li>
<li>Regulation of cellular metabolism</li>
<li>Certain heme synthesis reactions <i>(see also: porphyrin)</i></li>
<li>Steroid synthesis.</li>
<li>Hormonal signaling  Mitochondria are sensitive and responsive to hormones, in part by the action of mitochondrial estrogen receptors (mtERs). These receptors have been found in various tissues and cell types, including brain  and heart </li></ul>
<p>Some mitochondrial functions are performed only in specific types of cells. For example, mitochondria in liver cells contain enzymes that allow them to detoxify ammonia, a waste product of protein metabolism. A mutation in the genes regulating any of these functions can result in mitochondrial diseases.
<span style="border: 3px solid blue; color:blue;font-weight: bold">67</span></p>
<h2><span class="mw-headline" id="Cellular_proliferation_regulation">Cellular proliferation regulation</span></h2>
<p>The relationship between cellular proliferation and mitochondria has been investigated using cervical cancer HeLa cells. Tumor cells require an ample amount of ATP (Adenosine triphosphate) in order to synthesize bioactive compounds such as lipids, proteins, and nucleotides for rapid cell proliferation. The majority of ATP in tumor cells is generated via the oxidative phosphorylation pathway (OxPhos). Interference with OxPhos have shown to cause cell cycle arrest suggesting that mitochondria play a role in cell proliferation. Mitochondrial ATP production is also vital for cell division in addition to other basic functions in the cell including the regulation of cell volume, solute concentration, and cellular architecture. ATP levels differ at various stages of the cell cycle suggesting that there is a relationship between the abundance of ATP and the cell's ability to enter a new cell cycle. ATP's role in the basic functions of the cell make the cell cycle sensitive to changes in the availability of mitochondrial derived ATP. The variation in ATP levels at different stages of the cell cycle support the hypothesis that mitochondria play an important role in cell cycle regulation. Although the specific mechanisms between mitochondria and the cell cycle regulation is not well understood, studies have shown that low energy cell cycle checkpoints monitor the energy capability before committing to another round of cell division.
<span style="border: 3px solid blue; color:blue;font-weight: bold">68</span></p>
<h2><span class="mw-headline" id="Genome">Genome</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" class="thumbimage" data-file-height="991" data-file-width="1040" decoding="async" height="286" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/15/Map_of_the_human_mitochondrial_genome.svg/300px-Map_of_the_human_mitochondrial_genome.svg.png" srcset width="300"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>7</h1></center></div>The circular 16,569 bp human mitochondrial genome encoding 37 genes, <i>i.e.,</i> 28 on the H-strand and 9 on the L-strand.</div></div></div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Mitochondrial DNA</div>
<p>Mitochondria contain their own genome, an indication that they are derived from bacteria through endosymbiosis. However, the ancestral endosymbiont genome has lost most of its genes so that the mitochondrial genome (mitogenome) is one of the most reduced genomes across organisms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">69</span></p><p>The <b>human</b> mitochondrial genome is a circular DNA molecule of about 16Â kilobases. It encodes 37 genes: 13 for subunits of respiratory complexes I, III, IV and V, 22 for mitochondrial tRNA (for the 20 standard amino acids, plus an extra gene for leucine and serine), and 2 for rRNA. One mitochondrion can contain two to ten copies of its DNA.
<span style="border: 3px solid blue; color:blue;font-weight: bold">70</span></p><p>As in prokaryotes, there is a very high proportion of coding DNA and an absence of repeats. Mitochondrial genes are transcribed as multigenic transcripts, which are cleaved and polyadenylated to yield mature mRNAs. Not all proteins necessary for mitochondrial function are encoded by the mitochondrial genome; most are coded by genes in the cell nucleus and the corresponding proteins are imported into the mitochondrion. The exact number of genes encoded by the nucleus and the mitochondrial genome differs between species. Most mitochondrial genomes are circular, although exceptions have been reported. In general, mitochondrial DNA lacks introns, as is the case in the human mitochondrial genome; however, introns have been observed in some eukaryotic mitochondrial DNA, such as that of yeast and protists, including <i>Dictyostelium discoideum</i>. Between protein-coding regions, tRNAs are present. During transcription, the tRNAs acquire their characteristic L-shape that gets recognized and cleaved by specific enzymes. Mitochondrial tRNA genes have different sequences from the nuclear tRNAs but lookalikes of mitochondrial tRNAs have been found in the nuclear chromosomes with high sequence similarity.
<span style="border: 3px solid blue; color:blue;font-weight: bold">71</span></p><p>In <b>animals</b>, the mitochondrial genome is typically a single circular chromosome that is approximately 16 kb long and has 37 genes. The genes, while highly conserved, may vary in location. Curiously, this pattern is not found in the human body louse (<i>Pediculus humanus</i>). Instead, this mitochondrial genome is arranged in 18 minicircular chromosomes, each of which is 3â€“4 kb long and has one to three genes. This pattern is also found in other sucking lice, but not in chewing lice. Recombination has been shown to occur between the minichromosomes. The reason for this difference is not known.
<span style="border: 3px solid blue; color:blue;font-weight: bold">72</span></p>
<h3><span class="mw-headline" id="Alternative_genetic_code">Alternative genetic code</span></h3>
<p>While slight variations on the standard genetic code had been predicted earlier, none was discovered until 1979, when researchers studying human mitochondrial genes determined that they used an alternative code. However, the mitochondria of many other eukaryotes, including most plants, use the standard code. Many slight variants have been discovered since, including various alternative mitochondrial codes. Further, the AUA, AUC, and AUU codons are all allowable start codons.
<span style="border: 3px solid blue; color:blue;font-weight: bold">73</span></p>
<table >
<caption>Exceptions to the standard genetic code in mitochondria
</caption>
<tbody><tr>
<th>Organism</th>
<th>Codon</th>
<th>Standard</th>
<th>Mitochondria
</th></tr>
<tr>
<td>Mammals
</td>
<td>AGA,Â AGG
</td>
<td>Arginine
</td>
<td>StopÂ codon
</td></tr>
<tr>
<td>Invertebrates
</td>
<td>AGA, AGG
</td>
<td>Arginine
</td>
<td>Serine
</td></tr>
<tr>
<td>Fungi
</td>
<td>CUA
</td>
<td>Leucine
</td>
<td>Threonine
</td></tr>
<tr valign="top">
<td rowspan="2">All of the above
</td>
<td>AUA
</td>
<td>Isoleucine
</td>
<td>Methionine
</td></tr>
<tr>
<td>UGA
</td>
<td>Stop codon
</td>
<td>Tryptophan
</td></tr></tbody></table><a name="table1"></a><span style='border: 3px solid #00cc00;color:#00cc00;font-size: 24px;'>1</span>
<p>Some of these differences should be regarded as pseudo-changes in the genetic code due to the phenomenon of RNA editing, which is common in mitochondria. In higher plants, it was thought that CGG encoded for tryptophan and not arginine; however, the codon in the processed RNA was discovered to be the UGG codon, consistent with the standard genetic code for tryptophan. Of note, the arthropod mitochondrial genetic code has undergone parallel evolution within a phylum, with some organisms uniquely translating AGG to lysine.
<span style="border: 3px solid blue; color:blue;font-weight: bold">74</span></p>
<h3><span class="mw-headline" id="Evolution_and_diversity">Evolution and diversity</span></h3>
<p>Mitochondrial genomes have far fewer genes than the bacteria from which they are thought to be descended. Although some have been lost altogether, many have been transferred to the nucleus, such as the respiratory complex II protein subunits. This is thought to be relatively common over evolutionary time. A few organisms, such as the <i>Cryptosporidium</i>, actually have mitochondria that lack any DNA, presumably because all their genes have been lost or transferred. In <i>Cryptosporidium</i>, the mitochondria have an altered ATP generation system that renders the parasite resistant to many classical mitochondrial inhibitors such as cyanide, azide, and atovaquone.
<span style="border: 3px solid blue; color:blue;font-weight: bold">75</span></p>
<h3><span class="mw-headline" id="Replication_and_inheritance">Replication and inheritance</span></h3>
<p>Mitochondria divide by binary fission, similar to bacterial cell division. The regulation of this division differs between eukaryotes. In many single-celled eukaryotes, their growth and division is linked to the cell cycle. For example, a single mitochondrion may divide synchronously with the nucleus. This division and segregation process must be tightly controlled so that each daughter cell receives at least one mitochondrion. In other eukaryotes (in mammals for example), mitochondria may replicate their DNA and divide mainly in response to the energy needs of the cell, rather than in phase with the cell cycle. When the energy needs of a cell are high, mitochondria grow and divide. When the energy use is low, mitochondria are destroyed or become inactive. In such examples, and in contrast to the situation in many single celled eukaryotes, mitochondria are apparently randomly distributed to the daughter cells during the division of the cytoplasm. Understanding of mitochondrial dynamics, which is described as the balance between mitochondrial fusion and fission, has revealed that functional and structural alterations in mitochondrial morphology are important factors in pathologies associated with several disease conditions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">76</span></p><p>The hypothesis of mitochondrial binary fission has relied on the visualization by fluorescence microscopy and conventional transmission electron microscopy (TEM). The resolution of fluorescence microscopy(~200Â nm) is insufficient to distinguish structural details, such as double mitochondrial membrane in mitochondrial division or even to distinguish individual mitochondria when several are close together. Conventional TEM has also some technical limitations in verifying mitochondrial division. Cryo-electron tomography was recently used to visualize mitochondrial division in frozen hydrated intact cells. It revealed that mitochondria divide by budding.
<span style="border: 3px solid blue; color:blue;font-weight: bold">77</span></p><p>An individual's mitochondrial genes are not inherited by the same mechanism as nuclear genes. Typically, the mitochondria are inherited from one parent only. In humans, when an egg cell is fertilized by a sperm, the egg nucleus and sperm nucleus each contribute equally to the genetic makeup of the zygote nucleus. In contrast, the mitochondria, and therefore the mitochondrial DNA, usually come from the egg only. The sperm's mitochondria enter the egg, but do not contribute genetic information to the embryo. Instead, paternal mitochondria are marked with ubiquitin to select them for later destruction inside the embryo. The egg cell contains relatively few mitochondria, but it is these mitochondria that survive and divide to populate the cells of the adult organism. Mitochondria are, therefore, in most cases inherited only from mothers, a pattern known as maternal inheritance. This mode is seen in most organisms, including the majority of animals. However, mitochondria in some species can sometimes be inherited paternally. This is the norm among certain coniferous plants, although not in pine trees and yews. For Mytilids, paternal inheritance only occurs within males of the species. It has been suggested that it occurs at a very low level in humans. There is a recent suggestion that mitochondria that shorten male lifespan stay in the system because they are inherited only through the mother. By contrast, natural selection weeds out mitochondria that reduce female survival as such mitochondria are less likely to be passed on to the next generation. Therefore, it is suggested that human females and female animals tend to live longer than males. The authors claim that this is a partial explanation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">78</span></p><p>Uniparental inheritance leads to little opportunity for genetic recombination between different lineages of mitochondria, although a single mitochondrion can contain 2â€“10 copies of its DNA. For this reason, mitochondrial DNA is usually thought to reproduce by binary fission. What recombination does take place maintains genetic integrity rather than maintaining diversity. However, there are studies showing evidence of recombination in mitochondrial DNA. It is clear that the enzymes necessary for recombination are present in mammalian cells. Further, evidence suggests that animal mitochondria can undergo recombination. The data are a bit more controversial in humans, although indirect evidence of recombination exists. If recombination does not occur, the whole mitochondrial DNA sequence represents a single haplotype, which makes it useful for studying the evolutionary history of populations.
<span style="border: 3px solid blue; color:blue;font-weight: bold">79</span></p><p>Entities undergoing uniparental inheritance and with little to no recombination may be expected to be subject to Muller's ratchet, the inexorable accumulation of deleterious mutations until functionality is lost. Animal populations of mitochondria avoid this buildup through a developmental process known as the mtDNA bottleneck. The bottleneck exploits stochastic processes in the cell to increase in the cell-to-cell variability in mutant load as an organism develops: a single egg cell with some proportion of mutant mtDNA thus produces an embryo where different cells have different mutant loads. Cell-level selection may then act to remove those cells with more mutant mtDNA, leading to a stabilisation or reduction in mutant load between generations. The mechanism underlying the bottleneck is debated, with a recent mathematical and experimental metastudy providing evidence for a combination of random partitioning of mtDNAs at cell divisions and random turnover of mtDNA molecules within the cell.
<span style="border: 3px solid blue; color:blue;font-weight: bold">80</span></p>
<h3><span class="mw-headline" id="DNA_repair">DNA repair</span></h3>
<p>Mitochondria can repair oxidative DNA damage by mechanisms that are analogous to those occurring in the cell nucleus.  The proteins that are employed in mtDNA repair are encoded by nuclear genes, and are translocated to the mitochondria.  The DNA repair pathways in mammalian mitochondria include base excision repair, double-strand break repair, direct reversal and mismatch repair.  Also DNA damages may be bypassed, rather than repaired, by translesion synthesis.
<span style="border: 3px solid blue; color:blue;font-weight: bold">81</span></p><p>Of the several DNA repair process in mitochondria, the base excision repair pathway is the one that has been most comprehensively studied.  Base excision repair is carried out by a sequence of enzymatic catalyzed steps that include recognition and excision of a damaged DNA base, removal of the resulting abasic site, end processing, gap filling and ligation.  A common damage in mtDNA that is repaired by base excision repair is 8-oxoguanine produced by the oxidation of guanine.
<span style="border: 3px solid blue; color:blue;font-weight: bold">82</span></p><p>Double-strand breaks can be repaired by homologous recombinational repair in both mammalian mtDNA and plant mtDNA. Double-strand breaks in mtDNA can also be repaired by microhomology-mediated end joining.  Although there is evidence for the repair processes of direct reversal and mismatch repair in mtDNA, these processes are still not well characterized.
<span style="border: 3px solid blue; color:blue;font-weight: bold">83</span></p>
<h2><span class="mw-headline" id="Population_genetic_studies">Population genetic studies</span></h2>
<div class="hatnote navigation-not-searchable" role="note">Main article: Human mitochondrial genetics</div>
<p>The near-absence of genetic recombination in mitochondrial DNA makes it a useful source of information for scientists involved in population genetics and evolutionary biology. Because all the mitochondrial DNA is inherited as a single unit, or haplotype, the relationships between mitochondrial DNA from different individuals can be represented as a gene tree. Patterns in these gene trees can be used to infer the evolutionary history of populations. The classic example of this is in human evolutionary genetics, where the molecular clock can be used to provide a recent date for mitochondrial Eve. This is often interpreted as strong support for a recent modern human expansion out of Africa. Another human example is the sequencing of mitochondrial DNA from Neanderthal bones. The relatively large evolutionary distance between the mitochondrial DNA sequences of Neanderthals and living humans has been interpreted as evidence for the lack of interbreeding between Neanderthals and anatomically modern humans.
<span style="border: 3px solid blue; color:blue;font-weight: bold">84</span></p><p>However, mitochondrial DNA reflects only the history of the females in a population and so may not represent the history of the population as a whole. This can be partially overcome by the use of paternal genetic sequences, such as the non-recombining region of the Y-chromosome. In a broader sense, only studies that also include nuclear DNA can provide a comprehensive evolutionary history of a population.
<span style="border: 3px solid blue; color:blue;font-weight: bold">85</span></p><p>Recent measurements of the molecular clock for mitochondrial DNA reported a value of 1 mutation every 7884 years dating back to the most recent common ancestor of humans and apes, which is consistent with estimates of mutation rates of autosomal DNA (10 per base per generation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">86</span></p>
<h2><span class="mw-headline" id="Dysfunction_and_disease">Dysfunction and disease</span></h2>
<h3><span class="mw-headline" id="Mitochondrial_diseases">Mitochondrial diseases</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Mitochondrial disease</div>
<p>Damage and subsequent dysfunction in mitochondria is an important factor in a range of human diseases due to their influence in cell metabolism. Mitochondrial disorders often present themselves as neurological disorders, including autism. They can also manifest as myopathy, diabetes, multiple endocrinopathy, and a variety of other systemic disorders. Diseases caused by mutation in the mtDNA include Kearns-Sayre syndrome, MELAS syndrome and Leber's hereditary optic neuropathy. In the vast majority of cases, these diseases are transmitted by a female to her children, as the zygote derives its mitochondria and hence its mtDNA from the ovum. Diseases such as Kearns-Sayre syndrome, Pearson syndrome, and progressive external ophthalmoplegia are thought to be due to large-scale mtDNA rearrangements, whereas other diseases such as MELAS syndrome, Leber's hereditary optic neuropathy, myoclonic epilepsy with ragged red fibers (MERRF), and others are due to point mutations in mtDNA.
<span style="border: 3px solid blue; color:blue;font-weight: bold">87</span></p><p>In other diseases, defects in nuclear genes lead to dysfunction of mitochondrial proteins. This is the case in Friedreich's ataxia, hereditary spastic paraplegia, and Wilson's disease. These diseases are inherited in a dominance relationship, as applies to most other genetic diseases. A variety of disorders can be caused by nuclear mutations of oxidative phosphorylation enzymes, such as coenzyme Q10 deficiency and Barth syndrome. Environmental influences may interact with hereditary predispositions and cause mitochondrial disease. For example, there may be a link between pesticide exposure and the later onset of Parkinson's disease. Other pathologies with etiology involving mitochondrial dysfunction include schizophrenia, bipolar disorder, dementia, Alzheimer's disease, Parkinson's disease, epilepsy, stroke, cardiovascular disease, chronic fatigue syndrome, retinitis pigmentosa, and diabetes mellitus.
<span style="border: 3px solid blue; color:blue;font-weight: bold">88</span></p><p>Mitochondria-mediated oxidative stress plays a role in cardiomyopathy in Type 2 diabetics. Increased fatty acid delivery to the heart increases fatty acid uptake by cardiomyocytes, resulting in increased fatty acid oxidation in these cells. This process increases the reducing equivalents available to the electron transport chain of the mitochondria, ultimately increasing reactive oxygen species (ROS) production. ROS increases uncoupling proteins (UCPs) and potentiate proton leakage through the adenine nucleotide translocator (ANT), the combination of which uncouples the mitochondria. Uncoupling then increases oxygen consumption by the mitochondria, compounding the increase in fatty acid oxidation. This creates a vicious cycle of uncoupling; furthermore, even though oxygen consumption increases, ATP synthesis does not increase proportionally because the mitochondria is uncoupled. Less ATP availability ultimately results in an energy deficit presenting as reduced cardiac efficiency and contractile dysfunction. To compound the problem, impaired sarcoplasmic reticulum calcium release and reduced mitochondrial reuptake limits peak cytosolic levels of the important signaling ion during muscle contraction. The decreased intra-mitochondrial calcium concentration increases dehydrogenase activation and ATP synthesis. So in addition to lower ATP synthesis due to fatty acid oxidation, ATP synthesis is impaired by poor calcium signaling as well, causing cardiac problems for diabetics.
<span style="border: 3px solid blue; color:blue;font-weight: bold">89</span></p>
<h3><span class="mw-headline" id="Possible_relationships_to_aging">Possible relationships to aging</span></h3>
<p>Given the role of mitochondria as the cell's powerhouse, there may be some leakage of the high-energy electrons in the respiratory chain to form reactive oxygen species. This was thought to result in significant oxidative stress in the mitochondria with high mutation rates of mitochondrial DNA (mtDNA). Hypothesized links between aging and oxidative stress are not new and were proposed in 1956, which was later refined into the mitochondrial free radical theory of aging. A vicious cycle was thought to occur, as oxidative stress leads to mitochondrial DNA mutations, which can lead to enzymatic abnormalities and further oxidative stress.
<span style="border: 3px solid blue; color:blue;font-weight: bold">90</span></p><p>A number of changes can occur to mitochondria during the aging process. Tissues from elderly patients show a decrease in enzymatic activity of the proteins of the respiratory chain. However, mutated mtDNA can only be found in about 0.2% of very old cells. Large deletions in the mitochondrial genome have been hypothesized to lead to high levels of oxidative stress and neuronal death in Parkinson's disease.
<span style="border: 3px solid blue; color:blue;font-weight: bold">91</span></p>
<h2><span class="mw-headline" id="In_popular_culture">In popular culture</span></h2>
<p>Madeleine L'Engle's 1973 science fantasy novel <i>A Wind in the Door</i> prominently features the mitochondria of main character Charles Wallace Murry, as being inhabited by creatures known as the farandolae. The novel also features other characters travelling inside one of Murry's mitochondria.
<span style="border: 3px solid blue; color:blue;font-weight: bold">92</span></p><p>The 1995 horror fiction novel <i>Parasite Eve</i> by Hideaki Sena depicts mitochondria as having some consciousness and mind control abilities, attempting to use these to overtake eukaryotes as the dominant life form. This text was adapted into an eponymous film, video game, and video game sequel all involving a similar premise.
<span style="border: 3px solid blue; color:blue;font-weight: bold">93</span></p><p>In the <i>Star Wars</i> franchise, microorganisms referred to as "midi-chlorians" give some characters the ability to sense and use the Force. George Lucas, director of the 1999 film <i>Star Wars: Episode I â€“ The Phantom Menace</i>, in which midi-chlorians were introduced, described them as "a loose depiction of mitochondria". The non-fictional bacteria genus <i>Midichloria</i> was later named after the midi-chlorians of <i>Star Wars</i>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">94</span></p><p>As a result of the mitochondrion's prominence in modern science education, the phrase "the mitochondria is the powerhouse of the cell" became a popular Internet meme. The meme is used to imply that secondary education places an insufficient focus on life skills, compared to academic knowledge such as the role of the mitochondrion, which has been considered comparatively impractical.
<span style="border: 3px solid blue; color:blue;font-weight: bold">95</span></p>
<script>function offsetAnchor() {if(location.hash.length !== 0) {window.scrollTo(window.scrollX, window.scrollY - 250);}}window.addEventListener("hashchange", offsetAnchor);window.setTimeout(offsetAnchor, 1);</script>