<!DOCTYPE html>
<html class="client-nojs" dir="ltr" lang="en">
<head>
<meta charset="utf-8"/>
<title>Evidence of common descent - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Evidence_of_common_descent","wgTitle":"Evidence of common descent","wgCurRevisionId":862216718,"wgRevisionId":862216718,"wgArticleId":2339577,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["Webarchive template webcite links","Webarchive template wayback links","CS1 maint: Multiple names: authors list","CS1 maint: Uses authors parameter","CS1: Julian–Gregorian uncertainty","Use dmy dates from January 2018","All articles with unsourced statements","Articles with unsourced statements from October 2014","Articles with attributed pull quotes","Evolutionary biology"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Evidence_of_common_descent","wgRelevantArticleId":2339577,"wgRequestId":"W7TVYArAIFAAAHY4kf0AAAAI","wgCSPNonce":false,"wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgWikiEditorEnabledModules":[],"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":true,"wgPopupsShouldSendModuleToUser":true,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgMFExpandAllSectionsUserOption":true,"wgMFEnableFontChanger":true,"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgWikibaseItemId":"Q1075815","wgScoreNoteLanguages":{"arabic":"العربية","catalan":"català","deutsch":"Deutsch","english":"English","espanol":"español","italiano":"italiano","nederlands":"Nederlands","norsk":"norsk","portugues":"português","suomi":"suomi","svenska":"svenska","vlaams":"West-Vlams"},"wgScoreDefaultNoteLanguage":"nederlands","wgCentralAuthMobileDomain":false,"wgCodeMirrorEnabled":true,"wgVisualEditorToolbarScrollOffset":0,"wgVisualEditorUnsupportedEditParams":["undo","undoafter","veswitched"],"wgEditSubmitButtonLabelPublish":true});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready","user":"ready","user.options":"ready","user.tokens":"loading","ext.cite.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.toc.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","ext.wikimediaBadges":"ready","ext.3d.styles":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready"});mw.loader.implement("user.tokens@0tffind",function($,jQuery,require,module){/*@nomin*/mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});
});RLPAGEMODULES=["ext.cite.a11y","site","mediawiki.page.startup","mediawiki.user","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.popups","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging.subscriber","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.compactlinks","ext.uls.interface","ext.3d","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"];mw.loader.load(RLPAGEMODULES);});</script>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.3d.styles%7Cext.cite.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cmediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.interface%7Cmediawiki.toc.styles%7Cskins.vector.styles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<script async="" src="/w/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=vector"></script>
<meta content="" name="ResourceLoaderDynamicStyles"/>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=ext.gadget.charinsert-styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<link href="/w/load.php?debug=false&amp;lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector" rel="stylesheet"/>
<meta content="MediaWiki 1.32.0-wmf.23" name="generator"/>
<meta content="origin" name="referrer"/>
<meta content="origin-when-crossorigin" name="referrer"/>
<meta content="origin-when-cross-origin" name="referrer"/>
<link href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Evidence_of_common_descent" rel="alternate"/>
<link href="/w/index.php?title=Evidence_of_common_descent&amp;action=edit" rel="alternate" title="Edit this page" type="application/x-wiki"/>
<link href="/w/index.php?title=Evidence_of_common_descent&amp;action=edit" rel="edit" title="Edit this page"/>
<link href="/static/apple-touch/wikipedia.png" rel="apple-touch-icon"/>
<link href="/static/favicon/wikipedia.ico" rel="shortcut icon"/>
<link href="/w/opensearch_desc.php" rel="search" title="Wikipedia (en)" type="application/opensearchdescription+xml"/>
<link href="//en.wikipedia.org/w/api.php?action=rsd" rel="EditURI" type="application/rsd+xml"/>
<link href="//creativecommons.org/licenses/by-sa/3.0/" rel="license"/>
<link href="https://en.wikipedia.org/wiki/Evidence_of_common_descent" rel="canonical"/>
<link href="//login.wikimedia.org" rel="dns-prefetch"/>
<link href="//meta.wikimedia.org" rel="dns-prefetch"/>
<!--[if lt IE 9]><script src="/w/load.php?debug=false&amp;lang=en&amp;modules=html5shiv&amp;only=scripts&amp;skin=vector&amp;sync=1"></script><![endif]-->
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject page-Evidence_of_common_descent rootpage-Evidence_of_common_descent skin-vector action-view"> <div class="noprint" id="mw-page-base"></div>
<div class="noprint" id="mw-head-base"></div>
<div class="mw-body" id="content" role="main">

<div class="mw-body-content" id="siteNotice"><!-- CentralNotice --></div><div class="mw-indicators mw-body-content">
</div>
<h1 class="firstHeading" id="firstHeading" lang="en">Evidence of common descent</h1> <div class="mw-body-content" id="bodyContent">
<div class="noprint" id="siteSub">From Wikipedia, the free encyclopedia</div> <div id="contentSub"></div>
<div id="jump-to-nav"></div> Jump to navigation
Jump to search
<div class="mw-content-ltr" dir="ltr" id="mw-content-text" lang="en"><div class="mw-parser-output"><p class="mw-empty-elt">
<span style="border: 3px solid blue; color:blue;font-weight: bold">1</span></p>

<p><b>Evidence of common descent</b> of living organisms has been discovered by scientists researching in a variety of disciplines over many decades, demonstrating that all life on Earth comes from a single ancestor. This forms an important part of the evidence on which evolutionary theory rests, demonstrates that evolution does occur, and illustrates the processes that created Earth's biodiversity. It supports the modern evolutionary synthesis—the current scientific theory that explains how and why life changes over time. Evolutionary biologists document evidence of common descent, all the way back to the last universal common ancestor, by developing testable predictions, testing hypotheses, and constructing theories that illustrate and describe its causes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">2</span></p><p>Comparison of the DNA genetic sequences of organisms has revealed that organisms that are phylogenetically close have a higher degree of DNA sequence similarity than organisms that are phylogenetically distant. Genetic fragments such as pseudogenes, regions of DNA that are orthologous to a gene in a related organism, but are no longer active and appear to be undergoing a steady process of degeneration from cumulative mutations support common descent alongside the universal biochemical organization and molecular variance patterns found in all organisms. Additional genetic information conclusively supports the relatedness of life and has allowed scientists (since the discovery of DNA) to develop phylogenetic trees: a construction of organisms evolutionary relatedness. It has also led to the development of molecular clock techniques to date taxon divergence times and to calibrate these with the fossil record.
<span style="border: 3px solid blue; color:blue;font-weight: bold">3</span></p><p>Fossils are important for estimating when various lineages developed in geologic time. As fossilization is an uncommon occurrence, usually requiring hard body parts and death near a site where sediments are being deposited, the fossil record only provides sparse and intermittent information about the evolution of life. Evidence of organisms prior to the development of hard body parts such as shells, bones and teeth is especially scarce, but exists in the form of ancient microfossils, as well as impressions of various soft-bodied organisms. The comparative study of the anatomy of groups of animals shows structural features that are fundamentally similar (homologous), demonstrating phylogenetic and ancestral relationships with other organisms, most especially when compared with fossils of ancient extinct organisms. Vestigial structures and comparisons in embryonic development are largely a contributing factor in anatomical resemblance in concordance with common descent. Since metabolic processes do not leave fossils, research into the evolution of the basic cellular processes is done largely by comparison of existing organisms' physiology and biochemistry. Many lineages diverged at different stages of development, so it is possible to determine when certain metabolic processes appeared by comparing the traits of the descendants of a common ancestor.
<span style="border: 3px solid blue; color:blue;font-weight: bold">4</span></p><p>Evidence from animal coloration was gathered by some of Darwin's contemporaries; camouflage, mimicry, and warning coloration are all readily explained by natural selection. Special cases like the seasonal changes in the plumage of the ptarmigan, camouflaging it against snow in winter and against brown moorland in summer provide compelling evidence that selection is at work. Further evidence comes from the field of biogeography because evolution with common descent provides the best and most thorough explanation for a variety of facts concerning the geographical distribution of plants and animals across the world. This is especially obvious in the field of insular biogeography. Combined with the well-established geological theory of plate tectonics, common descent provides a way to combine facts about the current distribution of species with evidence from the fossil record to provide a logically consistent explanation of how the distribution of living organisms has changed over time.
<span style="border: 3px solid blue; color:blue;font-weight: bold">5</span></p><p>The development and spread of antibiotic resistant bacteria provides evidence that evolution due to natural selection is an ongoing process in the natural world. Natural selection is ubiquitous in all research pertaining to evolution, taking note of the fact that all of the following examples in each section of the article document the process. Alongside this are observed instances of the separation of populations of species into sets of new species (speciation). Speciation has been observed in the lab and in nature. Multiple forms of such have been described and documented as examples for individual modes of speciation. Furthermore, evidence of common descent extends from direct laboratory experimentation with the selective breeding of organisms—historically and currently—and other controlled experiments involving many of the topics in the article. This article summarizes the varying disciplines that provide the evidence for evolution and the common descent of all life on Earth, accompanied by numerous and specialized examples, indicating a compelling consilience of evidence.
<span style="border: 3px solid blue; color:blue;font-weight: bold">6</span></p>
<div class="toclimit-3"><div class="toc" id="toc"><input class="toctogglecheckbox" id="toctogglecheckbox" role="button" style="display:none" type="checkbox"/><div class="toctitle" dir="ltr" lang="en"><h2>Contents</h2><span class="toctogglespan"><label class="toctogglelabel" for="toctogglecheckbox"></label></span></div>
<ul>
<li class="toclevel-1 tocsection-1"><span class="tocnumber">1</span> <span class="toctext">Evidence from comparative physiology and biochemistry</span>
<ul>
<li class="toclevel-2 tocsection-2"><span class="tocnumber">1.1</span> <span class="toctext">Genetics</span>
<ul>
<li class="toclevel-3 tocsection-3"><span class="tocnumber">1.1.1</span> <span class="toctext">Universal biochemical organisation and molecular variance patterns</span></li>
<li class="toclevel-3 tocsection-4"><span class="tocnumber">1.1.2</span> <span class="toctext">DNA sequencing</span></li>
<li class="toclevel-3 tocsection-5"><span class="tocnumber">1.1.3</span> <span class="toctext">Proteins</span></li>
<li class="toclevel-3 tocsection-6"><span class="tocnumber">1.1.4</span> <span class="toctext">Pseudogenes</span></li>
<li class="toclevel-3 tocsection-7"><span class="tocnumber">1.1.5</span> <span class="toctext">Other mechanisms</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-8"><span class="tocnumber">1.2</span> <span class="toctext">Specific examples from comparative physiology and biochemistry</span>
<ul>
<li class="toclevel-3 tocsection-9"><span class="tocnumber">1.2.1</span> <span class="toctext">Chromosome 2 in humans</span></li>
<li class="toclevel-3 tocsection-10"><span class="tocnumber">1.2.2</span> <span class="toctext">Cytochrome c and b</span></li>
<li class="toclevel-3 tocsection-11"><span class="tocnumber">1.2.3</span> <span class="toctext">Endogenous retroviruses</span></li>
<li class="toclevel-3 tocsection-12"><span class="tocnumber">1.2.4</span> <span class="toctext">Recent African origin of modern humans</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-13"><span class="tocnumber">2</span> <span class="toctext">Evidence from comparative anatomy</span>
<ul>
<li class="toclevel-2 tocsection-14"><span class="tocnumber">2.1</span> <span class="toctext">Atavisms</span></li>
<li class="toclevel-2 tocsection-15"><span class="tocnumber">2.2</span> <span class="toctext">Evolutionary developmental biology and embryonic development</span></li>
<li class="toclevel-2 tocsection-16"><span class="tocnumber">2.3</span> <span class="toctext">Homologous structures and divergent (adaptive) evolution</span></li>
<li class="toclevel-2 tocsection-17"><span class="tocnumber">2.4</span> <span class="toctext">Nested hierarchies and classification</span>
<ul>
<li class="toclevel-3 tocsection-18"><span class="tocnumber">2.4.1</span> <span class="toctext">Evolutionary trees</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-19"><span class="tocnumber">2.5</span> <span class="toctext">Vestigial structures</span></li>
<li class="toclevel-2 tocsection-20"><span class="tocnumber">2.6</span> <span class="toctext">Specific examples from comparative anatomy</span>
<ul>
<li class="toclevel-3 tocsection-21"><span class="tocnumber">2.6.1</span> <span class="toctext">Insect mouthparts and appendages</span></li>
<li class="toclevel-3 tocsection-22"><span class="tocnumber">2.6.2</span> <span class="toctext">Pelvic structure of dinosaurs</span></li>
<li class="toclevel-3 tocsection-23"><span class="tocnumber">2.6.3</span> <span class="toctext">Pentadactyl limb</span></li>
<li class="toclevel-3 tocsection-24"><span class="tocnumber">2.6.4</span> <span class="toctext">Recurrent laryngeal nerve in giraffes</span></li>
<li class="toclevel-3 tocsection-25"><span class="tocnumber">2.6.5</span> <span class="toctext">Route of the vas deferens</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-26"><span class="tocnumber">3</span> <span class="toctext">Evidence from paleontology</span>
<ul>
<li class="toclevel-2 tocsection-27"><span class="tocnumber">3.1</span> <span class="toctext">Fossil record</span>
<ul>
<li class="toclevel-3 tocsection-28"><span class="tocnumber">3.1.1</span> <span class="toctext">Extent of the fossil record</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-29"><span class="tocnumber">3.2</span> <span class="toctext">Limitations</span></li>
<li class="toclevel-2 tocsection-30"><span class="tocnumber">3.3</span> <span class="toctext">Specific examples from paleontology</span>
<ul>
<li class="toclevel-3 tocsection-31"><span class="tocnumber">3.3.1</span> <span class="toctext">Evolution of the horse</span></li>
<li class="toclevel-3 tocsection-32"><span class="tocnumber">3.3.2</span> <span class="toctext">Transition from fish to amphibians</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-33"><span class="tocnumber">4</span> <span class="toctext">Evidence from biogeography</span>
<ul>
<li class="toclevel-2 tocsection-34"><span class="tocnumber">4.1</span> <span class="toctext">Continental distribution</span></li>
<li class="toclevel-2 tocsection-35"><span class="tocnumber">4.2</span> <span class="toctext">Island biogeography</span>
<ul>
<li class="toclevel-3 tocsection-36"><span class="tocnumber">4.2.1</span> <span class="toctext">Types of species found on islands</span></li>
<li class="toclevel-3 tocsection-37"><span class="tocnumber">4.2.2</span> <span class="toctext">Endemism</span></li>
<li class="toclevel-3 tocsection-38"><span class="tocnumber">4.2.3</span> <span class="toctext">Adaptive radiations</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-39"><span class="tocnumber">4.3</span> <span class="toctext">Ring species</span></li>
<li class="toclevel-2 tocsection-40"><span class="tocnumber">4.4</span> <span class="toctext">Specific examples from biogeography</span>
<ul>
<li class="toclevel-3 tocsection-41"><span class="tocnumber">4.4.1</span> <span class="toctext">Distribution of <i>Glossopteris</i></span></li>
<li class="toclevel-3 tocsection-42"><span class="tocnumber">4.4.2</span> <span class="toctext">Metatherian distribution</span></li>
<li class="toclevel-3 tocsection-43"><span class="tocnumber">4.4.3</span> <span class="toctext">Migration, isolation, and distribution of the camel</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-44"><span class="tocnumber">5</span> <span class="toctext">Evidence from selection</span>
<ul>
<li class="toclevel-2 tocsection-45"><span class="tocnumber">5.1</span> <span class="toctext">Artificial selection and experimental evolution</span></li>
<li class="toclevel-2 tocsection-46"><span class="tocnumber">5.2</span> <span class="toctext">Invertebrates</span>
<ul>
<li class="toclevel-3 tocsection-47"><span class="tocnumber">5.2.1</span> <span class="toctext">Historical lead tolerance in <i>Daphnia</i></span></li>
<li class="toclevel-3 tocsection-48"><span class="tocnumber">5.2.2</span> <span class="toctext">Peppered moths</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-49"><span class="tocnumber">5.3</span> <span class="toctext">Microbes</span>
<ul>
<li class="toclevel-3 tocsection-50"><span class="tocnumber">5.3.1</span> <span class="toctext">Antimicrobial resistance</span></li>
<li class="toclevel-3 tocsection-51"><span class="tocnumber">5.3.2</span> <span class="toctext">Nylon-eating bacteria</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-52"><span class="tocnumber">5.4</span> <span class="toctext">Plants and fungi</span>
<ul>
<li class="toclevel-3 tocsection-53"><span class="tocnumber">5.4.1</span> <span class="toctext">Monkeyflower radiation</span></li>
<li class="toclevel-3 tocsection-54"><span class="tocnumber">5.4.2</span> <span class="toctext">Radiotrophic fungi</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-55"><span class="tocnumber">5.5</span> <span class="toctext">Vertebrates</span>
<ul>
<li class="toclevel-3 tocsection-56"><span class="tocnumber">5.5.1</span> <span class="toctext">Guppies</span></li>
<li class="toclevel-3 tocsection-57"><span class="tocnumber">5.5.2</span> <span class="toctext">Humans</span></li>
<li class="toclevel-3 tocsection-58"><span class="tocnumber">5.5.3</span> <span class="toctext">Italian wall lizards</span></li>
<li class="toclevel-3 tocsection-59"><span class="tocnumber">5.5.4</span> <span class="toctext">PAH resistance in killifish</span></li>
<li class="toclevel-3 tocsection-60"><span class="tocnumber">5.5.5</span> <span class="toctext">PCB resistance in codfish</span></li>
<li class="toclevel-3 tocsection-61"><span class="tocnumber">5.5.6</span> <span class="toctext">Urban wildlife</span></li>
<li class="toclevel-3 tocsection-62"><span class="tocnumber">5.5.7</span> <span class="toctext">White Sands lizards</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-63"><span class="tocnumber">6</span> <span class="toctext">Evidence from speciation</span>
<ul>
<li class="toclevel-2 tocsection-64"><span class="tocnumber">6.1</span> <span class="toctext">Fossils</span>
<ul>
<li class="toclevel-3 tocsection-65"><span class="tocnumber">6.1.1</span> <span class="toctext"><i>Globorotalia</i></span></li>
<li class="toclevel-3 tocsection-66"><span class="tocnumber">6.1.2</span> <span class="toctext">Radiolaria</span></li>
<li class="toclevel-3 tocsection-67"><span class="tocnumber">6.1.3</span> <span class="toctext"><i>Rhizosolenia</i></span></li>
<li class="toclevel-3 tocsection-68"><span class="tocnumber">6.1.4</span> <span class="toctext"><i>Turborotalia</i></span></li>
<li class="toclevel-3 tocsection-69"><span class="tocnumber">6.1.5</span> <span class="toctext">Vertebrates</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-70"><span class="tocnumber">6.2</span> <span class="toctext">Invertebrates</span>
<ul>
<li class="toclevel-3 tocsection-71"><span class="tocnumber">6.2.1</span> <span class="toctext"><i>Drosophila melanogaster</i></span></li>
<li class="toclevel-3 tocsection-72"><span class="tocnumber">6.2.2</span> <span class="toctext">Gall wasps</span></li>
<li class="toclevel-3 tocsection-73"><span class="tocnumber">6.2.3</span> <span class="toctext">Hawthorn fly</span></li>
<li class="toclevel-3 tocsection-74"><span class="tocnumber">6.2.4</span> <span class="toctext">London Underground mosquito</span></li>
<li class="toclevel-3 tocsection-75"><span class="tocnumber">6.2.5</span> <span class="toctext">Snapping shrimp and the isthmus of Panama</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-76"><span class="tocnumber">6.3</span> <span class="toctext">Plants</span>
<ul>
<li class="toclevel-3 tocsection-77"><span class="tocnumber">6.3.1</span> <span class="toctext"><i>Mimulus peregrinus</i></span></li>
<li class="toclevel-3 tocsection-78"><span class="tocnumber">6.3.2</span> <span class="toctext"><i>Raphanobrassica</i></span></li>
<li class="toclevel-3 tocsection-79"><span class="tocnumber">6.3.3</span> <span class="toctext"><i>Senecio</i> (groundsel)</span></li>
<li class="toclevel-3 tocsection-80"><span class="tocnumber">6.3.4</span> <span class="toctext">Thale cress</span></li>
<li class="toclevel-3 tocsection-81"><span class="tocnumber">6.3.5</span> <span class="toctext"><i>Tragopogon</i> (salsify)</span></li>
</ul>
</li>
<li class="toclevel-2 tocsection-82"><span class="tocnumber">6.4</span> <span class="toctext">Vertebrates</span>
<ul>
<li class="toclevel-3 tocsection-83"><span class="tocnumber">6.4.1</span> <span class="toctext">Blackcap</span></li>
<li class="toclevel-3 tocsection-84"><span class="tocnumber">6.4.2</span> <span class="toctext">Mollies</span></li>
<li class="toclevel-3 tocsection-85"><span class="tocnumber">6.4.3</span> <span class="toctext">Polar bear</span></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1 tocsection-86"><span class="tocnumber">7</span> <span class="toctext">Evidence from coloration</span>
<ul>
<li class="toclevel-2 tocsection-87"><span class="tocnumber">7.1</span> <span class="toctext">Mimicry and aposematism</span></li>
<li class="toclevel-2 tocsection-88"><span class="tocnumber">7.2</span> <span class="toctext">Camouflage</span></li>
</ul>
</li>
<li class="toclevel-1 tocsection-89"><span class="tocnumber">8</span> <span class="toctext">Evidence from mathematical modeling</span></li>
<li class="toclevel-1 tocsection-90"><span class="tocnumber">9</span> <span class="toctext">See also</span></li>
<li class="toclevel-1 tocsection-91"><span class="tocnumber">10</span> <span class="toctext">References</span></li>
<li class="toclevel-1 tocsection-92"><span class="tocnumber">11</span> <span class="toctext">Sources</span></li>
<li class="toclevel-1 tocsection-93"><span class="tocnumber">12</span> <span class="toctext">External links</span></li>
</ul>
</div>
</div>
<h2><span class="mw-headline" id="Evidence_from_comparative_physiology_and_biochemistry">Evidence from comparative physiology and biochemistry</span></h2>
<div class="hatnote navigation-not-searchable" role="note">See also: Archaeogenetics, Common descent, Last universal ancestor, Most recent common ancestor, and Universal Code (Biology)</div>
<h3><span class="mw-headline" id="Genetics">Genetics</span></h3>
<p>One of the strongest evidences for common descent comes from gene sequences. Comparative sequence analysis examines the relationship between the DNA sequences of different species, producing several lines of evidence that confirm Darwin's original hypothesis of common descent. If the hypothesis of common descent is true, then species that share a common ancestor inherited that ancestor's DNA sequence, as well as mutations unique to that ancestor. More closely related species have a greater fraction of identical sequence and shared substitutions compared to more distantly related species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">7</span></p>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" class="thumbimage" data-file-height="821" data-file-width="1235" height="146" src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/HMS_Beagle_by_Conrad_Martens.jpg/220px-HMS_Beagle_by_Conrad_Martens.jpg" srcset width="220"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>1</h1></center></div><b>Figure 1a:</b> While on board HMS <i>Beagle</i>, Charles Darwin collected numerous specimens, many new to science, which supported his later theory of evolution by natural selection.</div></div></div>
<p>The simplest and most powerful evidence is provided by phylogenetic reconstruction. Such reconstructions, especially when done using slowly evolving protein sequences, are often quite robust and can be used to reconstruct a great deal of the evolutionary history of modern organisms (and even in some instances of the evolutionary history of extinct organisms, such as the recovered gene sequences of mammoths or Neanderthals). These reconstructed phylogenies recapitulate the relationships established through morphological and biochemical studies. The most detailed reconstructions have been performed on the basis of the mitochondrial genomes shared by all eukaryotic organisms, which are short and easy to sequence; the broadest reconstructions have been performed either using the sequences of a few very ancient proteins or by using ribosomal RNA sequence.
<span style="border: 3px solid blue; color:blue;font-weight: bold">8</span></p><p>Phylogenetic relationships extend to a wide variety of nonfunctional sequence elements, including repeats, transposons, pseudogenes, and mutations in protein-coding sequences that do not change the amino-acid sequence. While a minority of these elements might later be found to harbor function, in aggregate they demonstrate that identity must be the product of common descent rather than common function.
<span style="border: 3px solid blue; color:blue;font-weight: bold">9</span></p>
<h4><span class="mw-headline" id="Universal_biochemical_organisation_and_molecular_variance_patterns">Universal biochemical organisation and molecular variance patterns</span></h4>
<p>All known extant (surviving) organisms are based on the same biochemical processes: genetic information encoded as nucleic acid (DNA, or RNA for many viruses), transcribed into RNA, then translated into proteins (that is, polymers of amino acids) by highly conserved ribosomes. Perhaps most tellingly, the Genetic Code (the "translation table" between DNA and amino acids) is the same for almost every organism, meaning that a piece of DNA in a bacterium codes for the same amino acid as in a human cell. ATP is used as energy currency by all extant life. A deeper understanding of developmental biology shows that common morphology is, in fact, the product of shared genetic elements. For example, although camera-like eyes are believed to have evolved independently on many separate occasions, they share a common set of light-sensing proteins (opsins), suggesting a common point of origin for all sighted creatures. Another example is the familiar vertebrate body plan, whose structure is controlled by the homeobox (Hox) family of genes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">10</span></p>
<h4><span class="mw-headline" id="DNA_sequencing">DNA sequencing</span></h4>
<p>Comparison of DNA sequences allows organisms to be grouped by sequence similarity, and the resulting phylogenetic trees are typically congruent with traditional taxonomy, and are often used to strengthen or correct taxonomic classifications. Sequence comparison is considered a measure robust enough to correct erroneous assumptions in the phylogenetic tree in instances where other evidence is scarce. For example, neutral human DNA sequences are approximately 1.2% divergent (based on substitutions) from those of their nearest genetic relative, the chimpanzee, 1.6% from gorillas, and 6.6% from baboons. Genetic sequence evidence thus allows inference and quantification of genetic relatedness between humans and other apes. The sequence of the 16S ribosomal RNA gene, a vital gene encoding a part of the ribosome, was used to find the broad phylogenetic relationships between all extant life. The analysis by Carl Woese resulted in the three-domain system, arguing for two major splits in the early evolution of life. The first split led to modern Bacteria and the subsequent split led to modern Archaea and Eukaryotes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">11</span></p><p>Some DNA sequences are shared by very different organisms. It has been predicted by the theory of evolution that the differences in such DNA sequences between two organisms should roughly resemble both the biological difference between them according to their anatomy and the time that had passed since these two organisms have separated in the course of evolution, as seen in fossil evidence. The rate of accumulating such changes should be low for some sequences, namely those that code for critical RNA or proteins, and high for others that code for less critical RNA or proteins; but for every specific sequence, the rate of change should be roughly constant over time. These results have been experimentally confirmed. Two examples are DNA sequences coding for rRNA, which is highly conserved, and DNA sequences coding for fibrinopeptides (amino acid chains that are discarded during the formation of fibrin), which are highly non-conserved.
<span style="border: 3px solid blue; color:blue;font-weight: bold">12</span></p>
<h4><span class="mw-headline" id="Proteins">Proteins</span></h4>
<p>Proteomic evidence also supports the universal ancestry of life. Vital proteins, such as the ribosome, DNA polymerase, and RNA polymerase, are found in everything from the most primitive bacteria to the most complex mammals. The core part of the protein is conserved across all lineages of life, serving similar functions. Higher organisms have evolved additional protein subunits, largely affecting the regulation and protein-protein interaction of the core. Other overarching similarities between all lineages of extant organisms, such as DNA, RNA, amino acids, and the lipid bilayer, give support to the theory of common descent. Phylogenetic analyses of protein sequences from various organisms produce similar trees of relationship between all organisms. The chirality of DNA, RNA, and amino acids is conserved across all known life. As there is no functional advantage to right- or left-handed molecular chirality, the simplest hypothesis is that the choice was made randomly by early organisms and passed on to all extant life through common descent. Further evidence for reconstructing ancestral lineages comes from junk DNA such as pseudogenes, "dead" genes that steadily accumulate mutations.
<span style="border: 3px solid blue; color:blue;font-weight: bold">13</span></p>
<h4><span class="mw-headline" id="Pseudogenes">Pseudogenes</span></h4>
<p>Pseudogenes, also known as noncoding DNA, are extra DNA in a genome that do not get transcribed into RNA to synthesize proteins. Some of this noncoding DNA has known functions, but much of it has no known function and is called "Junk DNA". This is an example of a vestige since replicating these genes uses energy, making it a waste in many cases. A pseudogene can be produced when a coding gene accumulates mutations that prevent it from being transcribed, making it non-functional. But since it is not transcribed, it may disappear without affecting fitness, unless it has provided some beneficial function as non-coding DNA. Non-functional pseudogenes may be passed on to later species, thereby labeling the later species as descended from the earlier species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">14</span></p>
<h4><span class="mw-headline" id="Other_mechanisms">Other mechanisms</span></h4>
<p>A large body of molecular evidence supports a variety of mechanisms for large evolutionary changes, including: genome and gene duplication, which facilitates rapid evolution by providing substantial quantities of genetic material under weak or no selective constraints; horizontal gene transfer, the process of transferring genetic material to another cell that is not an organism's offspring, allowing for species to acquire beneficial genes from each other; and recombination, capable of reassorting large numbers of different alleles and of establishing reproductive isolation. The endosymbiotic theory explains the origin of mitochondria and plastids (including chloroplasts), which are organelles of eukaryotic cells, as the incorporation of an ancient prokaryotic cell into ancient eukaryotic cell. Rather than evolving eukaryotic organelles slowly, this theory offers a mechanism for a sudden evolutionary leap by incorporating the genetic material and biochemical composition of a separate species. Evidence supporting this mechanism has been found in the protist <i>Hatena</i>: as a predator it engulfs a green algal cell, which subsequently behaves as an endosymbiont, nourishing <i>Hatena</i>, which in turn loses its feeding apparatus and behaves as an autotroph.
<span style="border: 3px solid blue; color:blue;font-weight: bold">15</span></p><p>Since metabolic processes do not leave fossils, research into the evolution of the basic cellular processes is done largely by comparison of existing organisms. Many lineages diverged when new metabolic processes appeared, and it is theoretically possible to determine when certain metabolic processes appeared by comparing the traits of the descendants of a common ancestor or by detecting their physical manifestations. As an example, the appearance of oxygen in the earth's atmosphere is linked to the evolution of photosynthesis.
<span style="border: 3px solid blue; color:blue;font-weight: bold">16</span></p>
<h3><span class="mw-headline" id="Specific_examples_from_comparative_physiology_and_biochemistry">Specific examples from comparative physiology and biochemistry</span></h3>
<h4><span class="mw-headline" id="Chromosome_2_in_humans">Chromosome 2 in humans</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main article: Chromosome 2 (human)</div>
<div class="hatnote navigation-not-searchable" role="note">Further information: Chimpanzee Genome Project § Genes of the Chromosome 2 fusion site</div>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" class="thumbimage" data-file-height="180" data-file-width="270" height="147" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5d/Chromosome_2_merge_en.svg/220px-Chromosome_2_merge_en.svg.png" srcset width="220"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>2</h1></center></div><b>Figure 1b: </b>Fusion of ancestral chromosomes left distinctive remnants of telomeres, and a vestigial centromere</div></div></div>
<p>Evidence for the evolution of <i>Homo sapiens</i> from a common ancestor with chimpanzees is found in the number of chromosomes in humans as compared to all other members of Hominidae. All hominidae have 24 pairs of chromosomes, except humans, who have only 23 pairs. Human chromosome 2 is a result of an end-to-end fusion of two ancestral chromosomes.
<span style="border: 3px solid blue; color:blue;font-weight: bold">17</span></p><p>The evidence for this includes:
<span style="border: 3px solid blue; color:blue;font-weight: bold">18</span></p>
<ul><li>The correspondence of chromosome 2 to two ape chromosomes. The closest human relative, the common chimpanzee, has near-identical DNA sequences to human chromosome 2, but they are found in two separate chromosomes. The same is true of the more distant gorilla and orangutan.</li>
<li>The presence of a vestigial centromere. Normally a chromosome has just one centromere, but in chromosome 2 there are remnants of a second centromere.</li>
<li>The presence of vestigial telomeres. These are normally found only at the ends of a chromosome, but in chromosome 2 there are additional telomere sequences in the middle.</li></ul>
<p>Chromosome 2 thus presents strong evidence in favour of the common descent of humans and other apes. According to J. W. Ijdo, "We conclude that the locus cloned in cosmids c8.1 and c29B is the relic of an ancient telomere-telomere fusion and marks the point at which two ancestral ape chromosomes fused to give rise to human chromosome 2."
<span style="border: 3px solid blue; color:blue;font-weight: bold">19</span></p>
<h4><span class="mw-headline" id="Cytochrome_c_and_b">Cytochrome c and b</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main article: Cytochrome c</div>
<p>A classic example of biochemical evidence for evolution is the variance of the ubiquitous (i.e. all living organisms have it, because it performs very basic life functions) protein Cytochrome c in living cells. The variance of cytochrome c of different organisms is measured in the number of differing amino acids, each differing amino acid being a result of a base pair substitution, a mutation. If each differing amino acid is assumed the result of one base pair substitution, it can be calculated how long ago the two species diverged by multiplying the number of base pair substitutions by the estimated time it takes for a substituted base pair of the cytochrome c gene to be successfully passed on. For example, if the average time it takes for a base pair of the cytochrome c gene to mutate is N years, the number of amino acids making up the cytochrome c protein in monkeys differ by one from that of humans, this leads to the conclusion that the two species diverged N years ago.
<span style="border: 3px solid blue; color:blue;font-weight: bold">20</span></p><p>The primary structure of cytochrome c consists of a chain of about 100 amino acids. Many higher order organisms possess a chain of 104 amino acids.
<span style="border: 3px solid blue; color:blue;font-weight: bold">21</span></p><p>The cytochrome c molecule has been extensively studied for the glimpse it gives into evolutionary biology. Both chicken and turkeys have identical sequence homology (amino acid for amino acid), as do pigs, cows and sheep. Both humans and chimpanzees share the identical molecule, while rhesus monkeys share all but one of the amino acids: the 66th amino acid is isoleucine in the former and threonine in the latter.
<span style="border: 3px solid blue; color:blue;font-weight: bold">22</span></p><p>What makes these homologous similarities particularly suggestive of common ancestry in the case of cytochrome c, in addition to the fact that the phylogenies derived from them match other phylogenies very well, is the high degree of functional redundancy of the cytochrome c molecule. The different existing configurations of amino acids do not significantly affect the functionality of the protein, which indicates that the base pair substitutions are not part of a directed design, but the result of random mutations that aren't subject to selection.
<span style="border: 3px solid blue; color:blue;font-weight: bold">23</span></p><p>In addition, Cytochrome b is commonly used as a region of mitochondrial DNA to determine phylogenetic relationships between organisms due to its sequence variability. It is considered most useful in determining relationships within families and genera. Comparative studies involving cytochrome b have resulted in new classification schemes and have been used to assign newly described species to a genus, as well as deepen the understanding of evolutionary relationships.
<span style="border: 3px solid blue; color:blue;font-weight: bold">24</span></p>
<h4><span class="mw-headline" id="Endogenous_retroviruses">Endogenous retroviruses</span></h4>
<p>Endogenous retroviruses (or ERVs) are remnant sequences in the genome left from ancient viral infections in an organism. The retroviruses (or virogenes) are always passed on to the next generation of that organism that received the infection. This leaves the virogene left in the genome. Because this event is rare and random, finding identical chromosomal positions of a virogene in two different species suggests common ancestry. Cats (Felidae) present a notable instance of virogene sequences demonstrating common descent. The standard phylogenetic tree for Felidae have smaller cats (<i>Felis chaus</i>, <i>Felis silvestris</i>, <i>Felis nigripes</i>, and <i>Felis catus</i>) diverging from larger cats such as the subfamily Pantherinae and other carnivores. The fact that small cats have an ERV where the larger cats do not suggests that the gene was inserted into the ancestor of the small cats after the larger cats had diverged. Another example of this is with humans and chimps. Humans contain numerous ERVs that comprise a considerable percentage of the genome. Sources vary, but 1% to 8% has been proposed. Humans and chimps share seven different occurrences of virogenes, while all primates share similar retroviruses congruent with phylogeny.
<span style="border: 3px solid blue; color:blue;font-weight: bold">25</span></p>
<h4><span class="mw-headline" id="Recent_African_origin_of_modern_humans">Recent African origin of modern humans</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main article: Recent single-origin hypothesis</div>
<div class="hatnote navigation-not-searchable" role="note">See also: Human mitochondrial DNA haplogroup and Human Y-chromosome DNA haplogroup</div>
<p>Mathematical models of evolution, pioneered by the likes of Sewall Wright, Ronald Fisher and J. B. S. Haldane and extended via diffusion theory by Motoo Kimura, allow predictions about the genetic structure of evolving populations. Direct examination of the genetic structure of modern populations via DNA sequencing has allowed verification of many of these predictions. For example, the Out of Africa theory of human origins, which states that modern humans developed in Africa and a small sub-population migrated out (undergoing a population bottleneck), implies that modern populations should show the signatures of this migration pattern. Specifically, post-bottleneck populations (Europeans and Asians) should show lower overall genetic diversity and a more uniform distribution of allele frequencies compared to the African population. Both of these predictions are borne out by actual data from a number of studies.
<span style="border: 3px solid blue; color:blue;font-weight: bold">26</span></p>
<h2><span class="mw-headline" id="Evidence_from_comparative_anatomy">Evidence from comparative anatomy</span></h2>
<p>Comparative study of the anatomy of groups of animals or plants reveals that certain structural features are basically similar. For example, the basic structure of all flowers consists of sepals, petals, stigma, style and ovary; yet the size, colour, number of parts and specific structure are different for each individual species. The neural anatomy of fossilized remains may also be compared using advanced imaging techniques.
<span style="border: 3px solid blue; color:blue;font-weight: bold">27</span></p>
<h3><span class="mw-headline" id="Atavisms">Atavisms</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Atavism</div>
<div class="thumb tright"><div class="thumbinner" style="width:162px;"><img alt="" class="thumbimage" data-file-height="2598" data-file-width="1605" height="259" src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/bc/HindlegsOfHumpbackWhale.jpg/160px-HindlegsOfHumpbackWhale.jpg" srcset width="160"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>3</h1></center></div><b>Figure 2a:</b> In July 1919, a humpback whale was caught by a ship operating out of Vancouver that had legs 4 ft 2 in (1.27 m) long. This image shows the hindlegs of another humpback whale reported in 1921 by the American Museum of Natural History.</div></div></div>
<p>Once thought of as a refutation to evolutionary theory, atavisms are "now seen as potent evidence of how much genetic potential is retained...after a particular structure has disappeared from a species". "Atavisms are the reappearance of a lost character typical of remote ancestors and not seen in the parents or recent ancestors..." and are an "[indication] of the developmental plasticity that exists within embryos..." Atavisms occur because genes for previously existing phenotypical features are often preserved in DNA, even though the genes are not expressed in some or most of the organisms possessing them. Numerous examples have documented the occurrence of atavisms alongside experimental research triggering their formation. Due to the complexity and interrelatedness of the factors involved in the development of atavisms, both biologists and medical professionals find it "difficult, if not impossible, to distinguish [them] from malformations."
<span style="border: 3px solid blue; color:blue;font-weight: bold">28</span></p><p>Some examples of atavisms found in the scientific literature include:
<span style="border: 3px solid blue; color:blue;font-weight: bold">29</span></p>
<ul><li>Hind limbs in whales. (see figure 2a)</li>
<li>Reappearance of limbs in limbless vertebrates.</li>
<li>Back pair of flippers on a bottlenose dolphin.</li>
<li>Extra toes of the modern horse.</li>
<li>Human tails (not pseudo-tails) and extra nipples in humans.</li>
<li>Re-evolution of sexuality from parthenogenesis in orbitid mites.</li>
<li>Teeth in chickens.</li>
<li>Dewclaws in dogs.</li>
<li>Reappearance of wings on wingless stick insects and earwigs.</li>
<li>Atavistic muscles in several birds and mammals such as the beagle and the jerboa.</li>
<li>Extra toes in guinea pigs.</li></ul>
<h3><span class="mw-headline" id="Evolutionary_developmental_biology_and_embryonic_development">Evolutionary developmental biology and embryonic development</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Evolutionary developmental biology</div>
<div class="hatnote navigation-not-searchable" role="note">See also: Embryogenesis</div>
<p>Evolutionary developmental biology is the biological field that compares the developmental process of different organisms to determine ancestral relationships between species. A large variety of organism's genomes contain a small fraction of genes that control the organisms development. Hox genes are an example of these types of nearly universal genes in organisms pointing to an origin of common ancestry. Embryological evidence comes from the development of organisms at the embryological level with the comparison of different organisms embryos similarity. Remains of ancestral traits often appear and disappear in different stages of the embryological development process.
<span style="border: 3px solid blue; color:blue;font-weight: bold">30</span></p><p>Some examples include:
<span style="border: 3px solid blue; color:blue;font-weight: bold">31</span></p>
<ul><li>Hair growth and loss (lanugo) during human development.</li>
<li>Development and degeneration of a yolk sac.</li>
<li>Terrestrial frogs and salamanders passing through the larval stage within the egg—with features of typically aquatic larvae—but hatch ready for life on land;</li>
<li>The appearance of gill-like structures (pharyngeal arch) in vertebrate embryo development. Note that in fish, the arches continue to develop as branchial arches while in humans, for example, they give rise to a variety of structures within the head and neck.</li></ul>
<h3><span id="Homologous_structures_and_divergent_.28adaptive.29_evolution"></span><span class="mw-headline" id="Homologous_structures_and_divergent_(adaptive)_evolution">Homologous structures and divergent (adaptive) evolution</span></h3>
<p>If widely separated groups of organisms are originated from a common ancestry, they are expected to have certain basic features in common. The degree of resemblance between two organisms should indicate how closely related they are in evolution:
<span style="border: 3px solid blue; color:blue;font-weight: bold">32</span></p>
<ul><li>Groups with little in common are assumed to have diverged from a common ancestor much earlier in geological history than groups with a lot in common;</li>
<li>In deciding how closely related two animals are, a comparative anatomist looks for structures that are fundamentally similar, even though they may serve different functions in the adult. Such structures are described as homologous and suggest a common origin.</li>
<li>In cases where the similar structures serve different functions in adults, it may be necessary to trace their origin and embryonic development. A similar developmental origin suggests they are the same structure, and thus likely derived from a common ancestor.</li></ul>
<p>When a group of organisms share a homologous structure that is specialized to perform a variety of functions to adapt different environmental conditions and modes of life, it is called adaptive radiation. The gradual spreading of organisms with adaptive radiation is known as divergent evolution.
<span style="border: 3px solid blue; color:blue;font-weight: bold">33</span></p>
<h3><span class="mw-headline" id="Nested_hierarchies_and_classification">Nested hierarchies and classification</span></h3>
<p>Taxonomy is based on the fact that all organisms are related to each other in nested hierarchies based on shared characteristics. Most existing species can be organized rather easily in a nested hierarchical classification. This is evident from the Linnaean classification scheme. Based on shared derived characters, closely related organisms can be placed in one group (such as a genus), several genera can be grouped together into one family, several families can be grouped together into an order, etc. The existence of these nested hierarchies was recognized by many biologists before Darwin, but he showed that his theory of evolution with its branching pattern of common descent could explain them. Darwin described how common descent could provide a logical basis for classification:
<span style="border: 3px solid blue; color:blue;font-weight: bold">34</span></p>
<table class="cquote pullquote" role="presentation" style="margin:auto; border-collapse: collapse; border: none; background-color: transparent; width: auto;">
<tbody><tr>
<td style="width: 20px; vertical-align: top; border: none; color: #B2B7F2; font-size: 60px; font-family: 'Times New Roman', Times, serif; font-weight: bold; line-height: .6em; text-align: left; padding: 10px 10px;">“
</td>
<td style="vertical-align: top; border: none; padding: 4px 10px;">All the foregoing rules and aids and difficulties in classification are explained, if I do not greatly deceive myself, on the view that the natural system is founded on descent with modification; that the characters which naturalists consider as showing true affinity between any two or more species, are those which have been inherited from a common parent, and, in so far, all true classification is genealogical; that community of descent is the hidden bond which naturalists have been unconsciously seeking, ...
</td>
<td style="width: 20px; vertical-align: bottom; border: none; color: #B2B7F2; font-size: 60px; font-family: 'Times New Roman', Times, serif; font-weight: bold; line-height: .6em; text-align: right; padding: 10px 10px;">
</td></tr>
<tr>
<td class="cquotecite" colspan="3" style="border: none; padding-right: 4%; font-size: smaller; line-height:normal; text-align: right;"><cite>— Charles Darwin, <i>On the Origin of Species</i>, page 577</cite>
</td></tr></tbody></table><a name="table1"></a><span style='border: 3px solid #00cc00;color:#00cc00;font-size: 24px;'>1</span>
<h4><span class="mw-headline" id="Evolutionary_trees">Evolutionary trees</span></h4>
<p>An evolutionary tree (of Amniota, for example, the last common ancestor of mammals and reptiles, and all its descendants) illustrates the initial conditions causing evolutionary patterns of similarity (e.g., all Amniotes produce an egg that possesses the amnios) and the patterns of divergence amongst lineages (e.g., mammals and reptiles branching from the common ancestry in Amniota). Evolutionary trees provide conceptual models of evolving systems once thought limited in the domain of making predictions out of the theory. However, the method of phylogenetic bracketing is used to infer predictions with far greater probability than raw speculation. For example, paleontologists use this technique to make predictions about nonpreservable traits in fossil organisms, such as feathered dinosaurs, and molecular biologists use the technique to posit predictions about RNA metabolism and protein functions. Thus evolutionary trees are evolutionary hypotheses that refer to specific facts, such as the characteristics of organisms (e.g., scales, feathers, fur), providing evidence for the patterns of descent, and a causal explanation for modification (i.e., natural selection or neutral drift) in any given lineage (e.g., Amniota). Evolutionary biologists test evolutionary theory using phylogenetic systematic methods that measure how much the hypothesis (a particular branching pattern in an evolutionary tree) increases the likelihood of the evidence (the distribution of characters among lineages). The severity of tests for a theory increases if the predictions "are the least probable of being observed if the causal event did not occur." "Testability is a measure of how much the hypothesis increases the likelihood of the evidence."
<span style="border: 3px solid blue; color:blue;font-weight: bold">35</span></p>
<h3><span class="mw-headline" id="Vestigial_structures">Vestigial structures</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Vestigiality</div>
<div class="hatnote navigation-not-searchable" role="note">See also: Human vestigiality</div>
<p>Evidence for common descent comes from the existence of vestigial structures. These rudimentary structures are often homologous to structures that correspond in related or ancestral species. A wide range of structures exist such as mutated and non-functioning genes, parts of a flower, muscles, organs, and even behaviors. This variety can be found across many different groups of species. In many cases they are degenerated or underdeveloped. The existence of vestigial organs can be explained in terms of changes in the environment or modes of life of the species. Those organs are typically functional in the ancestral species but are now either semi-functional, nonfunctional, or re-purposed.
<span style="border: 3px solid blue; color:blue;font-weight: bold">36</span></p><p>Scientific literature concerning vestigial structures abounds. One study complied 64 examples of vestigial structures found in the literature across a wide range of disciplines within the 21st century. The following non-exhaustive list summarizes Senter et al. alongside various other examples:
<span style="border: 3px solid blue; color:blue;font-weight: bold">37</span></p>
<ul><li>The presence of remnant mitochondria (mitosomes) that have lost the ability to synthesize ATP in <i>Entamoeba histolytica, Trachipleistophora hominis, Cryptosporidium parvum, Blastocystis hominis</i>, and <i>Giardia intestinalis</i>.</li>
<li>Remnant chloroplast organelles (leucoplasts) in non-photosynthetic algae species (<i>Plasmodium falciparum</i>, <i>Toxoplasma gondii</i>, <i>Aspasia longa</i>, <i>Anthophysa vegetans</i>, <i>Ciliophrys infusionum</i>, <i>Pteridomonas danica</i>, <i>Paraphysomonas</i>, <i>Spumella</i> and <i>Epifagus americana</i>.</li>
<li>Missing stamens (unvascularized staminodes) on <i>Gilliesia</i> and <i>Gethyum</i> flowers.</li>
<li>Non-functioning androecium in female flowers and non-functioning gynoecium in male flowers of the cactus species <i>Consolea spinosissima</i>.</li>
<li>Remnant stamens on female flowers of <i>Fragaria virginiana</i>; all species in the genus <i>Schiedea</i>; and on <i>Penstemon centranthifolius</i>, <i>P. rostriflorus</i>, <i>P. ellipticus</i>, and <i>P. palmeri</i>.</li>
<li>Vestigial anthers on <i>Nemophila menziesii</i>.</li></ul>
<div class="thumb tright"><div class="thumbinner" style="width:352px;"><img alt="" class="thumbimage" data-file-height="143" data-file-width="545" height="92" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/0c/Mystice_pelvis_%28whale%29.png/350px-Mystice_pelvis_%28whale%29.png" srcset width="350"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>4</h1></center></div><b>Figure 2b:</b> Skeleton of a baleen whale with the hind limb and pelvic bone structure circled in red. This bone structure stays internal during the entire life of the species.</div></div></div>
<ul><li>Reduced hindlimbs and pelvic girdle embedded in the muscles of extant whales (see figure 2b). Occasionally, the genes that code for longer extremities cause a modern whale to develop legs. On 28 October 2006, a four-finned bottlenose dolphin was caught and studied due to its extra set of hind limbs. These legged Cetacea display an example of an atavism predicted from their common ancestry.</li>
<li>Nonfunctional hind wings in <i>Carabus solieri</i> and other beetles.</li>
<li>Remnant eyes (and eye structures) in animals that have lost sight such as blind cavefish (e.g. <i>Astyanax mexicanus</i>), mole rats, snakes, spiders, salamanders, shrimp, crayfish, and beetles.</li>
<li>Vestigial eye in the extant <i>Rhineura floridana</i> and remnant jugal in the extinct <i>Rhineura hatchery</i> (reclassified as <i>Protorhineura hatcherii</i>).</li>
<li>Functionless wings in flightless birds such as ostriches, kiwis, cassowaries, and emus.</li>
<li>The presence of the plica semilunaris in the human eye—a vestigial remnant of the nictitating membrane.</li>
<li>Harderian gland in primates.</li>
<li>Reduced hind limbs and pelvic girdle structures in legless lizards, skinks, amphisbaenians, and some snakes.</li>
<li>Reduced and missing olfactory apparatus in whales that still possess vestigial olfactory receptor subgenomes.</li>
<li>Vestigial teeth in narwhal.</li>
<li>Rudimentary digits of <i>Ateles geoffroyi</i>, <i>Colobus guereza</i>, and <i>Perodicticus potto</i>.</li>
<li>Vestigial dental primordia in the embryonic tooth pattern in mice.</li>
<li>Reduced or absent vomeronasal organ in humans and Old World monkeys.</li>
<li>Presence of non-functional sinus hair muscles in humans used in whisker movement.</li>
<li>Degenerating palmaris longus muscle in humans.</li>
<li>Teleost fish, anthropoid primates (Simians), guinea pigs, some bat species, and some Passeriformes have lost the ability to synthesize vitamin C (ascorbic acid), yet still possess the genes involved. This inability is due to mutations of the L-gulono-γ-lactone oxidase (<i>GLO</i>) gene— and in primates, teleost fish, and guinea pigs it is irreversible.</li>
<li>Remnant abdominal segments in cirripedes (barnacles).</li>
<li>Non-mammalian vertebrate embryos depend on nutrients from the yolk sac. Humans and other mammal genomes contain broken, non-functioning genes that code for the production of yolk. alongside the presence of an empty yolk sac with the embryo.</li>
<li>Dolphin embryonic limb buds.</li>
<li>Leaf formation in some cacti species.</li>
<li>Presence of a vestigial endosymbiont <i>Lepidodinium viride</i> within the dinoflagellate <i>Gymnodinium chlorophorum</i>.</li>
<li>The species <i>Dolabrifera dolabrifera</i> has an ink gland but is "incapable of producing ink or its associated anti-predator proteins".</li></ul>
<h3><span class="mw-headline" id="Specific_examples_from_comparative_anatomy">Specific examples from comparative anatomy</span></h3>
<style data-mw-deduplicate="TemplateStyles:r856303569">@media all and (max-width:720px){.mw-parser-output .tmulti>.thumbinner{width:100%!important;max-width:none!important}.mw-parser-output .tmulti .tsingle{float:none!important;max-width:none!important;width:100%!important;text-align:center}}</style><div class="thumb tmulti tright"><div class="thumbinner" style="width:414px;max-width:414px"><div class="tsingle" style="margin:1px;width:412px;max-width:412px"><div class="thumbimage"><img alt="Figure 2c" data-file-height="600" data-file-width="630" height="390" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/Evolution_insect_mouthparts_coloured.svg/410px-Evolution_insect_mouthparts_coloured.svg.png" srcset width="410"/></div><div class="thumbcaption" style="clear:left"><b>Figure 2c</b>: Adaptation of insect mouthparts: a, antennae; c, compound eye; lb, labrium; lr, labrum; md, mandibles; mx, maxillae.<br/>
(A) Primitive state — biting and chewing: <i>e.g.</i> grasshopper. Strong mandibles and maxillae for manipulating food.<br/> (B) Ticking and biting: <i>e.g.</i> honey bee. Labium long to lap up nectar; mandibles chew pollen and mould wax.<br/> (C) Sucking: <i>e.g.</i> butterfly. Labrum reduced; mandibles lost; maxillae long forming sucking tube.<br/> (D) Piercing and sucking, <i>e.g.</i>. female mosquito. Labrum and maxillae form tube; mandibles form piercing stylets; labrum grooved to hold other parts.</div></div><div class="tsingle" style="margin:1px;width:412px;max-width:412px"><div class="thumbimage"><img alt="Figure 2d" data-file-height="1900" data-file-width="3600" height="216" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e0/Saurischian_%26_Ornithischian_pelvis_bone_modification_w-_selected_species_of_Dinosauria.svg/410px-Saurischian_%26_Ornithischian_pelvis_bone_modification_w-_selected_species_of_Dinosauria.svg.png" srcset width="410"/></div><div class="thumbcaption" style="clear:left"><b>Figure 2d:</b> Illustration of the <i>Eoraptor lunensis</i> pelvis of the order Saurischia and the <i>Lesothosaurus diagnosticus</i> pelvis of the order Ornithischia in the clade Dinosauria. The parts of the pelvis show modification over time. The cladogram is shown to illustrate the distance of divergence between the two species.</div></div><div class="tsingle" style="margin:1px;width:412px;max-width:412px"><div class="thumbimage"><img alt="Figure 2e" data-file-height="1578" data-file-width="2222" height="291" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5b/Evolution_pl.png/410px-Evolution_pl.png" srcset width="410"/></div><div class="thumbcaption" style="clear:left"><b>Figure 2e</b>: The principle of homology illustrated by the adaptive radiation of the forelimb of mammals. All conform to the basic pentadactyl pattern but are modified for different usages. The third metacarpal is shaded throughout; the shoulder is crossed-hatched.</div></div><div class="tsingle" style="margin:1px;width:412px;max-width:412px"><div class="thumbimage"><img alt="Figure 2f" data-file-height="305" data-file-width="450" height="278" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/7e/GiraffaRecurrEn.svg/410px-GiraffaRecurrEn.svg.png" srcset width="410"/></div><div class="thumbcaption" style="clear:left"><b>Figure 2f:</b> The path of the recurrent laryngeal nerve in giraffes. The laryngeal nerve is compensated for by subsequent tinkering from natural selection.</div></div><div class="tsingle" style="margin:1px;width:412px;max-width:412px"><div class="thumbimage"><img alt="Figure 2g" data-file-height="426" data-file-width="294" height="594" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/01/Route_of_vas_deferens_en.svg/410px-Route_of_vas_deferens_en.svg.png" srcset width="410"/></div><div class="thumbcaption" style="clear:left"><b>Figure 2g:</b> Route of the vas deferens from the testis to the penis.</div></div></div></div>
<h4><span class="mw-headline" id="Insect_mouthparts_and_appendages">Insect mouthparts and appendages</span></h4>
<p>Many different species of insects have mouthparts derived from the same embryonic structures, indicating that the mouthparts are modifications of a common ancestor's original features. These include a labrum (upper lip), a pair of mandibles, a hypopharynx (floor of mouth), a pair of maxillae, and a labium. (Fig. 2c) Evolution has caused enlargement and modification of these structures in some species, while it has caused the reduction and loss of them in other species. The modifications enable the insects to exploit a variety of food materials.
<span style="border: 3px solid blue; color:blue;font-weight: bold">38</span></p><p>Insect mouthparts and antennae are considered homologues of insect legs. Parallel developments are seen in some arachnids: The anterior pair of legs may be modified as analogues of antennae, particularly in whip scorpions, which walk on six legs. These developments provide support for the theory that complex modifications often arise by duplication of components, with the duplicates modified in different directions.
<span style="border: 3px solid blue; color:blue;font-weight: bold">39</span></p>
<h4><span class="mw-headline" id="Pelvic_structure_of_dinosaurs">Pelvic structure of dinosaurs</span></h4>
<div class="hatnote navigation-not-searchable" role="note">See also: Evolution of dinosaurs and Evolution of birds</div>
<p>Similar to the pentadactyl limb in mammals, the earliest dinosaurs split into two distinct orders—the <i>saurischia</i> and <i>ornithischia</i>. They are classified as one or the other in accordance with what the fossils demonstrate. Figure 2d, shows that early <i>saurischians</i> resembled early <i>ornithischians</i>. The pattern of the pelvis in all species of dinosaurs is an example of homologous structures. Each order of dinosaur has slightly differing pelvis bones providing evidence of common descent. Additionally, modern birds show a similarity to ancient <i>saurischian</i> pelvic structures indicating the evolution of birds from dinosaurs. This can also be seen in Figure 5c as the Aves branch off the Theropoda suborder.
<span style="border: 3px solid blue; color:blue;font-weight: bold">40</span></p>
<h4><span class="mw-headline" id="Pentadactyl_limb">Pentadactyl limb</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Further information: Evolution of mammals</div>
<p>The pattern of limb bones called pentadactyl limb is an example of homologous structures (Fig. 2e). It is found in all classes of tetrapods (<i>i.e.</i> from amphibians to mammals). It can even be traced back to the fins of certain fossil fishes from which the first amphibians evolved such as tiktaalik. The limb has a single proximal bone (humerus), two distal bones (radius and ulna), a series of carpals (wrist bones), followed by five series of metacarpals (palm bones) and phalanges (digits). Throughout the tetrapods, the fundamental structures of pentadactyl limbs are the same, indicating that they originated from a common ancestor. But in the course of evolution, these fundamental structures have been modified. They have become superficially different and unrelated structures to serve different functions in adaptation to different environments and modes of life. This phenomenon is shown in the forelimbs of mammals. For example:
<span style="border: 3px solid blue; color:blue;font-weight: bold">41</span></p>
<ul><li>In monkeys, the forelimbs are much elongated, forming a grasping hand used for climbing and swinging among trees.</li>
<li>Pigs have lost their first digit, while the second and fifth digits are reduced. The remaining two digits are longer and stouter than the rest and bear a hoof for supporting the body.</li>
<li>In horses, the forelimbs are highly adapted for strength and support. Fast and long-distance running is possible due to the extensive elongation of the third digit that bears a hoof.</li>
<li>The mole has a pair of short, spade-like forelimbs for burrowing.</li>
<li>Anteaters use their enlarged third digit for tearing into ant and termite nests.</li>
<li>In cetaceans, the forelimbs become flippers for steering and maintaining equilibrium during swimming.</li>
<li>In bats, the forelimbs have become highly modified and evolved into functioning wings. Four digits have become elongated, while the hook-like first digit remains free and is used to hang upside down.</li></ul>
<h4><span class="mw-headline" id="Recurrent_laryngeal_nerve_in_giraffes">Recurrent laryngeal nerve in giraffes</span></h4>
<p>The recurrent laryngeal nerve is a fourth branch of the vagus nerve, which is a cranial nerve. In mammals, its path is unusually long. As a part of the vagus nerve, it comes from the brain, passes through the neck down to heart, rounds the dorsal aorta and returns up to the larynx, again through the neck. (Fig. 2f)
<span style="border: 3px solid blue; color:blue;font-weight: bold">42</span></p><p>This path is suboptimal even for humans, but for giraffes it becomes even more suboptimal. Due to the lengths of their necks, the recurrent laryngeal nerve may be up to 4 m (13 ft) long, despite its optimal route being a distance of just several inches.
<span style="border: 3px solid blue; color:blue;font-weight: bold">43</span></p><p>The indirect route of this nerve is the result of evolution of mammals from fish, which had no neck and had a relatively short nerve that innervated one gill slit and passed near the gill arch. Since then, the gill it innervated has become the larynx and the gill arch has become the dorsal aorta in mammals.
<span style="border: 3px solid blue; color:blue;font-weight: bold">44</span></p>
<h4><span class="mw-headline" id="Route_of_the_vas_deferens">Route of the vas deferens</span></h4>
<p>Similar to the laryngeal nerve in giraffes, the vas deferens is part of the male anatomy of many vertebrates; it transports sperm from the epididymis in anticipation of ejaculation. In humans, the vas deferens routes up from the testicle, looping over the ureter, and back down to the urethra and penis. It has been suggested that this is due to the descent of the testicles during the course of human evolution—likely associated with temperature. As the testicles descended, the vas deferens lengthened to accommodate the accidental "hook" over the ureter.
<span style="border: 3px solid blue; color:blue;font-weight: bold">45</span></p>
<h2><span class="mw-headline" id="Evidence_from_paleontology">Evidence from paleontology</span></h2>
<link href="mw-data:TemplateStyles:r856303569" rel="mw-deduplicated-inline-style"/><div class="thumb tmulti tright"><div class="thumbinner" style="width:304px;max-width:304px"><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="Figure 3a" data-file-height="1360" data-file-width="980" height="416" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Resin_with_insect_%28aka%29.jpg/300px-Resin_with_insect_%28aka%29.jpg" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 3a:</b> An insect trapped in amber</div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="Figure 3b" data-file-height="1316" data-file-width="1511" height="261" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Kainops_invius_lateral_and_ventral.JPG/300px-Kainops_invius_lateral_and_ventral.JPG" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 3b</b>: Fossil trilobite, <i>Kainops invius</i>, from the early Devonian. Trilobites were hard-shelled arthropods, related to living horseshoe crabs and spiders, that first appeared in significant numbers around 540 mya, dying out 250 mya.</div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="Figure 3c" data-file-height="1210" data-file-width="2239" height="162" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6d/Cynognathus.JPG/300px-Cynognathus.JPG" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 3c:</b> Skull of <i>Cynognathus</i>, a eucynodont, one of a grouping of therapsids ("mammal-like reptiles") that is ancestral to all modern mammals.</div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="Figure 3d" data-file-height="774" data-file-width="1600" height="145" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Glyptodon-1.jpg/300px-Glyptodon-1.jpg" srcset width="300"/></div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="Figure 3e" data-file-height="686" data-file-width="1050" height="196" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/Dasypus_novemcinctus.jpg/300px-Dasypus_novemcinctus.jpg" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 3d:</b> Charles Darwin collected fossils in South America, and found fragments of armor he thought were like giant versions of the scutes on the modern armadillos living nearby. The anatomist Richard Owen showed him that the fragments were from the gigantic extinct <i>Glyptodon</i>, related to the armadillos. This was one of the patterns of distribution that helped Darwin to develop his theory.</div></div></div></div>
<p>When organisms die, they often decompose rapidly or are consumed by scavengers, leaving no permanent evidences of their existence. However, occasionally, some organisms are preserved. The remains or traces of organisms from a past geologic age embedded in rocks by natural processes are called fossils. They are extremely important for understanding the evolutionary history of life on Earth, as they provide direct evidence of evolution and detailed information on the ancestry of organisms. Paleontology is the study of past life based on fossil records and their relations to different geologic time periods.
<span style="border: 3px solid blue; color:blue;font-weight: bold">46</span></p><p>For fossilization to take place, the traces and remains of organisms must be quickly buried so that weathering and decomposition do not occur. Skeletal structures or other hard parts of the organisms are the most commonly occurring form of fossilized remains. There are also some trace "fossils" showing moulds, cast or imprints of some previous organisms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">47</span></p><p>As an animal dies, the organic materials gradually decay, such that the bones become porous. If the animal is subsequently buried in mud, mineral salts infiltrate into the bones and gradually fill up the pores. The bones harden into stones and are preserved as fossils. This process is known as petrification. If dead animals are covered by wind-blown sand, and if the sand is subsequently turned into mud by heavy rain or floods, the same process of mineral infiltration may occur. Apart from petrification, the dead bodies of organisms may be well preserved in ice, in hardened resin of coniferous trees (figure 3a), in tar, or in anaerobic, acidic peat. Fossilization can sometimes be a trace, an impression of a form. Examples include leaves and footprints, the fossils of which are made in layers that then harden.
<span style="border: 3px solid blue; color:blue;font-weight: bold">48</span></p>
<h3><span class="mw-headline" id="Fossil_record">Fossil record</span></h3>
<p>It is possible to decipher how a particular group of organisms evolved by arranging its fossil record in a chronological sequence. Such a sequence can be determined because fossils are mainly found in sedimentary rock. Sedimentary rock is formed by layers of silt or mud on top of each other; thus, the resulting rock contains a series of horizontal layers, or strata. Each layer contains fossils typical for a specific time period when they formed. The lowest strata contain the oldest rock and the earliest fossils, while the highest strata contain the youngest rock and more recent fossils.
<span style="border: 3px solid blue; color:blue;font-weight: bold">49</span></p><p>A succession of animals and plants can also be seen from fossil discoveries. By studying the number and complexity of different fossils at different stratigraphic levels, it has been shown that older fossil-bearing rocks contain fewer types of fossilized organisms, and they all have a simpler structure, whereas younger rocks contain a greater variety of fossils, often with increasingly complex structures.
<span style="border: 3px solid blue; color:blue;font-weight: bold">50</span></p><p>For many years, geologists could only roughly estimate the ages of various strata and the fossils found. They did so, for instance, by estimating the time for the formation of sedimentary rock layer by layer. Today, by measuring the proportions of radioactive and stable elements in a given rock, the ages of fossils can be more precisely dated by scientists. This technique is known as radiometric dating.
<span style="border: 3px solid blue; color:blue;font-weight: bold">51</span></p><p>Throughout the fossil record, many species that appear at an early stratigraphic level disappear at a later level. This is interpreted in evolutionary terms as indicating the times when species originated and became extinct. Geographical regions and climatic conditions have varied throughout Earth's history. Since organisms are adapted to particular environments, the constantly changing conditions favoured species that adapted to new environments through the mechanism of natural selection.
<span style="border: 3px solid blue; color:blue;font-weight: bold">52</span></p>
<h4><span class="mw-headline" id="Extent_of_the_fossil_record">Extent of the fossil record</span></h4>
<div class="hatnote navigation-not-searchable" role="note">See also: Transitional fossil and List of transitional fossils</div>
<p>Despite the relative rarity of suitable conditions for fossilization, an estimated 250,000 fossil species have been named. The number of individual fossils this represents varies greatly from species to species, but many millions of fossils have been recovered: for instance, more than three million fossils from the last ice age have been recovered from the La Brea Tar Pits in Los Angeles. Many more fossils are still in the ground, in various geological formations known to contain a high fossil density, allowing estimates of the total fossil content of the formation to be made. An example of this occurs in South Africa's Beaufort Formation (part of the Karoo Supergroup, which covers most of South Africa), which is rich in vertebrate fossils, including therapsids (reptile-mammal transitional forms). It has been estimated that this formation contains 800 billion vertebrate fossils. Palentologists have documented numerous transitional forms and have constructed "an astonishingly comprehensive record of the key transitions in animal evolution". Conducting a survey of the paleontological literature, one would find that there is "abundant evidence for how all the major groups of animals are related, much of it in the form of excellent transitional fossils".
<span style="border: 3px solid blue; color:blue;font-weight: bold">53</span></p>
<h3><span class="mw-headline" id="Limitations">Limitations</span></h3>
<p>The fossil record is an important source for scientists when tracing the evolutionary history of organisms. However, because of limitations inherent in the record, there are not fine scales of intermediate forms between related groups of species. This lack of continuous fossils in the record is a major limitation in tracing the descent of biological groups. When transitional fossils are found that show intermediate forms in what had previously been a gap in knowledge, they are often popularly referred to as "missing links".
<span style="border: 3px solid blue; color:blue;font-weight: bold">54</span></p><p>There is a gap of about 100 million years between the beginning of the Cambrian period and the end of the Ordovician period. The early Cambrian period was the period from which numerous fossils of sponges, cnidarians (<i>e.g.</i>, jellyfish), echinoderms (<i>e.g.</i>, eocrinoids), molluscs (<i>e.g.</i>, snails) and arthropods (<i>e.g.</i>, trilobites) are found. The first animal that possessed the typical features of vertebrates, the <i>Arandaspis</i>, was dated to have existed in the later Ordovician period. Thus few, if any, fossils of an intermediate type between invertebrates and vertebrates have been found, although likely candidates include the Burgess Shale animal, <i>Pikaia gracilens</i>, and its Maotianshan shales relatives, <i>Myllokunmingia</i>, <i>Yunnanozoon</i>, <i>Haikouella lanceolata</i>, and <i>Haikouichthys</i>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">55</span></p><p>Some of the reasons for the incompleteness of fossil records are:
<span style="border: 3px solid blue; color:blue;font-weight: bold">56</span></p>
<ul><li>In general, the probability that an organism becomes fossilized is very low;</li>
<li>Some species or groups are less likely to become fossils because they are soft-bodied;</li>
<li>Some species or groups are less likely to become fossils because they live (and die) in conditions that are not favourable for fossilization;</li>
<li>Many fossils have been destroyed through erosion and tectonic movements;</li>
<li>Most fossils are fragmentary;</li>
<li>Some evolutionary change occurs in populations at the limits of a species' ecological range, and as these populations are likely small, the probability of fossilization is lower (see punctuated equilibrium);</li>
<li>Similarly, when environmental conditions change, the population of a species is likely to be greatly reduced, such that any evolutionary change induced by these new conditions is less likely to be fossilized;</li>
<li>Most fossils convey information about external form, but little about how the organism functioned;</li>
<li>Using present-day biodiversity as a guide, this suggests that the fossils unearthed represent only a small fraction of the large number of species of organisms that lived in the past.</li></ul>
<h3><span class="mw-headline" id="Specific_examples_from_paleontology">Specific examples from paleontology</span></h3>
<div class="hatnote navigation-not-searchable" role="note">See also: Evidence of common descent § Fossils</div>
<h4><span class="mw-headline" id="Evolution_of_the_horse">Evolution of the horse</span></h4>
<div class="thumb tleft"><div class="thumbinner" style="width:327px;"><img alt="" class="thumbimage" data-file-height="1392" data-file-width="1146" height="395" src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Horseevolution.png/325px-Horseevolution.png" srcset width="325"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>5</h1></center></div><b>Figure 3e:</b> Evolution of the horse showing reconstruction of the fossil species obtained from successive rock strata. The foot diagrams are all front views of the left forefoot. The third metacarpal is shaded throughout. The teeth are shown in longitudinal section.</div></div></div>
<div class="hatnote navigation-not-searchable" role="note">Main article: Evolution of the horse</div>
<p>Due to an almost-complete fossil record found in North American sedimentary deposits from the early Eocene to the present, the horse provides one of the best examples of evolutionary history (phylogeny).
<span style="border: 3px solid blue; color:blue;font-weight: bold">57</span></p><p>This evolutionary sequence starts with a small animal called <i>Hyracotherium</i> (commonly referred to as <i>Eohippus</i>), which lived in North America about 54 million years ago then spread across to Europe and Asia. Fossil remains of <i>Hyracotherium</i> show it to have differed from the modern horse in three important respects: it was a small animal (the size of a fox), lightly built and adapted for running; the limbs were short and slender, and the feet elongated so that the digits were almost vertical, with four digits in the forelimbs and three digits in the hindlimbs; and the incisors were small, the molars having low crowns with rounded cusps covered in enamel.
<span style="border: 3px solid blue; color:blue;font-weight: bold">58</span></p><p>The probable course of development of horses from <i>Hyracotherium</i> to <i>Equus</i> (the modern horse) involved at least 12 genera and several hundred species. The major trends seen in the development of the horse to changing environmental conditions may be summarized as follows:
<span style="border: 3px solid blue; color:blue;font-weight: bold">59</span></p>
<ul><li>Increase in size (from 0.4 m to 1.5 m — from 15 in to 60 in);</li>
<li>Lengthening of limbs and feet;</li>
<li>Reduction of lateral digits;</li>
<li>Increase in length and thickness of the third digit;</li>
<li>Increase in width of incisors;</li>
<li>Replacement of premolars by molars; and</li>
<li>Increases in tooth length, crown height of molars.</li></ul>
<p>Fossilized plants found in different strata show that the marshy, wooded country in which <i>Hyracotherium</i> lived became gradually drier. Survival now depended on the head being in an elevated position for gaining a good view of the surrounding countryside, and on a high turn of speed for escape from predators, hence the increase in size and the replacement of the splayed-out foot by the hoofed foot. The drier, harder ground would make the original splayed-out foot unnecessary for support. The changes in the teeth can be explained by assuming that the diet changed from soft vegetation to grass. A dominant genus from each geological period has been selected (see figure 3e) to show the slow alteration of the horse lineage from its ancestral to its modern form.
<span style="border: 3px solid blue; color:blue;font-weight: bold">60</span></p>
<h4><span class="mw-headline" id="Transition_from_fish_to_amphibians">Transition from fish to amphibians</span></h4>
<p>Prior to 2004, paleontologists had found fossils of amphibians with necks, ears, and four legs, in rock no older than 365 million years old. In rocks more than 385 million years old they could only find fish, without these amphibian characteristics. Evolutionary theory predicted that since amphibians evolved from fish, an intermediate form should be found in rock dated between 365 and 385 million years ago. Such an intermediate form should have many fish-like characteristics, conserved from 385 million years ago or more, but also have many amphibian characteristics as well. In 2004, an expedition to islands in the Canadian arctic searching specifically for this fossil form in rocks that were 375 million years old discovered fossils of Tiktaalik. Some years later, however, scientists in Poland found evidence of fossilised tetrapod tracks predating <i>Tiktaalik</i>.

<span style="border: 3px solid blue; color:blue;font-weight: bold">61</span></p>
<h2><span class="mw-headline" id="Evidence_from_biogeography">Evidence from biogeography</span></h2>
<p>Data about the presence or absence of species on various continents and islands (biogeography) can provide evidence of common descent and shed light on patterns of speciation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">62</span></p>
<h3><span class="mw-headline" id="Continental_distribution">Continental distribution</span></h3>
<p>All organisms are adapted to their environment to a greater or lesser extent. If the abiotic and biotic factors within a habitat are capable of supporting a particular species in one geographic area, then one might assume that the same species would be found in a similar habitat in a similar geographic area, e.g. in Africa and South America. This is not the case. Plant and animal species are discontinuously distributed throughout the world:
<span style="border: 3px solid blue; color:blue;font-weight: bold">63</span></p>
<ul><li>Africa has Old World monkeys, apes, elephants, leopards, giraffes, and hornbills.</li>
<li>South America has New World monkeys, cougars, jaguars, sloths, llamas, and toucans.</li>
<li>Deserts in North and South America have native cacti, but deserts in Africa, Asia, and Australia have succulent (apart from <i>Rhipsalis baccifera</i>) which are native euphorbs that resemble cacti but are very different.</li></ul>
<p>Even greater differences can be found if Australia is taken into consideration, though it occupies the same latitude as much of South America and Africa. Marsupials like kangaroos, bandicoots, and quolls make up about half of Australia's indigenous mammal species. By contrast, marsupials are today totally absent from Africa and form a smaller portion of the mammalian fauna of South America, where opossums, shrew opossums, and the monito del monte occur. The only living representatives of primitive egg-laying mammals (monotremes) are the echidnas and the platypus. The short-beaked echidna (<i>Tachyglossus aculeatus</i>) and its subspecies populate Australia, Tasmania, New Guinea, and Kangaroo Island while the long-beaked echidna (<i>Zaglossus bruijni</i>) lives only in New Guinea. The platypus lives in the waters of eastern Australia. They have been introduced to Tasmania, King Island, and Kangaroo Island. These Monotremes are totally absent in the rest of the world. On the other hand, Australia is missing many groups of placental mammals that are common on other continents (carnivorans, artiodactyls, shrews, squirrels, lagomorphs), although it does have indigenous bats and murine rodents; many other placentals, such as rabbits and foxes, have been introduced there by humans.
<span style="border: 3px solid blue; color:blue;font-weight: bold">64</span></p><p>Other animal distribution examples include bears, located on all continents excluding Africa, Australia and Antarctica, and the polar bear solely in the Arctic Circle and adjacent land masses. Penguins are found only around the South Pole despite similar weather conditions at the North Pole. Families of sirenians are distributed around the earth's waters, where manatees are located in western Africa waters, northern South American waters, and West Indian waters only while the related family, the dugongs, are located only in Oceanic waters north of Australia, and the coasts surrounding the Indian Ocean. The now extinct Steller's sea cow resided in the Bering Sea.
<span style="border: 3px solid blue; color:blue;font-weight: bold">65</span></p><p>The same kinds of fossils are found from areas known to be adjacent to one another in the past but that, through the process of continental drift, are now in widely divergent geographic locations. For example, fossils of the same types of ancient amphibians, arthropods and ferns are found in South America, Africa, India, Australia and Antarctica, which can be dated to the Paleozoic Era, when these regions were united as a single landmass called Gondwana. Sometimes the descendants of these organisms can be identified and show unmistakable similarity to each other, even though they now inhabit very different regions and climates.
<span style="border: 3px solid blue; color:blue;font-weight: bold">66</span></p>
<h3><span class="mw-headline" id="Island_biogeography">Island biogeography</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:222px;"><img alt="" class="thumbimage" data-file-height="236" data-file-width="250" height="208" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Darwin%27s_finches.jpeg/220px-Darwin%27s_finches.jpeg" srcset width="220"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>6</h1></center></div><b>Figure 4a:</b> Four of the 13 finch species found on the Galápagos Archipelago, have evolved by an adaptive radiation that diversified their beak shapes to adapt them to different food sources.</div></div></div>
<h4><span class="mw-headline" id="Types_of_species_found_on_islands">Types of species found on islands</span></h4>
<p>Evidence from island biogeography has played an important and historic role in the development of evolutionary biology. For purposes of biogeography, islands are divided into two classes. Continental islands are islands like Great Britain, and Japan that have at one time or another been part of a continent. Oceanic islands, like the Hawaiian islands, the Galápagos Islands and St. Helena, on the other hand are islands that have formed in the ocean and never been part of any continent. Oceanic islands have distributions of native plants and animals that are unbalanced in ways that make them distinct from the biotas found on continents or continental islands. Oceanic islands do not have native terrestrial mammals (they do sometimes have bats and seals), amphibians, or fresh water fish. In some cases they have terrestrial reptiles (such as the iguanas and giant tortoises of the Galápagos Islands) but often (such as in Hawaii) they do not. This is despite the fact that when species such as rats, goats, pigs, cats, mice, and cane toads, are introduced to such islands by humans they often thrive. Starting with Charles Darwin, many scientists have conducted experiments and made observations that have shown that the types of animals and plants found, and not found, on such islands are consistent with the theory that these islands were colonized accidentally by plants and animals that were able to reach them. Such accidental colonization could occur by air, such as plant seeds carried by migratory birds, or bats and insects being blown out over the sea by the wind, or by floating from a continent or other island by sea (for example, by some kinds of plant seeds like coconuts that can survive immersion in salt water), and reptiles that can survive for extended periods on rafts of vegetation carried to sea by storms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">67</span></p>
<h4><span class="mw-headline" id="Endemism">Endemism</span></h4>
<p>Many of the species found on remote islands are endemic to a particular island or group of islands, meaning they are found nowhere else on earth. Examples of species endemic to islands include many flightless birds of New Zealand, lemurs of Madagascar, the Komodo dragon of Komodo, the dragon's blood tree of Socotra, Tuatara of New Zealand, and others. However, many such endemic species are related to species found on other nearby islands or continents; the relationship of the animals found on the Galápagos Islands to those found in South America is a well-known example. All of these facts, the types of plants and animals found on oceanic islands, the large number of endemic species found on oceanic islands, and the relationship of such species to those living on the nearest continents, are most easily explained if the islands were colonized by species from nearby continents that evolved into the endemic species now found there.
<span style="border: 3px solid blue; color:blue;font-weight: bold">68</span></p><p>Other types of endemism do not have to include, in the strict sense, islands. Islands can mean isolated lakes or remote and isolated areas. Examples of these would include the highlands of Ethiopia, Lake Baikal, fynbos of South Africa, forests of New Caledonia, and others. Examples of endemic organisms living in isolated areas include the kagu of New Caledonia, cloud rats of the Luzon tropical pine forests of the Philippines, the boojum tree (<i>Fouquieria columnaris</i>) of the Baja California peninsula, the Baikal seal and the omul of Lake Baikal.
<span style="border: 3px solid blue; color:blue;font-weight: bold">69</span></p>
<h4><span class="mw-headline" id="Adaptive_radiations">Adaptive radiations</span></h4>
<p>Oceanic islands are frequently inhabited by clusters of closely related species that fill a variety of ecological niches, often niches that are filled by very different species on continents. Such clusters, like the finches of the Galápagos, Hawaiian honeycreepers, members of the sunflower family on the Juan Fernandez Archipelago and wood weevils on St. Helena are called adaptive radiations because they are best explained by a single species colonizing an island (or group of islands) and then diversifying to fill available ecological niches. Such radiations can be spectacular; 800 species of the fruit fly family <i>Drosophila</i>, nearly half the world's total, are endemic to the Hawaiian islands. Another illustrative example from Hawaii is the silversword alliance, which is a group of thirty species found only on those islands. Members range from the silverswords that flower spectacularly on high volcanic slopes to trees, shrubs, vines and mats that occur at various elevations from mountain top to sea level, and in Hawaiian habitats that vary from deserts to rainforests. Their closest relatives outside Hawaii, based on molecular studies, are tarweeds found on the west coast of North America. These tarweeds have sticky seeds that facilitate distribution by migrant birds. Additionally, nearly all of the species on the island can be crossed and the hybrids are often fertile, and they have been hybridized experimentally with two of the west coast tarweed species as well. Continental islands have less distinct biota, but those that have been long separated from any continent also have endemic species and adaptive radiations, such as the 75 lemur species of Madagascar, and the eleven extinct moa species of New Zealand.
<span style="border: 3px solid blue; color:blue;font-weight: bold">70</span></p>
<h3><span class="mw-headline" id="Ring_species">Ring species</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Main article: Ring species</div>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><img alt="" class="thumbimage" data-file-height="2340" data-file-width="3493" height="201" src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/26/Ring_Species_%28gene_flow_around_a_barrier%29.png/300px-Ring_Species_%28gene_flow_around_a_barrier%29.png" srcset width="300"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>7</h1></center></div>In a ring species, gene flow occurs between neighboring populations, but at the ends of the "ring", the populations cannot interbreed.</div></div></div>
<p>A ring species is a connected series of populations, each of which can interbreed with its neighbors, with at least two "end" populations which are too distantly related to interbreed, though with the potential for gene flow between all the populations. Ring species represent speciation and have been cited as evidence of evolution. They illustrate what happens over time as populations genetically diverge, specifically because they represent, in living populations, what normally happens over time between long deceased ancestor populations and living populations, in which the intermediates have become extinct. Richard Dawkins says that ring species "are only showing us in the spatial dimension something that must always happen in the time dimension".
<span style="border: 3px solid blue; color:blue;font-weight: bold">71</span></p>
<h3><span class="mw-headline" id="Specific_examples_from_biogeography">Specific examples from biogeography</span></h3>
<link href="mw-data:TemplateStyles:r856303569" rel="mw-deduplicated-inline-style"/><div class="thumb tmulti tright"><div class="thumbinner" style="width:304px;max-width:304px"><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="" data-file-height="389" data-file-width="633" height="184" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/67/Pangaea_Glossopteris.jpg/300px-Pangaea_Glossopteris.jpg" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 4c:</b> Current distribution of <i>Glossopteris</i> placed on a Permian map showing the connection of the continents. (1, South America; 2, Africa; 3, Madagascar; 4, India; 5, Antarctica; and 6, Australia). Note that the map is a rough approximation of which leaves out additional land masses such as the Eurasian and North American plates.</div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="" data-file-height="3935" data-file-width="2944" height="401" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Phylogenetic_tree_of_marsupials_derived_from_retroposon_data_-_journal.pbio.1000436.g002.png/300px-Phylogenetic_tree_of_marsupials_derived_from_retroposon_data_-_journal.pbio.1000436.g002.png" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 4d</b>: A simplified phylogenetic tree of marsupials showing which groups reside on each continent.</div></div><div class="tsingle" style="margin:1px;width:302px;max-width:302px"><div class="thumbimage"><img alt="" data-file-height="958" data-file-width="1714" height="168" src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/Global_Camelid_Distribution_and_Migration.png/300px-Global_Camelid_Distribution_and_Migration.png" srcset width="300"/></div><div class="thumbcaption" style="clear:left"><b>Figure 4e</b>: A dymaxion map of the biogeographic distribution of Camelidae species. Light blue indicates the Tertiary distribution, dark blue indicates the present-day distributions, and green indicates the introduced (feral) distributions. The yellow dot is the origin of the family Camelidae and the black arrows are the historic migration routes that explain the present day distribution.</div></div></div></div>
<h4><span class="mw-headline" id="Distribution_of_Glossopteris">Distribution of <i>Glossopteris</i></span></h4>
<p>The combination of continental drift and evolution can sometimes be used to predict what will be found in the fossil record. <i>Glossopteris</i> is an extinct species of seed fern plants from the Permian. <i>Glossopteris</i> appears in the fossil record around the beginning of the Permian on the ancient continent of Gondwana. Continental drift explains the current biogeography of the tree. Present day <i>Glossopteris</i> fossils are found in Permian strata in southeast South America, southeast Africa, all of Madagascar, northern India, all of Australia, all of New Zealand, and scattered on the southern and northern edges of Antarctica. During the Permian, these continents were connected as Gondwana (see figure 4c) in agreement with magnetic striping, other fossil distributions, and glacial scratches pointing away from the temperate climate of the South Pole during the Permian.
<span style="border: 3px solid blue; color:blue;font-weight: bold">72</span></p>
<h4><span class="mw-headline" id="Metatherian_distribution">Metatherian distribution</span></h4>
<p>The history of metatherians (the clade containing marsupials and their extinct, primitive ancestors) provides an example of how evolutionary theory and the movement of continents can be combined to make predictions concerning fossil stratigraphy and distribution. The oldest metatherian fossils are found in present-day China. Metatherians spread westward into modern North America (still attached to Eurasia) and then to South America, which was connected to North America until around 65 mya. Marsupials reached Australia via Antarctica about 50 mya, shortly after Australia had split off suggesting a single dispersion event of just one species. Evolutionary theory suggests that the Australian marsupials descended from the older ones found in the Americas. Geologic evidence suggests that between 30 and 40 million years ago South America and Australia were still part of the Southern Hemisphere super continent of Gondwana and that they were connected by land that is now part of Antarctica. Therefore, when combining the models, scientists could predict that marsupials migrated from what is now South America, through Antarctica, and then to present-day Australia between 40 and 30 million years ago. A first marsupial fossil of the extinct family Polydolopidae was found on Seymour Island on the Antarctic Peninsula in 1982. Further fossils have subsequently been found, including members of the marsupial orders Didelphimorphia (opossum) and Microbiotheria, as well as ungulates and a member of the enigmatic extinct order Gondwanatheria, possibly <i>Sudamerica ameghinoi</i>.
<span style="border: 3px solid blue; color:blue;font-weight: bold">73</span></p>
<h4><span id="Migration.2C_isolation.2C_and_distribution_of_the_camel"></span><span class="mw-headline" id="Migration,_isolation,_and_distribution_of_the_camel">Migration, isolation, and distribution of the camel</span></h4>
<p>The history of the camel provides an example of how fossil evidence can be used to reconstruct migration and subsequent evolution. The fossil record indicates that the evolution of camelids started in North America (see figure 4e), from which, six million years ago, they migrated across the Bering Strait into Asia and then to Africa, and 3.5 million years ago through the Isthmus of Panama into South America. Once isolated, they evolved along their own lines, giving rise to the Bactrian camel and dromedary in Asia and Africa and the llama and its relatives in South America. Camelids then became extinct in North America at the end of the last ice age.
<span style="border: 3px solid blue; color:blue;font-weight: bold">74</span></p>
<h2><span class="mw-headline" id="Evidence_from_selection">Evidence from selection</span></h2>
<p>Examples for the evidence for evolution often stem from direct observation of natural selection in the field and the laboratory. This section is unique in that it provides a narrower context concerning the process of selection. All of the examples provided prior to this have described the evidence that evolution has occurred, but has not provided the major underlying mechanism: natural selection. This section explicitly provides evidence that natural selection occurs, has been replicated artificially, and can be replicated in laboratory experiments.
<span style="border: 3px solid blue; color:blue;font-weight: bold">75</span></p><p>Scientists have observed and documented a multitude of events where natural selection is in action. The most well known examples are antibiotic resistance in the medical field along with better-known laboratory experiments documenting evolution's occurrence. Natural selection is tantamount to common descent in that long-term occurrence and selection pressures can lead to the diversity of life on earth as found today. All adaptations—documented and undocumented changes concerned—are caused by natural selection (and a few other minor processes). It is well established that, "...natural selection is a ubiquitous part of speciation...", and is the primary driver of speciation; therefore, the following examples of natural selection <i>and</i> speciation will often interdepend or correspond with one another. The examples below are only a small fraction of the actual experiments and observations.
<span style="border: 3px solid blue; color:blue;font-weight: bold">76</span></p>
<h3><span class="mw-headline" id="Artificial_selection_and_experimental_evolution">Artificial selection and experimental evolution</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:227px;"><img alt="" class="thumbimage" data-file-height="504" data-file-width="600" height="189" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e9/Big_and_little_dog_1.jpg/225px-Big_and_little_dog_1.jpg" srcset width="225"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>8</h1></center></div><b>Figure 5a:</b> The Chihuahua mix and Great Dane illustrate the range of sizes among dog breeds.</div></div></div>
<p>Artificial selection demonstrates the diversity that can exist among organisms that share a relatively recent common ancestor. In artificial selection, one species is bred selectively at each generation, allowing only those organisms that exhibit desired characteristics to reproduce. These characteristics become increasingly well developed in successive generations. Artificial selection was successful long before science discovered the genetic basis. Examples of artificial selection include dog breeding, genetically modified food, flower breeding, and the cultivation of foods such as wild cabbage, and others.
<span style="border: 3px solid blue; color:blue;font-weight: bold">77</span></p><p>Experimental evolution uses controlled experiments to test hypotheses and theories of evolution. In one early example, William Dallinger set up an experiment shortly before 1880, subjecting microbes to heat with the aim of forcing adaptive changes. His experiment ran for around seven years, and his published results were acclaimed, but he did not resume the experiment after the apparatus failed.
<span style="border: 3px solid blue; color:blue;font-weight: bold">78</span></p><p>A large-scale example of experimental evolution is Richard Lenski's multi-generation experiment with <i>Escherichia coli</i>. Lenski observed that some strains of <i>E. coli</i> evolved a complex new ability, the ability to metabolize citrate, after tens of thousands of generations. The evolutionary biologist Jerry Coyne commented as a critique of creationism, saying, "the thing I like most is it says you can get these complex traits evolving by a combination of unlikely events. That's just what creationists say can't happen." In addition to the metabolic changes, the different bacterial populations were found to have diverged in respect to both morphology (the overall size of the cell) and fitness (of which was measured in competition with the ancestors). The <i>E. coli</i> long-term evolution experiment that began in 1988 is still in progress, and has shown adaptations including the evolution of a strain of <i>E. coli</i> that was able to grow on citric acid in the growth media—a trait absent in all other known forms of <i>E. coli</i>, including the initial strain.
<span style="border: 3px solid blue; color:blue;font-weight: bold">79</span></p>
<h3><span class="mw-headline" id="Invertebrates">Invertebrates</span></h3>
<h4><span class="mw-headline" id="Historical_lead_tolerance_in_Daphnia">Historical lead tolerance in <i>Daphnia</i></span></h4>
<p>A study of species of <i>Daphnia</i> and lead pollution in the 20th century predicted that an increase in lead pollution would lead to strong selection of lead tolerance. Researchers were able to use "resurrection ecology", hatching decades-old <i>Daphnia</i> eggs from the time when lakes were heavily polluted with lead. The hatchlings in the study were compared to current-day <i>Daphnia</i>, and demonstrated "dramatic fitness differences between old and modern phenotypes when confronted with a widespread historical environmental stressor". Essentially, the modern-day <i>Daphnia</i> were unable to resist or tolerate high levels of lead (this is due to the huge reduction of lead pollution in 21st century lakes). The old hatchlings, however, were able to tolerate high lead pollution. The authors concluded that "by employing the techniques of resurrection ecology, we were able to show clear phenotypic change over decades...".
<span style="border: 3px solid blue; color:blue;font-weight: bold">80</span></p>
<h4><span class="mw-headline" id="Peppered_moths">Peppered moths</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main article: Peppered moth evolution</div>
<p>A classic example was the phenotypic change, light-to-dark color adaptation, in the peppered moth, due to pollution from the Industrial Revolution in England.
<span style="border: 3px solid blue; color:blue;font-weight: bold">81</span></p>
<h3><span class="mw-headline" id="Microbes">Microbes</span></h3>
<h4><span class="mw-headline" id="Antimicrobial_resistance">Antimicrobial resistance</span></h4>
<div class="hatnote navigation-not-searchable" role="note">Main article: Antimicrobial resistance</div>
<p>The development and spread of antibiotic-resistant bacteria is evidence for the process of evolution of species. Thus the appearance of vancomycin-resistant <i>Staphylococcus aureus</i>, and the danger it poses to hospital patients, is a direct result of evolution through natural selection. The rise of <i>Shigella</i> strains resistant to the synthetic antibiotic class of sulfonamides also demonstrates the generation of new information as an evolutionary process. Similarly, the appearance of DDT resistance in various forms of <i>Anopheles</i> mosquitoes, and the appearance of myxomatosis resistance in breeding rabbit populations in Australia, are both evidence of the existence of evolution in situations of evolutionary selection pressure in species in which generations occur rapidly.
<span style="border: 3px solid blue; color:blue;font-weight: bold">82</span></p><p>All classes of microbes develop resistance: including fungi (antifungal resistance), viruses (antiviral resistance), protozoa (antiprotozoal resistance), and bacteria (antibiotic resistance). This is to be expected when considering that all life exhibits universal genetic code and is therefore subject to the process of evolution through its various mechanisms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">83</span></p>
<h4><span class="mw-headline" id="Nylon-eating_bacteria">Nylon-eating bacteria</span></h4>
<p>Another example of organisms adapting to human-caused conditions are Nylon-eating bacteria: a strain of <i>Flavobacterium</i> that are capable of digesting certain byproducts of nylon 6 manufacturing. There is scientific consensus that the capacity to synthesize nylonase most probably developed as a single-step mutation that survived because it improved the fitness of the bacteria possessing the mutation. This is seen as a good example of evolution through mutation and natural selection that has been observed as it occurs and could not have come about until the production of nylon by humans.
<span style="border: 3px solid blue; color:blue;font-weight: bold">84</span></p>
<h3><span class="mw-headline" id="Plants_and_fungi">Plants and fungi</span></h3>
<h4><span class="mw-headline" id="Monkeyflower_radiation">Monkeyflower radiation</span></h4>
<p>Both subspecies <i>Mimulus aurantiacus puniceus</i> (red-flowered) and <i>Mimulus aurantiacus australis</i> (yellow-flowered) of monkeyflowers are isolated due to the preferences of their hummingbird and hawkmoth pollinators. The radiation of <i>M. aurantiacus</i> subspecies are mostly yellow colored; however, both <i>M. a.</i> ssp. <i>puniceus</i> and <i>M. a.</i> ssp. <i>flemingii</i> are red. Phylogenetic analysis suggests two independent origins of red-colored flowers that arose due to <i>cis</i>-regulatory mutations in the gene <i>MaMyb2</i> that is present in all <i>M. aurantiacus</i> subspecies. Further research suggested that two independent mutations did not take place, but one <i>MaMyb2</i> allele was transferred via introgressive hybridization. This study presents an example of the overlap of research in various disciplines. Gene isolation and <i>cis</i>-regulatory functions; phylogenetic analysis; geographic location and pollinator preference; and species hybridization and speciation are just some of the areas in which data can be obtained to document the occurrence of evolution.
<span style="border: 3px solid blue; color:blue;font-weight: bold">85</span></p>
<h4><span class="mw-headline" id="Radiotrophic_fungi">Radiotrophic fungi</span></h4>
<p>Like the codfish, human-caused pollution can come in different forms. Radiotrophic fungi is a perfect example of natural selection taking place after a chemical accident. Radiotrophic fungi appears to use the pigment melanin to convert gamma radiation into chemical energy for growth and were first discovered in 2007 as black molds growing inside and around the Chernobyl Nuclear Power Plant. Research at the Albert Einstein College of Medicine showed that three melanin-containing fungi, <i>Cladosporium sphaerospermum</i>, <i>Wangiella dermatitidis</i>, and <i>Cryptococcus neoformans</i>, increased in biomass and accumulated acetate faster in an environment in which the radiation level was 500 times higher than in the normal environment.
<span style="border: 3px solid blue; color:blue;font-weight: bold">86</span></p>
<h3><span class="mw-headline" id="Vertebrates">Vertebrates</span></h3>
<h4><span class="mw-headline" id="Guppies">Guppies</span></h4>
<p>While studying guppies (<i>Poecilia reticulata</i>) in Trinidad, biologist John Endler detected selection at work on the fish populations. To rule out alternative possibilities, Endler set up a highly controlled experiment to mimic the natural habitat by constructing ten ponds within a laboratory greenhouse at Princeton University. Each pond contained gravel to exactly match that of the natural ponds. After capturing a random sample of guppies from ponds in Trinidad, he raised and mixed them to create similar genetically diverse populations and measured each fish (spot length, spot height, spot area, relative spot length, relative spot height, total patch area, and standard body lengths). For the experiment he added <i>Crenicichla alta</i> (<i>P. reticulata'</i>s main predator) in four of the ponds, <i>Rivulus hartii</i> (a non-predator fish) in four of the ponds, and left the remaining two ponds empty with only the guppies. After 10 generations, comparisons were made between each pond's guppy populations and measurements were taken again. Endler found that the populations had evolved dramatically different color patterns in the control and non-predator pools and drab color patterns in the predator pool. Predation pressure had caused a selection against standing out from background gravel.
<span style="border: 3px solid blue; color:blue;font-weight: bold">87</span></p>
<div class="thumb tleft"><div class="thumbinner" style="width:227px;"><img alt="" class="thumbimage" data-file-height="1506" data-file-width="2719" height="125" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5e/Guppy_CS_pair_4_20130113.jpg/225px-Guppy_CS_pair_4_20130113.jpg" srcset width="225"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>9</h1></center></div><b>Figure 5b</b>: Endler's Trinadadian guppies (<i>Poecilia reticulata</i>)</div></div></div>
<p>In parallel, during this experiment, Endler conducted a field experiment in Trinidad where he caught guppies from ponds where they had predators and relocated them to ponds upstream where the predators did not live. After 15 generations, Endler found that the relocated guppies had evolved dramatic and colorful patterns. Essentially, both experiments showed convergence due to similar selection pressures (i.e. predator selection against contrasting color patterns and sexual selection for contrasting color patterns).
<span style="border: 3px solid blue; color:blue;font-weight: bold">88</span></p><p>In a later study by David Reznick, the field population was examined 11 years later after Endler relocated the guppies to high streams. The study found that the populations has evolved in a number of different ways: bright color patterns, late maturation, larger sizes, smaller litter sizes, and larger offspring within litters. Further studies of <i>P. reticulata</i> and their predators in the streams of Trinidad have indicated that varying modes of selection through predation have not only changed the guppies color patterns, sizes, and behaviors, but their life histories and life history patterns.
<span style="border: 3px solid blue; color:blue;font-weight: bold">89</span></p>
<h4><span class="mw-headline" id="Humans">Humans</span></h4>
<p>Natural selection is observed in contemporary human populations, with recent findings demonstrating that the population at risk of the severe debilitating disease kuru has significant over-representation of an immune variant of the prion protein gene G127V versus non-immune alleles. Scientists postulate one of the reasons for the rapid selection of this genetic variant is the lethality of the disease in non-immune persons. Other reported evolutionary trends in other populations include a lengthening of the reproductive period, reduction in cholesterol levels, blood glucose and blood pressure.
<span style="border: 3px solid blue; color:blue;font-weight: bold">90</span></p><p>A well known example of selection occurring in human populations is lactose tolerance. Lactose intolerance is the inability to metabolize lactose, because of a lack of the required enzyme lactase in the digestive system. The normal mammalian condition is for the young of a species to experience reduced lactase production at the end of the weaning period (a species-specific length of time). In humans, in non-dairy consuming societies, lactase production usually drops about 90% during the first four years of life, although the exact drop over time varies widely. Lactase activity persistence in adults is associated with two polymorphisms: C/T 13910 and G/A 22018 located in the <i>MCM6</i> gene. This gene difference eliminates the shutdown in lactase production, making it possible for members of these populations to continue consumption of raw milk and other fresh and fermented dairy products throughout their lives without difficulty. This appears to be an evolutionarily recent (around 10,000 years ago [and 7,500 years ago in Europe]) adaptation to dairy consumption, and has occurred independently in both northern Europe and east Africa in populations with a historically pastoral lifestyle.
<span style="border: 3px solid blue; color:blue;font-weight: bold">91</span></p>
<h4><span class="mw-headline" id="Italian_wall_lizards">Italian wall lizards</span></h4>
<p>In 1971, ten adult specimens of <i>Podarcis sicula</i> (the Italian wall lizard) were transported from the Croatian island of Pod Kopište to the island Pod Mraru (about 3.5 km to the east). Both islands lie in the Adriatic Sea near Lastovo, where the lizards founded a new bottlenecked population. The two islands have similar size, elevation, microclimate, and a general absence of terrestrial predators and the <i>P. sicula</i> expanded for decades without human interference, even out-competing the (now locally extinct) <i>Podarcis melisellensis</i> population.
<span style="border: 3px solid blue; color:blue;font-weight: bold">92</span></p><p>In the 1990s, scientists returned to Pod Mraru and found that the lizards there differed greatly from those on Kopište. While mitochondrial DNA analyses have verified that <i>P. sicula</i> currently on Mraru are genetically very similar to the Kopište source population, the new Mraru population of <i>P. sicula</i> had a larger average size, shorter hind limbs, lower maximal sprint speed and altered response to simulated predatory attacks compared to the original Kopište population. These changes were attributed to "relaxed predation intensity" and greater protection from vegetation on Mraru.
<span style="border: 3px solid blue; color:blue;font-weight: bold">93</span></p><p>In 2008, further analysis revealed that the Mraru population of <i>P. sicula</i> have significantly different head morphology (longer, wider, and taller heads) and increased bite force compared to the original Kopište population. This change in head shape corresponded with a shift in diet: Kopište <i>P. sicula</i> are primarily insectivorous, but those on Mraru eat substantially more plant matter. The changes in foraging style may have contributed to a greater population density and decreased territorial behavior of the Mraru population.
<span style="border: 3px solid blue; color:blue;font-weight: bold">94</span></p><p>Another difference found between the two populations was the discovery, in the Mraru lizards, of cecal valves, which slow down food passage and provide fermenting chambers, allowing commensal microorganisms to convert cellulose to nutrients digestible by the lizards. Additionally, the researchers discovered that nematodes were common in the guts of Mraru lizards, but absent from Kopište <i>P. sicula</i>, which do not have cecal valves. The cecal valves, which occur in less than 1 percent of all known species of scaled reptiles, have been described as an "adaptive novelty, a brand new feature not present in the ancestral population and newly evolved in these lizards".
<span style="border: 3px solid blue; color:blue;font-weight: bold">95</span></p>
<h4><span class="mw-headline" id="PAH_resistance_in_killifish">PAH resistance in killifish</span></h4>
<p>A similar study was also done regarding the polycyclic aromatic hydrocarbons (PAHs) that pollute the waters of the Elizabeth River in Portsmouth, Virginia. This chemical is a product of creosote, a type of tar. The Atlantic killifish (<i>Fundulus heteroclitus</i>) has evolved a resistance to PAHs involving the AHR gene (the same gene involved in the tomcods). This particular study focused on the resistance to "acute toxicity and cardiac teratogenesis" caused by PAHs.
that mutated within the tomcods in the Hudson River.
<span style="border: 3px solid blue; color:blue;font-weight: bold">96</span></p>
<h4><span class="mw-headline" id="PCB_resistance_in_codfish">PCB resistance in codfish</span></h4>
<p>An example involving the direct observation of gene modification due to selection pressures is the resistance to PCBs in codfish. After General Electric dumped polychlorinated biphenyls (PCBs) in the Hudson River from 1947 through 1976, tomcods (<i>Microgadus tomcod</i>) living in the river were found to have evolved an increased resistance to the compound's toxic effects. The tolerance to the toxins is due to a change in the coding section of specific gene. Genetic samples were taken from the cods from 8 different rivers in the New England region: the St. Lawrence River, Miramichi River, Margaree River, Squamscott River, Niantic River, the Shinnecock Basic, the Hudson River, and the Hackensack River. Genetic analysis found that in the population of tomcods in the four southernmost rivers, the gene AHR2 (aryl hydrocarbon receptor 2) was present as an allele with a difference of two amino acid deletions. This deletion conferred a resistance to PCB in the fish species and was found in 99% of Hudson River tomcods, 92% in the Hackensack River, 6% in the Niantic River, and 5% in Shinnecock Bay. This pattern along the sampled bodies of waters infers a direct correlation of selective pressures leading to the evolution of PCB resistance in Atlantic tomcod fish.
<span style="border: 3px solid blue; color:blue;font-weight: bold">97</span></p>
<h4><span class="mw-headline" id="Urban_wildlife">Urban wildlife</span></h4>
<p>Urban wildlife is a broad and easily observable case of human-caused selection pressure on wildlife. With the growth in human habitats, different animals have adapted to survive within these urban environments. These types of environments can exert selection pressures on organisms, often leading to new adaptations. For example, the weed <i>Crepis sancta</i>, found in France, has two types of seed, heavy and fluffy. The heavy ones land nearby to the parent plant, whereas fluffy seeds float further away on the wind. In urban environments, seeds that float far often land on infertile concrete. Within about 5–12 generations, the weed evolves to produce significantly heavier seeds than its rural relatives. Other examples of urban wildlife are rock pigeons and species of crows adapting to city environments around the world; African penguins in Simon's Town; baboons in South Africa; and a variety of insects living in human habitations. Studies have been conducted and have found striking changes to animals' (more specifically mammals') behavior and physical brain size due to their interactions with human-created environments.
<span style="border: 3px solid blue; color:blue;font-weight: bold">98</span></p>
<h4><span class="mw-headline" id="White_Sands_lizards">White Sands lizards</span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:227px;"><img alt="" class="thumbimage" data-file-height="1260" data-file-width="2262" height="125" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/WhiteSandsBleachedEarlessLizard.jpg/225px-WhiteSandsBleachedEarlessLizard.jpg" srcset width="225"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>10</h1></center></div><b>Figure 5c</b>: <i>H. m. ruthveni</i>, a White Sands ecotonal variant of <i>Holbrookia maculata</i></div></div></div>
<p>Animals that exhibit ecotonal variations allow for research concerning the mechanisms that maintain population differentiation. A wealth of information about natural selection, genotypic, and phenotypic variation; adaptation and ecomorphology; and social signaling has been acquired from the studies of three species of lizards located in the White Sands desert of New Mexico. <i>Holbrookia maculata</i>, <i>Aspidoscelis inornata</i>, and <i>Sceloporus undulatus</i> exhibit ecotonal populations that match both the dark soils and the white sands in the region. Research conducted on these species has found significant phenotypic and genotypic differences between the dark and light populations due to strong selection pressures. For example, <i>H. maculata</i> exhibits the strongest phenotypic difference (matches best with the substrate) of the light colored population coinciding with the least amount of gene flow between the populations and the highest genetic differences when compared to the other two lizard species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">99</span></p><p>New Mexico's White Sands are a recent geologic formation (approximately 6000 years old to possibly 2000 years old). This recent origin of these gypsum sand dunes suggests that species exhibiting lighter-colored variations have evolved in a relatively short time frame. The three lizard species previously mentioned have been found to display variable social signal coloration in coexistence with their ecotonal variants. Not only have the three species convergently evolved their lighter variants due to the selection pressures from the environment, they've also evolved ecomorphological differences: morphology, behavior (in is case, escape behavior), and performance (in this case, sprint speed) collectively. Roches' work found surprising results in the escape behavior of <i>H. maculata</i> and <i>S. undulatus</i>. When dark morphs were placed on white sands, their startle response was significantly diminished. This result could be due to varying factors relating to sand temperature or visual acuity; however, regardless of the cause, "…failure of mismatched lizards to sprint could be maladaptive when faced with a predator".
<span style="border: 3px solid blue; color:blue;font-weight: bold">100</span></p>
<h2><span class="mw-headline" id="Evidence_from_speciation">Evidence from speciation</span></h2>
<div class="hatnote navigation-not-searchable" role="note">Main articles: Speciation and Laboratory experiments of speciation</div>
<div class="hatnote navigation-not-searchable" role="note">Further information: Allopatric speciation, Peripatric speciation, Parapatric speciation, Sympatric speciation, and Evidence for speciation by reinforcement</div>
<p>Speciation is the evolutionary process by which new biological species arise. Biologists research species using different theoretical frameworks for what constitutes a species (see species problem and species complex) and there exists debate with regard to delineation. Nevertheless, much of the current research suggests that, "...speciation is a process of emerging genealogical distinctness, rather than a discontinuity affecting all genes simultaneously" and, in allopatry (the most common form of speciation), "reproductive isolation is a byproduct of evolutionary change in isolated populations, and thus can be considered an evolutionary accident". Speciation occurs as the result of the latter (allopatry); however, a variety of differing agents have been documented and are often defined and classified in various forms (e.g. peripatric, parapatric, sympatric, polyploidization, hybridization, etc.). Instances of speciation have been observed in both nature and the laboratory. A.-B Florin and A. Ödeen note that, "strong laboratory evidence for allopatric speciation is lacking..."; however, contrary to laboratory studies (focused specifically on models of allopatric speciation), "speciation most definitely occurs; [and] the vast amount of evidence from nature makes it unreasonable to argue otherwise". Coyne and Orr compiled a list of 19 laboratory experiments on <i>Drosophila</i> presenting examples of allopatric speciation by divergent selection concluding that, "reproductive isolation in allopatry can evolve as a byproduct of divergent selection".
<span style="border: 3px solid blue; color:blue;font-weight: bold">101</span></p><p>Research documenting speciation is abundant. Biologists have documented numerous examples of speciation in nature—with evolution having produced far more species than any observer would consider necessary. For example, there are well over 350,000 described species of beetles. Examples of speciation come from the observations of island biogeography and the process of adaptive radiation, both explained previously. Evidence of common descent can also be found through paleontological studies of speciation within geologic strata. The examples described below represent different modes of speciation and provide strong evidence for common descent. It is important to acknowledge that not all speciation research directly observes divergence from "start-to-finish". This is by virtue of research delimitation and definition ambiguity, and occasionally leads research towards historical reconstructions. In light of this, examples abound, and the following are by no means exhaustive—comprising only a small fraction of the instances observed. Once again, take note of the established fact that, "...natural selection is a ubiquitous part of speciation...", and is the primary driver of speciation, so; hereinafter, examples of speciation will often interdepend and correspond with selection.
<span style="border: 3px solid blue; color:blue;font-weight: bold">102</span></p>
<h3><span class="mw-headline" id="Fossils">Fossils</span></h3>
<p>Limitations exist within the fossil record when considering the concept of what constitutes a species. Paleontologists largely rely on a different framework: the morphological species concept. Due to the absence of information such as reproductive behavior or genetic material in fossils, paleontologists distinguish species by their phenotypic differences. Extensive investigation of the fossil record has led to numerous theories concerning speciation (in the context of paleontology) with many of the studies suggesting that stasis, punctuation, and lineage branching are common. In 1995, D. H. Erwin, et al. published a major work—<i>New Approaches to Speciation in the Fossil Record</i>—which compiled 58 studies of fossil speciation (between 1972 and 1995) finding most of the examples suggesting stasis (involving anagenesis or punctuation) and 16 studies suggesting speciation. Despite stasis appearing to be the predominate conclusion at first glance, this particular meta-study investigated deeper, concluding that, "...no single pattern appears dominate..." with "...the preponderance of studies illustrating <i>both</i> stasis and gradualism in the history of a single lineage". Many of the studies conducted utilize seafloor sediments that can provide a significant amount of data concerning planktonic microfossils. The succession of fossils in stratigraphy can be used to determine evolutionary trends among fossil organisms. In addition, incidences of speciation can be interpreted from the data and numerous studies have been conducted documenting both morphological evolution and speciation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">103</span></p>
<h4><span class="mw-headline" id="Globorotalia"><i>Globorotalia</i></span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:227px;"><img alt="" class="thumbimage" data-file-height="915" data-file-width="535" height="385" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/Globorotalia_Speciation_and_Phylogeny.png/225px-Globorotalia_Speciation_and_Phylogeny.png" srcset width="225"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>11</h1></center></div><b>Figure 6a</b>: Morphologic change of <i>Globorotalia crassaformis, G. tosaensis, and G. truncatulinoides</i> over 3.5 Ma. Superimposed is a phylogenetic tree of the group. Adapted from Lazarus <i>et al.</i> (1995).</div></div></div>
<p>Extensive research on the planktonic foraminifer <i>Globorotalia truncatulinoides</i> has provided insight into paleobiogeographical and paleoenvironmental studies alongside the relationship between the environment and evolution. In an extensive study of the paleobiogeography of <i>G. truncatulinoides</i>, researchers found evidence that suggested the formation of a new species (via the sympatric speciation framework). Cores taken of the sediment containing the three species <i>G. crassaformis</i>, <i>G. tosaensis</i>, and <i>G. truncatulinoides</i> found that before 2.7 Ma, only <i>G. crassaformis</i> and <i>G. tosaensis</i> existed. A speciation event occurred at that time, whereby intermediate forms existed for quite some time. Eventually <i>G. tosaensis</i> disappears from the record (suggesting extinction) but exists as an intermediate between the extant <i>G. crassaformis</i> and <i>G. truncatulinoides</i>. This record of the fossils also matched the already existing phylogeny constructed by morphological characters of the three species. See figure 6a.
<span style="border: 3px solid blue; color:blue;font-weight: bold">104</span></p>
<h4><span class="mw-headline" id="Radiolaria">Radiolaria</span></h4>
<p>In a large study of five species of radiolarians (<i>Calocycletta caepa</i>, <i>Pterocanium prismatium</i>, <i>Pseudoculous vema</i>, <i>Eucyrtidium calvertense</i>, and <i>Eucyrtidium matuyamai</i>), the researchers documented considerable evolutionary change in each lineage. Alongside this, trends with the closely related species <i>E. calvertense</i> and <i>E. matuyamai</i> showed that about 1.9 Mya <i>E. calvertense</i> invaded a new region of the Pacific, becoming isolated from the main population. The stratigraphy of this species clearly shows that this isolated population evolved into <i>E. Matuyamai</i>. It then reinvaded the region of the still-existing and static <i>E. calvertense</i> population whereby a sudden decrease in body size occurred. Eventually the invader <i>E. matuyamai</i> disappeared from the stratum (presumably due to extinction) coinciding with a desistance of size reduction of the <i>E. calvertense</i> population. From that point on, the change in size leveled to a constant. The authors suggest competition-induced character displacement.
<span style="border: 3px solid blue; color:blue;font-weight: bold">105</span></p>
<h4><span class="mw-headline" id="Rhizosolenia"><i>Rhizosolenia</i></span></h4>
<p>Researchers conducted measurements on 5,000 <i>Rhizosolenia</i> (a planktonic diatom) specimens from eight sedimentary cores in the Pacific Ocean. The core samples spanned two million years and were chronologized using sedimentary magnetic field reversal measurements. All the core samples yielded a similar pattern of divergence: with a single lineage (<i>R. bergonii</i>) occurring before 3.1 Mya and two morphologically distinct lineages (daughter species: <i>R. praebergonii</i>) appearing after. The parameters used to measure the samples were consistent throughout each core. An additional study of the daughter species <i>R. praebergonii</i> found that, after the divergence, it invaded the Indian Ocean.
<span style="border: 3px solid blue; color:blue;font-weight: bold">106</span></p>
<h4><span class="mw-headline" id="Turborotalia"><i>Turborotalia</i></span></h4>
<p>A recent study was conducted involving the planktonic foraminifer Turborotalia. The authors extracted "51 stratigraphically ordered samples from a site within the oceanographically stable tropical North Pacific gyre". Two hundred individual species were examined using ten specific morphological traits (size, compression index, chamber aspect ratio, chamber inflation, aperture aspect ratio, test height, test expansion, umbilical angle, coiling direction, and the number of chambers in the final whorl). Utilizing multivariate statistical clustering methods, the study found that the species continued to evolve non-directionally within the Eocene from 45 Ma to about 36 Ma. However, from 36 Ma to approximately 34 Ma, the stratigraphic layers showed two distinct clusters with significantly defining characteristics distinguishing one another from a single species. The authors concluded that speciation must have occurred and that the two new species were ancestral to the prior species. Just as in most of evolutionary biology, this example represents the interdisciplinary nature of the field and the necessary collection of data from various fields (e.g. oceanography, paleontology) and the integration of mathematical analysis (e.g. biometry).
<span style="border: 3px solid blue; color:blue;font-weight: bold">107</span></p>
<h4><span class="mw-headline" id="Vertebrates_2">Vertebrates</span></h4>
<p>There exists evidence for vertebrate speciation despite limitations imposed by the fossil record. Studies have been conducted documenting similar patterns seen in marine invertebrates. For example, extensive research documenting rates of morphological change, evolutionary trends, and speciation patterns in small mammals has significantly contributed to the scientific literature; once more, demonstrating that evolution (and speciation) occurred in the past and lends support common ancestry.
<span style="border: 3px solid blue; color:blue;font-weight: bold">108</span></p><p>A study of four mammalian genera: <i>Hyopsodus</i>, <i>Pelycodus</i>, <i>Haplomylus</i> (three from the Eocene), and <i>Plesiadapis</i> (from the Paleocene) found that—through a large number of stratigraphic layers and specimen sampling—each group exhibited, "gradual phyletic evolution, overall size increase, iterative evolution of small species, and character divergence following the origin of each new lineage". The authors of this study concluded that speciation was discernible. In another study concerning morphological trends and rates of evolution found that the European arvicolid rodent radiated into 52 distinct lineages over a time frame of 5 million years while documenting examples of phyletic gradualism, punctuation, and stasis.
<span style="border: 3px solid blue; color:blue;font-weight: bold">109</span></p>
<h3><span class="mw-headline" id="Invertebrates_2">Invertebrates</span></h3>
<h4><span class="mw-headline" id="Drosophila_melanogaster"><i>Drosophila melanogaster</i></span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:227px;"><img alt="" class="thumbimage" data-file-height="424" data-file-width="546" height="175" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Drosophila_melanogaster_-_side_%28aka%29.jpg/225px-Drosophila_melanogaster_-_side_%28aka%29.jpg" srcset width="225"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>12</h1></center></div><b>Figure 6b</b>: A common fruit fly (<i>Drosophila melanogaster</i>)</div></div></div>
<p>William R. Rice and George W. Salt found experimental evidence of sympatric speciation in the common fruit fly. They collected a population of <i>Drosophila melanogaster</i> from Davis, California and placed the pupae into a habitat maze. Newborn flies had to investigate the maze to find food. The flies had three choices to take in finding food. Light and dark (phototaxis), up and down (geotaxis), and the scent of acetaldehyde and the scent of ethanol (chemotaxis) were the three options. This eventually divided the flies into 42 spatio-temporal habitats.
<span style="border: 3px solid blue; color:blue;font-weight: bold">110</span></p><p>They then cultured two strains that chose opposite habitats. One of the strains emerged early, immediately flying upward in the dark attracted to the acetaldehyde. The other strain emerged late and immediately flew downward, attracted to light and ethanol. Pupae from the two strains were then placed together in the maze and allowed to mate at the food site. They then were collected. A selective penalty was imposed on the female flies that switched habitats. This entailed that none of their gametes would pass on to the next generation. After 25 generations of this mating test, it showed reproductive isolation between the two strains. They repeated the experiment again without creating the penalty against habitat switching and the result was the same; reproductive isolation was produced.
<span style="border: 3px solid blue; color:blue;font-weight: bold">111</span></p>
<h4><span class="mw-headline" id="Gall_wasps">Gall wasps</span></h4>
<p>A study of the gall-forming wasp species <i>Belonocnema treatae</i> found that populations inhabiting different host plants (<i>Quercus geminata</i> and <i>Q. Virginiana</i>) exhibited different body size and gall morphology alongside a strong expression of sexual isolation. The study hypothesized that <i>B. treatae</i> populations inhabiting different host plants would show evidence of divergent selection promoting speciation. The researchers sampled gall wasp species and oak tree localities, measured body size (right hand tibia of each wasp), and counted gall chamber numbers. In addition to measurements, they conducted mating assays and statistical analyses. Genetic analysis was also conducted on two mtDNA sites (416 base pairs from cytochrome C and 593 base pairs from cytochrome oxidase ) to "control for the confounding effects of time since divergence among allopatric populations".
<span style="border: 3px solid blue; color:blue;font-weight: bold">112</span></p><p>In an additional study, the researchers studied two gall wasp species <i>B. treatae</i> and <i>Disholcaspis quercusvirens</i> and found strong morphological and behavioral variation among host-associated populations. This study further confounded prerequisites to speciation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">113</span></p>
<h4><span class="mw-headline" id="Hawthorn_fly">Hawthorn fly</span></h4>
<p>One example of evolution at work is the case of the hawthorn fly, <i>Rhagoletis pomonella</i>, also known as the apple maggot fly, which appears to be undergoing sympatric speciation. Different populations of hawthorn fly feed on different fruits. A distinct population emerged in North America in the 19th century some time after apples, a non-native species, were introduced. This apple-feeding population normally feeds only on apples and not on the historically preferred fruit of hawthorns. The current hawthorn feeding population does not normally feed on apples. Some evidence, such as the fact that six out of thirteen allozyme loci are different, that hawthorn flies mature later in the season and take longer to mature than apple flies; and that there is little evidence of interbreeding (researchers have documented a 4–6% hybridization rate) suggests that speciation is occurring.
<span style="border: 3px solid blue; color:blue;font-weight: bold">114</span></p>
<h4><span class="mw-headline" id="London_Underground_mosquito">London Underground mosquito</span></h4>
<p>The London Underground mosquito is a species of mosquito in the genus <i>Culex</i> found in the London Underground. It evolved from the overground species <i>Culex pipiens</i>. This mosquito, although first discovered in the London Underground system, has been found in underground systems around the world. It is suggested that it may have adapted to human-made underground systems since the last century from local above-ground <i>Culex pipiens</i>, although more recent evidence suggests that it is a southern mosquito variety related to <i>Culex pipiens</i> that has adapted to the warm underground spaces of northern cities.
<span style="border: 3px solid blue; color:blue;font-weight: bold">115</span></p><p>The two species have very different behaviours, are extremely difficult to mate, and with different allele frequency, consistent with genetic drift during a founder event. More specifically, this mosquito, <i>Culex pipiens molestus</i>, breeds all-year round, is cold intolerant, and bites rats, mice, and humans, in contrast to the above ground species <i>Culex pipiens</i> that is cold tolerant, hibernates in the winter, and bites only birds. When the two varieties were cross-bred the eggs were infertile suggesting reproductive isolation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">116</span></p><p>The genetic data indicates that the <i>molestus</i> form in the London Underground mosquito appears to have a common ancestry, rather than the population at each station being related to the nearest aboveground population (i.e. the <i>pipiens</i> form). Byrne and Nichols' working hypothesis was that adaptation to the underground environment had occurred locally in London only once. These widely separated populations are distinguished by very minor genetic differences, which suggest that the molestus form developed: a single mtDNA difference shared among the underground populations of ten Russian cities; a single fixed microsatellite difference in populations spanning Europe, Japan, Australia, the middle East and Atlantic islands.
<span style="border: 3px solid blue; color:blue;font-weight: bold">117</span></p>
<h4><span class="mw-headline" id="Snapping_shrimp_and_the_isthmus_of_Panama">Snapping shrimp and the isthmus of Panama</span></h4>
<p>Debate exists determining when the isthmus of Panama closed. Much of the evidence supports a closure approximately 2.7 to 3.5 mya using "...multiple lines of evidence and independent surveys". However, a recent study suggests an earlier, transient bridge existed 13 to 15 mya. Regardless of the timing of the isthmus closer, biologists can study the species on the Pacific and Caribbean sides in, what has been called, "one of the greatest natural experiments in evolution." Studies of snapping shrimp in the genus <i>Alpheus</i> have provided direct evidence of allopatric speciation events, and contributed to the literature concerning rates of molecular evolution. Phylogenetic reconstructions using "multilocus datasets and coalescent-based analytical methods" support the relationships of the species in the group and molecular clock techniques support the separation of 15 pairs of <i>Alpheus</i> species between 3 and 15 million years ago.
<span style="border: 3px solid blue; color:blue;font-weight: bold">118</span></p>
<h3><span class="mw-headline" id="Plants">Plants</span></h3>
<p>The botanist Verne Grant pioneered the field of plant speciation with his research and major publications on the topic. As stated before, many biologists rely on the biological species concept, with some modern researchers utilizing the phylogenetic species concept. Debate exists in the field concerning which framework should be applied in the research. Regardless, reproductive isolation is the primary role in the process of speciation and has been studied extensively by biologists in their respective disciplines.
<span style="border: 3px solid blue; color:blue;font-weight: bold">119</span></p><p>Both hybridization and polyploidy have also been found to be major contributors to plant speciation. With the advent of molecular markers, "hybridization [is] considerably more frequent than previously believed". In addition to these two modes leading to speciation, pollinator preference and isolation, chromosomal rearrangements, and divergent natural selection have become critical to the speciation of plants. Furthermore, recent research suggests that sexual selection, epigenetic drivers, and the creation of incompatible allele combinations caused by balancing selection also contribute to the formation of new species. Instances of these modes have been researched in both the laboratory and in nature. Studies have also suggested that, due to "the sessile nature of plants... [it increases] the relative importance of ecological speciation..."
<span style="border: 3px solid blue; color:blue;font-weight: bold">120</span></p><p>Hybridization between two different species sometimes leads to a distinct phenotype. This phenotype can also be fitter than the parental lineage and as such, natural selection may then favor these individuals. Eventually, if reproductive isolation is achieved, it may lead to a separate species. However, reproductive isolation between hybrids and their parents is particularly difficult to achieve and thus hybrid speciation is considered a rare event. However, hybridization resulting in reproductive isolation is considered an important means of speciation in plants, since polyploidy (having more than two copies of each chromosome) is tolerated in plants more readily than in animals.
<span style="border: 3px solid blue; color:blue;font-weight: bold">121</span></p><p>Polyploidy is important in hybrids as it allows reproduction, with the two different sets of chromosomes each being able to pair with an identical partner during meiosis. Polyploids also have more genetic diversity, which allows them to avoid inbreeding depression in small populations. Hybridization without change in chromosome number is called homoploid hybrid speciation. It is considered very rare but has been shown in <i>Heliconius</i> butterflies and sunflowers. Polyploid speciation, which involves changes in chromosome number, is a more common phenomenon, especially in plant species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">122</span></p><p>Polyploidy is a mechanism that has caused many rapid speciation events in sympatry because offspring of, for example, tetraploid x diploid matings often result in triploid sterile progeny. Not all polyploids are reproductively isolated from their parental plants, and gene flow may still occur for example through triploid hybrid x diploid matings that produce tetraploids, or matings between meiotically unreduced gametes from diploids and gametes from tetraploids. It has been suggested that many of the existing plant and most animal species have undergone an event of polyploidization in their evolutionary history. Reproduction of successful polyploid species is sometimes asexual, by parthenogenesis or apomixis, as for unknown reasons many asexual organisms are polyploid. Rare instances of polyploid mammals are known, but most often result in prenatal death.
<span style="border: 3px solid blue; color:blue;font-weight: bold">123</span></p><p>Researchers consider reproductive isolation as key to speciation. A major aspect of speciation research is to determine the nature of the barriers that inhibit reproduction. Botanists often consider the zoological classifications of prezygotic and postzygotic barriers as inadequate. The examples provided below give insight into the process of speciation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">124</span></p>
<h4><span class="mw-headline" id="Mimulus_peregrinus"><i>Mimulus peregrinus</i></span></h4>
<p>The creation of a new allopolyploid species of monkeyflower (<i>Mimulus peregrinus</i>) was observed on the banks of the Shortcleuch Water—a river in Leadhills, South Lanarkshire, Scotland. Parented from the cross of the two species <i>Mimulus guttatus</i> (containing 14 pairs of chromosomes) and <i>Mimulus luteus</i> (containing 30-31 pairs from a chromosome duplication), <i>M. peregrinus</i> has six copies of its chromosomes (caused by the duplication of the sterile hybrid triploid). Due to the nature of these species, they have the ability to self-fertilize. Because of its number of chromosomes it is not able to pair with <i>M. guttatus</i>, <i>M. luteus</i>, or their sterile triploid offspring. <i>M. peregrinus</i> will either die, producing no offspring, or reproduce with itself effectively leading to a new species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">125</span></p>
<h4><span class="mw-headline" id="Raphanobrassica"><i>Raphanobrassica</i></span></h4>
<p><i>Raphanobrassica</i> includes all intergeneric hybrids between the genera <i>Raphanus</i> (radish) and <i>Brassica</i> (cabbages, etc.). The <i>Raphanobrassica</i> is an allopolyploid cross between the radish (<i>Raphanus sativus</i>) and cabbage (<i>Brassica oleracea</i>). Plants of this parentage are now known as radicole. Two other fertile forms of <i>Raphanobrassica</i> are known. Raparadish, an allopolyploid hybrid between <i>Raphanus sativus</i> and <i>Brassica rapa</i> is grown as a fodder crop. "Raphanofortii" is the allopolyploid hybrid between <i>Brassica tournefortii</i> and <i>Raphanus caudatus</i>. The <i>Raphanobrassica</i> is a fascinating plant, because (in spite of its hybrid nature), it is not sterile. This has led some botanists to propose that the accidental hybridization of a flower by pollen of another species in nature could be a mechanism of speciation common in higher plants.
<span style="border: 3px solid blue; color:blue;font-weight: bold">126</span></p>
<h4><span id="Senecio_.28groundsel.29"></span><span class="mw-headline" id="Senecio_(groundsel)"><i>Senecio</i> (groundsel)</span></h4>
<p>The Welsh groundsel is an allopolyploid, a plant that contains sets of chromosomes originating from two different species. Its ancestor was <i>Senecio × baxteri</i>, an infertile hybrid that can arise spontaneously when the closely related groundsel (<i>Senecio vulgaris</i>) and Oxford ragwort (<i>Senecio squalidus</i>) grow alongside each other. Sometime in the early 20th century, an accidental doubling of the number of chromosomes in an <i>S. × baxteri</i> plant led to the formation of a new fertile species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">127</span></p><p>The York groundsel (<i>Senecio eboracensis</i>) is a hybrid species of the self-incompatible <i>Senecio squalidus</i> (also known as Oxford ragwort) and the self-compatible <i>Senecio vulgaris</i> (also known as common groundsel). Like <i>S. vulgaris</i>, <i>S. eboracensis</i> is self-compatible; however, it shows little or no natural crossing with its parent species, and is therefore reproductively isolated, indicating that strong breed barriers exist between this new hybrid and its parents. It resulted from a backcrossing of the F1 hybrid of its parents to <i>S. vulgaris</i>. <i>S. vulgaris</i> is native to Britain, while <i>S. squalidus</i> was introduced from Sicily in the early 18th century; therefore, <i>S. eboracensis</i> has speciated from those two species within the last 300 years.
<span style="border: 3px solid blue; color:blue;font-weight: bold">128</span></p><p>Other hybrids descended from the same two parents are known. Some are infertile, such as <i>S.</i> x <i>baxteri</i>. Other fertile hybrids are also known, including <i>S. vulgaris</i> var. <i>hibernicus</i>, now common in Britain, and the allohexaploid <i>S. cambrensis</i>, which according to molecular evidence probably originated independently at least three times in different locations. Morphological and genetic evidence support the status of <i>S. eboracensis</i> as separate from other known hybrids.
<span style="border: 3px solid blue; color:blue;font-weight: bold">129</span></p>
<h4><span class="mw-headline" id="Thale_cress">Thale cress</span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:152px;"><img alt="" class="thumbimage" data-file-height="635" data-file-width="458" height="208" src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6f/Arabidopsis_thaliana.jpg/150px-Arabidopsis_thaliana.jpg" srcset width="150"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>13</h1></center></div><b>Figure 6c</b>: <i>Arabidopsis thaliana</i> (colloquially known as thale cress, mouse-ear cress or arabidopsis)</div></div></div>
<p>Kirsten Bomblies et al. from the Max Planck Institute for Developmental Biology discovered two genes in the thale cress plant, <i>Arabidopsis thaliana</i>. When both genes are inherited by an individual, it ignites a reaction in the hybrid plant that turns its own immune system against it. In the parents, the genes were not detrimental, but they evolved separately to react defectively when combined. To test this, Bomblies crossed 280 genetically different strains of <i>Arabidopsis</i> in 861 distinct ways and found that 2 percent of the resulting hybrids were necrotic. Along with allocating the same indicators, the 20 plants also shared a comparable collection of genetic activity in a group of 1,080 genes. In almost all of the cases, Bomblies discovered that only two genes were required to cause the autoimmune response. Bomblies looked at one hybrid in detail and found that one of the two genes belonged to the NB-LRR class, a common group of disease resistance genes involved in recognizing new infections. When Bomblies removed the problematic gene, the hybrids developed normally. Over successive generations, these incompatibilities could create divisions between different plant strains, reducing their chances of successful mating and turning distinct strains into separate species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">130</span></p>
<h4><span id="Tragopogon_.28salsify.29"></span><span class="mw-headline" id="Tragopogon_(salsify)"><i>Tragopogon</i> (salsify)</span></h4>
<div class="thumb tright"><div class="thumbinner" style="width:152px;"><img alt="" class="thumbimage" data-file-height="667" data-file-width="437" height="229" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ef/Tragopogon_porrifolius_flower.jpg/150px-Tragopogon_porrifolius_flower.jpg" srcset width="150"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>14</h1></center></div><b>Figure 6d</b>: Purple salsify, <i>Tragopogon porrifolius</i></div></div></div>
<p><i>Tragopogon</i> is one example where hybrid speciation has been observed. In the early 20th century, humans introduced three species of salsify into North America. These species, the western salsify (<i>Tragopogon dubius</i>), the meadow salsify (<i>Tragopogon pratensis</i>), and the oyster plant (<i>Tragopogon porrifolius</i>), are now common weeds in urban wastelands. In the 1950s, botanists found two new species in the regions of Idaho and Washington, where the three already known species overlapped. One new species, <i>Tragopogon miscellus</i>, is a tetraploid hybrid of <i>T. dubius</i> and <i>T. pratensis</i>. The other new species, <i>Tragopogon mirus</i>, is also an allopolyploid, but its ancestors were <i>T. dubius</i> and <i>T. porrifolius</i>. These new species are usually referred to as "the Ownbey hybrids" after the botanist who first described them. The <i>T. mirus</i> population grows mainly by reproduction of its own members, but additional episodes of hybridization continue to add to the <i>T. mirus</i> population.
<span style="border: 3px solid blue; color:blue;font-weight: bold">131</span></p><p><i>T. dubius</i> and <i>T. pratensis</i> mated in Europe but were never able to hybridize. A study published in March 2011 found that when these two plants were introduced to North America in the 1920s, they mated and doubled the number of chromosomes in there hybrid <i>Tragopogon miscellus</i> allowing for a "reset" of its genes, which in turn, allows for greater genetic variation. Professor Doug Soltis of the University of Florida said, "We caught evolution in the act…New and diverse patterns of gene expression may allow the new species to rapidly adapt in new environments". This observable event of speciation through hybridization further advances the evidence for the common descent of organisms and the time frame in which the new species arose in its new environment. The hybridizations have been reproduced artificially in laboratories from 2004 to present day.
<span style="border: 3px solid blue; color:blue;font-weight: bold">132</span></p>
<h3><span class="mw-headline" id="Vertebrates_3">Vertebrates</span></h3>
<h4><span class="mw-headline" id="Blackcap">Blackcap</span></h4>
<p>The bird species, <i>Sylvia atricapilla</i>, commonly referred to as blackcaps, lives in Germany and flies southwest to Spain while a smaller group flies northwest to Great Britain during the winter. Gregor Rolshausen from the University of Freiburg found that the genetic separation of the two populations is already in progress. The differences found have arisen in about 30 generations. With DNA sequencing, the individuals can be assigned to a correct group with an 85% accuracy. Stuart Bearhop from the University of Exeter reported that birds wintering in England tend to mate only among themselves, and not usually with those wintering in the Mediterranean. It is still inference to say that the populations will become two different species, but researchers expect it due to the continued genetic and geographic separation.
<span style="border: 3px solid blue; color:blue;font-weight: bold">133</span></p>
<h4><span class="mw-headline" id="Mollies">Mollies</span></h4>
<p>The shortfin molly (<i>Poecilia mexicana</i>) is a small fish that lives in the Sulfur Caves of Mexico. Years of study on the species have found that two distinct populations of mollies—the dark interior fish and the bright surface water fish—are becoming more genetically divergent. The populations have no obvious barrier separating the two; however, it was found that the mollies are hunted by a large water bug (<i>Belostoma spp</i>). Tobler collected the bug and both types of mollies, placed them in large plastic bottles, and put them back in the cave. After a day, it was found that, in the light, the cave-adapted fish endured the most damage, with four out of every five stab-wounds from the water bugs sharp mouthparts. In the dark, the situation was the opposite. The mollies' senses can detect a predator's threat in their own habitats, but not in the other ones. Moving from one habitat to the other significantly increases the risk of dying. Tobler plans on further experiments, but believes that it is a good example of the rise of a new species.
<span style="border: 3px solid blue; color:blue;font-weight: bold">134</span></p>
<h4><span class="mw-headline" id="Polar_bear">Polar bear</span></h4>
<div class="hatnote navigation-not-searchable" role="note">See also: Polar bear § Taxonomy and evolution, and Grizzly–polar bear hybrid</div>
<p>A remarkable example of natural selection, geographic isolation, and speciation in progress is the relationship between the polar bear (<i>Ursus maritimus</i>) and the brown bear (<i>Ursus arctos</i>). Considered separate species throughout their ranges; however, it has been documented that they possess the capability to interbreed and produce fertile offspring. This introgressive hybridization has occurred both in the wild and in captivity and has been documented and verified with DNA testing. The oldest known fossil evidence of polar bears dates around 130,000 to 110,000 years ago; however, molecular data has revealed varying estimates of divergence time. Mitochondrial DNA analysis has given an estimate of 150,000 years ago while nuclear genome analysis has shown an approximate divergence of 603,000 years ago. Recent research using the complete genomes (rather than mtDNA or partial nuclear genomes) establishes the divergence of polar and brown bears between 479-343 thousand years ago. Despite the differences in divergence rates, molecular research suggests the sister species have undergone a highly complex process of speciation and admixture between the two.
<span style="border: 3px solid blue; color:blue;font-weight: bold">135</span></p><p>Polar bears have acquired significant anatomical and physiological differences from the brown bear that allow it to comfortably survive in conditions that the brown bear likely could not. Notable examples include the ability to swim sixty miles or more at a time in freezing waters, fur that blends with the snow, and to stay warm in the arctic environment, an elongated neck that makes it easier to keep their heads above water while swimming, and oversized and heavy-matted webbed feet that act as paddles when swimming. It has also evolved small papillae and vacuole-like suction cups on the soles to make them less likely to slip on the ice, alongside smaller ears for a reduction of heat loss, eyelids that act like sunglasses, accommodations for their all-meat diet, a large stomach capacity to enable opportunistic feeding, and the ability to fast for up to nine months while recycling their urea. This example presents a macro-evolutionary change involving an amalgamation of several fields of evolutionary biology, e.g. adaptation through natural selection, geographic isolation, speciation, and hybridization.
<span style="border: 3px solid blue; color:blue;font-weight: bold">136</span></p>
<h2><span class="mw-headline" id="Evidence_from_coloration">Evidence from coloration</span></h2>
<div class="hatnote navigation-not-searchable" role="note">Main article: Coloration evidence for natural selection</div>
<div class="thumb tright"><div class="thumbinner" style="width:292px;"><img alt="" class="thumbimage" data-file-height="1323" data-file-width="1764" height="218" src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Rock_Ptarmigan_%28Lagopus_Muta%29.jpg/290px-Rock_Ptarmigan_%28Lagopus_Muta%29.jpg" srcset width="290"/> <div class="thumbcaption"><div class="magnify"></div><div style='border: 3px solid red'><center><h1 style='color:red'>15</h1></center></div>Natural selection has driven the ptarmigan to change from snow camouflage in winter to disruptive coloration suiting moorland in summer.</div></div></div>
<p>Animal coloration provided important early evidence for evolution by natural selection, at a time when little direct evidence was available. Three major functions of coloration were discovered in the second half of the 19th century, and subsequently used as evidence of selection: camouflage (protective coloration); mimicry, both Batesian and Müllerian; and aposematism. After the circumstantial evidence provided by Darwin in <i>On the Origin of Species</i>, and given the absence of mechanisms for genetic variation or heredity at that time, naturalists including Darwin's contemporaries, Henry Walter Bates and Fritz Müller sought evidence from what they could observe in the field.
<span style="border: 3px solid blue; color:blue;font-weight: bold">137</span></p>
<h3><span class="mw-headline" id="Mimicry_and_aposematism">Mimicry and aposematism</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Further information: Mimicry</div>
<p>Bates and Müller described forms of mimicry that now carry their names, based on their observations of tropical butterflies. These highly specific patterns of coloration are readily explained by natural selection, since predators such as birds which hunt by sight will more often catch and kill insects that are less good mimics of distasteful models than those that are better mimics; but the patterns are otherwise hard to explain. Darwinists such as Alfred Russel Wallace and Edward Bagnall Poulton, and in the 20th century Hugh Cott and Bernard Kettlewell, sought evidence that natural selection was taking place.
<span style="border: 3px solid blue; color:blue;font-weight: bold">138</span></p>
<h3><span class="mw-headline" id="Camouflage">Camouflage</span></h3>
<div class="hatnote navigation-not-searchable" role="note">Further information: Camouflage, Snow camouflage, and Disruptive coloration</div>
<p>In 1889, Wallace noted that snow camouflage, especially plumage and pelage that changed with the seasons, suggested an obvious explanation as an adaptation for concealment. Poulton's 1890 book, <i>The Colours of Animals</i>, written during Darwinism's lowest ebb, used all the forms of coloration to argue the case for natural selection. Cott described many kinds of camouflage, mimicry and warning coloration in his 1940 book <i>Adaptive Coloration in Animals</i>, and in particular his drawings of coincident disruptive coloration in frogs convinced other biologists that these deceptive markings were products of natural selection. Kettlewell experimented on peppered moth evolution, showing that the species had adapted as pollution changed the environment; this provided compelling evidence of Darwinian evolution.
<span style="border: 3px solid blue; color:blue;font-weight: bold">139</span></p>
<h2><span class="mw-headline" id="Evidence_from_mathematical_modeling">Evidence from mathematical modeling</span></h2>
<p>Computer science allows the iteration of self-changing complex systems to be studied, allowing a mathematical understanding of the nature of the processes behind evolution; providing evidence for the hidden causes of known evolutionary events. The evolution of specific cellular mechanisms like spliceosomes that can turn the cell's genome into a vast workshop of billions of interchangeable parts that can create tools that create tools that create tools that create us can be studied for the first time in an exact way.
<span style="border: 3px solid blue; color:blue;font-weight: bold">140</span></p><p>"It has taken more than five decades, but the electronic computer is now powerful enough to simulate evolution", assisting bioinformatics in its attempt to solve biological problems.
<span style="border: 3px solid blue; color:blue;font-weight: bold">141</span></p><p>Computational evolutionary biology has enabled researchers to trace the evolution of a large number of organisms by measuring changes in their DNA, rather than through physical taxonomy or physiological observations alone. It has compared entire genomes permitting the study of more complex evolutionary events, such as gene duplication, horizontal gene transfer, and the prediction of factors important in speciation. It has also helped build complex computational models of populations to predict the outcome of the system over time and track and share information on an increasingly large number of species and organisms.
<span style="border: 3px solid blue; color:blue;font-weight: bold">142</span></p><p>Future endeavors are to reconstruct a now more complex tree of life.
<span style="border: 3px solid blue; color:blue;font-weight: bold">143</span></p><p>Christoph Adami, a professor at the Keck Graduate Institute made this point in <i>Evolution of biological complexity</i>:
<span style="border: 3px solid blue; color:blue;font-weight: bold">144</span></p>
<dl><dd>To make a case for or against a trend in the evolution of complexity in biological evolution, complexity must be both rigorously defined and measurable. A recent information-theoretic (but intuitively evident) definition identifies genomic complexity with the amount of information a sequence stores about its environment. We investigate the evolution of genomic complexity in populations of digital organisms and monitor in detail the evolutionary transitions that increase complexity. We show that, because natural selection forces genomes to behave as a natural "Maxwell Demon", within a fixed environment, genomic complexity is forced to increase.</dd></dl>
<p>David J. Earl and Michael W. Deem—professors at Rice University made this point in <i>Evolvability is a selectable trait</i>:
<span style="border: 3px solid blue; color:blue;font-weight: bold">145</span></p>
<dl><dd>Not only has life evolved, but life has evolved to evolve. That is, correlations within protein structure have evolved, and mechanisms to manipulate these correlations have evolved in tandem. The rates at which the various events within the hierarchy of evolutionary moves occur are not random or arbitrary but are selected by Darwinian evolution. Sensibly, rapid or extreme environmental change leads to selection for greater evolvability. This selection is not forbidden by causality and is strongest on the largest-scale moves within the mutational hierarchy. Many observations within evolutionary biology, heretofore considered evolutionary happenstance or accidents, are explained by selection for evolvability. For example, the vertebrate immune system shows that the variable environment of antigens has provided selective pressure for the use of adaptable codons and low-fidelity polymerases during somatic hypermutation. A similar driving force for biased codon usage as a result of productively high mutation rates is observed in the hemagglutinin protein of influenza A.</dd></dl>
<p>"Computer simulations of the evolution of linear sequences have demonstrated the importance of recombination of blocks of sequence rather than point mutagenesis alone. Repeated cycles of point mutagenesis, recombination, and selection should allow <i>in vitro</i> molecular evolution of complex sequences, such as proteins." Evolutionary molecular engineering, also called directed evolution or <i>in vitro</i> molecular evolution involves the iterated cycle of mutation, multiplication with recombination, and selection of the fittest of individual molecules (proteins, DNA, and RNA). Natural evolution can be relived showing us possible paths from catalytic cycles based on proteins to based on RNA to based on DNA.
<span style="border: 3px solid blue; color:blue;font-weight: bold">146</span></p>
<script>function offsetAnchor() {if(location.hash.length !== 0) {window.scrollTo(window.scrollX, window.scrollY - 250);}}window.addEventListener("hashchange", offsetAnchor);window.setTimeout(offsetAnchor, 1);</script>